{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Good Notebooks to refer back to\n",
    "[1] https://www.kaggle.com/ivanegapratama/covid-eda-initial-exploration-tool\n",
    "\n",
    "[2] https://www.kaggle.com/midnitekoder/covid-19-citation-graph-embedding-using-deepwalk\n",
    "\n",
    "[3] https://www.kaggle.com/xhlulu/cord-19-eda-parse-json-and-generate-clean-csv\n",
    "\n",
    "[4] https://www.kaggle.com/maksimeren/covid-19-literature-clustering\n",
    "\n",
    "[5] https://www.kaggle.com/nofoosports/cord-19-analysis-with-sentence-embeddings/data#Building-task-reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# imports\n",
    "# inital code was mostly adapted from https://www.kaggle.com/ivanegapratama/covid-eda-initial-exploration-tool\n",
    "import os\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import matplotlib\n",
    "import glob\n",
    "import json\n",
    "import spacy\n",
    "import re\n",
    "import nltk\n",
    "import matplotlib.pyplot as plt\n",
    "plt.style.use('ggplot')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# read in metadata\n",
    "meta_df = pd.read_csv('metadata.csv', dtype={\n",
    "    'pubmed_id': str,\n",
    "    'Microsoft Academic Paper ID': str, \n",
    "    'doi': str\n",
    "})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Int64Index: 44220 entries, 0 to 44219\n",
      "Data columns (total 15 columns):\n",
      "sha                            28462 non-null object\n",
      "source_x                       44220 non-null object\n",
      "title                          43996 non-null object\n",
      "doi                            40750 non-null object\n",
      "pmcid                          23319 non-null object\n",
      "pubmed_id                      22943 non-null object\n",
      "license                        44220 non-null object\n",
      "abstract                       35806 non-null object\n",
      "publish_time                   34197 non-null object\n",
      "authors                        41074 non-null object\n",
      "journal                        33173 non-null object\n",
      "Microsoft Academic Paper ID    964 non-null object\n",
      "WHO #Covidence                 1767 non-null object\n",
      "has_full_text                  44220 non-null bool\n",
      "full_text_file                 32829 non-null object\n",
      "dtypes: bool(1), object(14)\n",
      "memory usage: 5.1+ MB\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>sha</th>\n",
       "      <th>source_x</th>\n",
       "      <th>title</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>license</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "      <th>journal</th>\n",
       "      <th>microsoft_academic_paper_id</th>\n",
       "      <th>who_#covidence</th>\n",
       "      <th>has_full_text</th>\n",
       "      <th>full_text_file</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1297</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Elsevier</td>\n",
       "      <td>An improved method for the routine identificat...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>els-covid</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1978-07-31</td>\n",
       "      <td>Caul, E.O.; Ashley, C.R.; Egglestone, S.I.</td>\n",
       "      <td>FEMS Microbiology Letters</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>False</td>\n",
       "      <td>custom_license</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1298</th>\n",
       "      <td>84aa394458d6daea92ecbcd504b90ed9fadb56e6</td>\n",
       "      <td>Elsevier</td>\n",
       "      <td>Translation of infectious bronchitis virus RNA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>els-covid</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1978-04-30</td>\n",
       "      <td>Highfield, P.E.; Morser, J.; Lomniczi, B.; Ste...</td>\n",
       "      <td>FEMS Microbiology Letters</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1299</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Elsevier</td>\n",
       "      <td>Author index to volume 5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>els-covid</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1979-06-30</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FEMS Microbiology Letters</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>False</td>\n",
       "      <td>custom_license</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1300</th>\n",
       "      <td>NaN</td>\n",
       "      <td>Elsevier</td>\n",
       "      <td>Subject index to volume 5 (1979)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>els-covid</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1979-06-30</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FEMS Microbiology Letters</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>False</td>\n",
       "      <td>custom_license</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1301</th>\n",
       "      <td>133db9d6eab9f982c996b3e2bdbacd782db63ed7</td>\n",
       "      <td>Elsevier</td>\n",
       "      <td>Preliminary studies on the isolation of corona...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>els-covid</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1979-02-28</td>\n",
       "      <td>Caul, E.O.; Ashley, C.R.; Ferguson, Morag; Egg...</td>\n",
       "      <td>FEMS Microbiology Letters</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40974</th>\n",
       "      <td>786da349a33148e24e09edd6378da80ea2f40e07; 2810...</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Common variations in TERT-CLPTM1L locus are re...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PMC4808031</td>\n",
       "      <td>26621837</td>\n",
       "      <td>cc-by</td>\n",
       "      <td>Associations between single nucleotide polymor...</td>\n",
       "      <td>2015 Nov 26</td>\n",
       "      <td>Zhang, Yang; Zhang, Xiaoai; Zhang, Hongxing; Z...</td>\n",
       "      <td>Oncotarget</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>comm_use_subset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41190</th>\n",
       "      <td>8e014e2133b319d93ba7e9341ffe33bab3f9e265</td>\n",
       "      <td>PMC</td>\n",
       "      <td>A HLA-A2-restricted CTL epitope induces anti-t...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PMC4808025</td>\n",
       "      <td>26621839</td>\n",
       "      <td>cc-by</td>\n",
       "      <td>Cancer immunotherapy is attractive for antigen...</td>\n",
       "      <td>2015 Nov 26</td>\n",
       "      <td>Sher, Yuh-Pyng; Lin, Su-I; Chen, I-Hua; Liu, H...</td>\n",
       "      <td>Oncotarget</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>comm_use_subset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41212</th>\n",
       "      <td>ad4f830c6a0985b878a66d4e797232ccba91aedc; 9de4...</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Targeting a ribonucleoprotein complex containi...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PMC4385842</td>\n",
       "      <td>25669982</td>\n",
       "      <td>cc-by</td>\n",
       "      <td>Tylophorine compounds have been the focus of d...</td>\n",
       "      <td>2014 Dec 10</td>\n",
       "      <td>Qiu, Ya-Qi; Yang, Cheng-Wei; Lee, Yue-Zhi; Yan...</td>\n",
       "      <td>Oncotarget</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>comm_use_subset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41930</th>\n",
       "      <td>NaN</td>\n",
       "      <td>WHO</td>\n",
       "      <td>Medicine's Challenges: Vaping and Coronavirus</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PMC7023952</td>\n",
       "      <td>32158035</td>\n",
       "      <td>unk</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020</td>\n",
       "      <td>DiRenna, James</td>\n",
       "      <td>Mo Med</td>\n",
       "      <td>NaN</td>\n",
       "      <td>#7371</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43512</th>\n",
       "      <td>c9708301ac6326b1214f31ab93fcd4b72485cf4b</td>\n",
       "      <td>PMC</td>\n",
       "      <td>GCN5 inhibits XBP-1S-mediated transcription by...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PMC4381594</td>\n",
       "      <td>25426559</td>\n",
       "      <td>cc-by</td>\n",
       "      <td>Cellular unfolded protein response (UPR) is in...</td>\n",
       "      <td>2014 Nov 16</td>\n",
       "      <td>Lew, Qiao Jing; Chu, Kai Ling; Chia, Yi Ling; ...</td>\n",
       "      <td>Oncotarget</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>comm_use_subset</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3470 rows Ã— 15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                     sha  source_x  \\\n",
       "1297                                                 NaN  Elsevier   \n",
       "1298            84aa394458d6daea92ecbcd504b90ed9fadb56e6  Elsevier   \n",
       "1299                                                 NaN  Elsevier   \n",
       "1300                                                 NaN  Elsevier   \n",
       "1301            133db9d6eab9f982c996b3e2bdbacd782db63ed7  Elsevier   \n",
       "...                                                  ...       ...   \n",
       "40974  786da349a33148e24e09edd6378da80ea2f40e07; 2810...       PMC   \n",
       "41190           8e014e2133b319d93ba7e9341ffe33bab3f9e265       PMC   \n",
       "41212  ad4f830c6a0985b878a66d4e797232ccba91aedc; 9de4...       PMC   \n",
       "41930                                                NaN       WHO   \n",
       "43512           c9708301ac6326b1214f31ab93fcd4b72485cf4b       PMC   \n",
       "\n",
       "                                                   title  doi       pmcid  \\\n",
       "1297   An improved method for the routine identificat...  NaN         NaN   \n",
       "1298      Translation of infectious bronchitis virus RNA  NaN         NaN   \n",
       "1299                            Author index to volume 5  NaN         NaN   \n",
       "1300                    Subject index to volume 5 (1979)  NaN         NaN   \n",
       "1301   Preliminary studies on the isolation of corona...  NaN         NaN   \n",
       "...                                                  ...  ...         ...   \n",
       "40974  Common variations in TERT-CLPTM1L locus are re...  NaN  PMC4808031   \n",
       "41190  A HLA-A2-restricted CTL epitope induces anti-t...  NaN  PMC4808025   \n",
       "41212  Targeting a ribonucleoprotein complex containi...  NaN  PMC4385842   \n",
       "41930      Medicine's Challenges: Vaping and Coronavirus  NaN  PMC7023952   \n",
       "43512  GCN5 inhibits XBP-1S-mediated transcription by...  NaN  PMC4381594   \n",
       "\n",
       "      pubmed_id    license                                           abstract  \\\n",
       "1297        NaN  els-covid                                                NaN   \n",
       "1298        NaN  els-covid                                                NaN   \n",
       "1299        NaN  els-covid                                                NaN   \n",
       "1300        NaN  els-covid                                                NaN   \n",
       "1301        NaN  els-covid                                                NaN   \n",
       "...         ...        ...                                                ...   \n",
       "40974  26621837      cc-by  Associations between single nucleotide polymor...   \n",
       "41190  26621839      cc-by  Cancer immunotherapy is attractive for antigen...   \n",
       "41212  25669982      cc-by  Tylophorine compounds have been the focus of d...   \n",
       "41930  32158035        unk                                                NaN   \n",
       "43512  25426559      cc-by  Cellular unfolded protein response (UPR) is in...   \n",
       "\n",
       "      publish_time                                            authors  \\\n",
       "1297    1978-07-31         Caul, E.O.; Ashley, C.R.; Egglestone, S.I.   \n",
       "1298    1978-04-30  Highfield, P.E.; Morser, J.; Lomniczi, B.; Ste...   \n",
       "1299    1979-06-30                                                NaN   \n",
       "1300    1979-06-30                                                NaN   \n",
       "1301    1979-02-28  Caul, E.O.; Ashley, C.R.; Ferguson, Morag; Egg...   \n",
       "...            ...                                                ...   \n",
       "40974  2015 Nov 26  Zhang, Yang; Zhang, Xiaoai; Zhang, Hongxing; Z...   \n",
       "41190  2015 Nov 26  Sher, Yuh-Pyng; Lin, Su-I; Chen, I-Hua; Liu, H...   \n",
       "41212  2014 Dec 10  Qiu, Ya-Qi; Yang, Cheng-Wei; Lee, Yue-Zhi; Yan...   \n",
       "41930         2020                                     DiRenna, James   \n",
       "43512  2014 Nov 16  Lew, Qiao Jing; Chu, Kai Ling; Chia, Yi Ling; ...   \n",
       "\n",
       "                         journal microsoft_academic_paper_id who_#covidence  \\\n",
       "1297   FEMS Microbiology Letters                         NaN            NaN   \n",
       "1298   FEMS Microbiology Letters                         NaN            NaN   \n",
       "1299   FEMS Microbiology Letters                         NaN            NaN   \n",
       "1300   FEMS Microbiology Letters                         NaN            NaN   \n",
       "1301   FEMS Microbiology Letters                         NaN            NaN   \n",
       "...                          ...                         ...            ...   \n",
       "40974                 Oncotarget                         NaN            NaN   \n",
       "41190                 Oncotarget                         NaN            NaN   \n",
       "41212                 Oncotarget                         NaN            NaN   \n",
       "41930                     Mo Med                         NaN          #7371   \n",
       "43512                 Oncotarget                         NaN            NaN   \n",
       "\n",
       "       has_full_text   full_text_file  \n",
       "1297           False   custom_license  \n",
       "1298            True   custom_license  \n",
       "1299           False   custom_license  \n",
       "1300           False   custom_license  \n",
       "1301            True   custom_license  \n",
       "...              ...              ...  \n",
       "40974           True  comm_use_subset  \n",
       "41190           True  comm_use_subset  \n",
       "41212           True  comm_use_subset  \n",
       "41930          False              NaN  \n",
       "43512           True  comm_use_subset  \n",
       "\n",
       "[3470 rows x 15 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# what I ran to get an initial look at the metadata\n",
    "meta_df.columns\n",
    "meta_df.dropna(subset = ['has_full_text']).info()\n",
    "meta_df['has_full_text'].value_counts()\n",
    "meta_df.columns = meta_df.columns.str.strip().str.lower().str.replace(' ', '_').str.replace('(', '').str.replace(')', '')\n",
    "meta_df.columns\n",
    "meta_df[meta_df.full_text_file.isnull() == False].query('has_full_text == False').head()\n",
    "meta_df[meta_df.full_text_file.isnull() == False]\n",
    "meta_df[meta_df.doi.isnull() == True]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0015023cc06b5362d332b3baf348d11567ca2fbb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\004f0f8bb66cf446678dc13cf2701feec4f36d76.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\00d16927588fb04d4be0e6b269fc02f0d3c2aa7b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0139ea4ca580af99b602c6435368e7fdbefacb03.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\013d9d1cba8a54d5d3718c229b812d7cf91b6c89.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\01d162d7fae6aaba8e6e60e563ef4c2fca7b0e18.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\01e3b313e78a352593be2ff64927192af66619b5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\02201e4601ab0eb70b6c26480cf2bfeae2625193.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0255ea4b2f26a51a3bfa3bd8f3e1978c82c976d5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\029c1c588047f1d612a219ee15494d2d19ff7439.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\03ce432f27c7df6af22b92245a614db2ecb5de5f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\03ea3a614b56409d3f099c9ad764864293132540.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\04030bba3035a58c7725ae267973206f6eb6c0b4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\041bae0a6de2b69979d39460b3f2ee8946534ec2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\05082393ba4c7ec530190dd887d99c74fd72f6d6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\05d47dd5b46f86428de058db4ecc8bca76a9ad16.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\05d99c07db59b6948e39bfa62c2cbbf62944059a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0624a12abfe85c8b5070850d912a2db4cd453236.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\06a1002f9fbea7179ac3572843f66b14568af6e4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\06c1b3535b83251cf92c01258b5048beeab7a460.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\078b9a1b8e59f6b032e37b4a07fc73c59e1f7332.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\07e833d0917cace550853f72923856d0fe1a7120.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\08660499ee722a74043f8417faee3e1eeb9d0f5f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\08a22278486e12768ce186677a6a89663d24586f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\090b6c8b3df30bc248221869f673a2d970caa1b9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\091a8e9a61e19e88caeb039f0e3888d111b20439.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\09b6706748f0c1ae0da436ac2dfac9052b84e4ea.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\09c9fcabc66a106e01ef42247cbd86b6d85bd67f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\09ec8daa8e32168d92d05b86de1784c639685fb4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0a32446730827ad8152c6a61e4738e4e0b231412.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0a43046c154d0e521a6c425df215d90f3c62681e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0ab795fc615df6457551a8e231dce1f268eef9d2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0acc1f9a1c333a9a6b2dbba4a252d7576f024783.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0aff00101d5ccc6592987185ab833f95d842f98b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0b282573f5c63c943021c10ca39a1ed21acfb429.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0b460e73926eb107001c95c0dfeae2362b4251b0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0b4b4e5bb8d0d5167eec1e203b5dad283bd364a5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0bd443591b5e10934beef050b516afeca6b668fe.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0c27c0ddaa4761f6155838df81a88d24619720f8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0cad40f50e41144e8b64744251515eca8cfac5f7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0cb9c296684ca5e71462d825cab2827854a01544.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0d4432c32daefa6f0d711ca35eca81188e0bccdf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0d7964c665ff0107c674bc6dab8f252cab08038e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0da2ec30d7dfdef624833a36890f0297f19d09ec.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0e2ba405f636e06821999876cd82c00e26404b43.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0e38333bff68345492526fd39b70d1b18969cb83.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0eda8331214ca028350b07e2953702f3078a105e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0eee9760aee0ca2fdf8d1d215c3689d5f8d84df5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0fa18c47543c3ce69a3c9e4d77958c4d3691b1ed.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0fbb18050e29ca78191625d42576b1c574027377.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0fd300aefb704c20f32152b97b6194015f1c74e7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\0ff4ad5359ee2df3568605385a8616d8da66bf2c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\10138816c67a562b55bc9c2b882b62c81fc4b062.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\10424da2599f0258bfaf71a3186b999ab2d747d0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\10525ac89e46be4cb9cb9fd1131d28411a902047.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\107f0d5a47f48f85d6645cac0e7008edc4d9d8cd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\10abb3e977aa476a39bce4f1feda37c2e1250106.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\110b4b089bd757bd54587c2ab3262a374462c098.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\111b9a6e91c938696fcdb4cb128b8ae739dbe11c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\114ed64f52f503d9d2e2ba1fd1ee62b0a168cd84.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\11875dfda6cd04d5032611edf0e340582ceb620f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\11edcc84add7b4bfe1f7cc84068a06ff9b7398b2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\11f96d05db6854ef95312aa3a4736724ce1f02d6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1218f278a4f8d83dac14b23c8f698062812ef9d5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\123db7cdec9fc63aa09782ea9f4267603be8f74d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\12920263b2846c61d6f2b6105189367a8ff1bc39.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\12c8ef5614d9a35bed3ff95a3b1dc971842514fe.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\12fac9aedb1a09a3922a3c084ce4723708e463d6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\136ee862ad53b606c8f2bea917b9705f5079e88e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\138d4617f4f4984bec9cf2c7540fed533d9e4d6e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\13bdc1078ca5091bfd346a4ecc5dacb6566480d9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\13bf9fe0df944c0ee0debb0d445527ad4418cc19.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\14220553c9160c06bb99f59d59e6a8ecf3595226.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\14783283da3482bf2aab4e6100fe4ea5c29d8db7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\158e546658e42ab0a89d174bc9facb437a034df6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\15ad534bb2ec7bc4b0ee83ddac11a1b837d743c6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\15bad8368522a05f32dee771e5a8259f0e9cbbfd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\15e5dd0eb126bfe384c25ba551317eb9ba2ad1ef.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\160ecaaa5a766c289fc2f6b5499f0dfe7aab971c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\164b0678afc42f1923aa100b624ca969a53fb3b2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\179df1e769292dd113cef1b54b0b43213e6b5c97.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\180c88b68816bbb6eb57aa515a3aec4020178729.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\181b7b57851e6f58a601b68e613d10c10616f774.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\18dc6a67045d2687d2b5c11c85797f42824ed243.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1983cbc32412a9a7b1601b6778605f9e98ab69e0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1a12cc7d49d8521bec5d447ebb413cbcf5aca8f4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1a791c5c6543aca9391b77ae58b79d7ddf3c1f3c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1a9c229f4f866db4f9c23a77b8c1275b52407c64.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1aa3e788fc6b03c147e37200c3b011ca7a289a5c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1ad1d4b84aea4ceaf05c62a1ad04e7150f7f4684.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1b4ab28a9391d516e1753874c73d2d244104f825.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1b531d266b4933bf8040739ed199914655b075d6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1b9fed2824e97db8cb35912549dd2e9cde7b6c18.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1c02f14de3ae4c5e26600b073ec6396caefb3830.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1c06378f629bfa8870c89b3abfdecfc19db9b459.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1c0b25c8e3c89243fd8797e2e964e12689bc8556.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1c66ec8950cbd1e0f0d4bc4a43ba9c79411cc5fa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1d2947d5c0addcbbcfa71d87ea074a8b6ff2a973.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1d882d90109fb4496088342cf2bcaa4fec40cf9f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1dd898b5ca1ae70ec0e3cad89fc87a165002a99e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1df9ce60e358f6cef0be3f6e15fe2a9c6f93f39c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1dfcd4615a8763e053bd2fdb845f09fab68109f1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1e45cd5feb7928bbc8bcd524d9c6b7adeae08a6b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1eaa329f608055620a57e6273e9d1c409de1e9ee.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1ee4cc235ce07b2d08d989e9001e2f4bd9af3138.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1f783a5e029f80516169b19fe6e0dacc1e171f87.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1f9d3f9a1a0e8db6a086e0a2b5ba50cf9f235dae.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\1fb909712eea684a4fae4f0b419ca6bc7ba8e396.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\20154fa12ca525582cd1b42019927e8f7816fb51.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\207dcb6e3cd43cdea91d29a15be1ee34068bd54e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\20bfe4f15b7b46b2754c06ef6ea1ba801cc48c55.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\20f2a638e2c9a1917b7c001c371335da3d288505.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\210b30ec755f5d8b0622c78a8e570290293bf49f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\21100aba41a4bfb48d7dc37f1bf5dbb38bf3867a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\221f592feb12c19df1f3e619021205e247d39b46.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\22264c65eb6c14ba96287ba4da536424dd083253.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\224a389bd5dbeab45b8a25965298016fbc44d669.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\228edf7f7e53e88e246fc77b7b7664ac35d7278e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\22a4d26bde67d878869443ea26140f3d8239bed0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\22f3be8bbf6d792549b7a6088df7440e425cb34f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\233978852b45429db49d2aabe1c9c9bfe73b1884.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2365df72b5a011f1a1a38b615f6cd87a126d311e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\23f1bab7d94bd0564e2319fddf75c1c7987ce75f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\24e17488d399c436305c819953beae2961214771.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\24ff82a421f1a33db3ebaa22652bd62e0cb77c57.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2536b83acf84368d7c13be81fe07aa0575115da7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\254c1e4d661d7fb5954c855387595c3cde0e8156.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\264a816e7099246c13a5dad3a581a474d85d50bd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\267729947ca478946d5a4bffb8e13d50c3545120.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\26cb6703ca72bf97887abbc29a40b1bd9d7890f4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\26e75d3c815aae7fd9b094c3e5c74d3f7132ca13.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\26f1c42474966b8daede2912e06175bd8b864f8e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2757708c4f72260abefb9d74302d868165cbc1a4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\278e2c52630f1dbbda6a79db15a2260510586150.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\28884786bfd579216f563c278e1730290dd4ccb8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\28b107243576723248ad4053261000311a22f134.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\28b53e0cab53b10ab87431d6cc4ac1e0a7c4d6b9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2913d91f13fd59c698f68ba63008d8e0550c0607.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\295036c0448244642eb6533de571def6cd8437e5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2989bf36068372cfedef82f58f4777e60ccbcfaa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\29b3eaeacecf2bfc95b9a493dbb1ba4b2323dd46.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\29bc34aad802ab6b35074f23c631540a84b16d0d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2ae710bf5f60c23c489e4c61245a35a4f85178cb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2af91bbb625d983d5ad3aadb9e5f1d82268a0f1f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2afff5fe89a3e3ec270a41c2b929dfeec68d7d50.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2ba7dfd6227e7f64d80204770ccbe140afa78117.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2c18d80cfc792faa3d6e35c72d4b6b58ee93fdad.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2cc809ed4d5c3640ee2351fdb1876e8ff6c01b51.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2cfe7a4cc1fceed0ad7ffbd6244a61e0b2c3ac8b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2d02192c3d251c2b561d37274350660277658a74.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2d0285c90976dfa43d58eca6b7d1d7ef29994fc2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2da0a7ba851e53d21dba05042a397eafcdab093d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2dc6766fe253b1b832d8a3074404fb5daae4762f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2e41aee6651cf4cab4edf5aa67cc401c2bef9d67.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2e5d79e1e6ebeed6cffdad2b474ca7675da969e7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2e7c17caecfe67b64f9b59df404f0761df96987f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2ea102f58147dab02e4dea90eb90dbc67149f678.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2eac7de1c407d9eab13a6d538142cd21a814ee7b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2f6efc4edc0c9fffd3acc810efd8616e7772b28a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2fe18190f7b54e589dccf1d3bbfd73e87bfd2ab9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2fe6b550f737baa47a5f2c8ab64cc3d9271c308a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2ff168d83528e6dd46e8c5358fb84fd63f7a8f2c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\2ffc83b88bc84615273e97d8109e5439ce4dce60.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3028628066ec2401f3981f4e70c5b1acd4cef573.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\30457ee3ce0001f33938fbc246b4ce4eacd74f5d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\30814fcc87b8f3e5d4f7431d75b9505b049021e0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\30b5c7b8faf95265f67ae59f4686eaf9b2772893.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\311a15d92b87b2bc41f00afed881b95e93c22fcc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\313dd0869f169624e78d770963eacb0d1641cafa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\315197247b230bab03786c197dc1fc795164c371.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\317215c463463ddb39fa7564f67239dc484b86a9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\317943dec06b88c1c62ef0ecd832f0d644b1d417.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\31e67053158e68f513214a7a33046b5cc5f26249.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3298d34e643c7a157f4226fde679f71194e417b5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\329dfdd2fbe4b3c510c5329219bfd33764ab1710.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\32c2e402c3c88c3024a4dec3d78ba67c24f89ca6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3319392ca8bd2b8f2e2d00bccdb80deaa8a77c2e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\331a7033bb8e6d4618e4e83b74aff1aec42764a6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\33565294e6bc67fb7ee14dcae6cfdb08148f4ea5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\33a6aa86bb6fe30251d7f2c1f64c5b7a94c391a0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\33bd85ea9ca61c4d0674c43ae5b901b271b46f1e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\33eff3ff8721d2abd9204e3aabd504e167f3f01d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\341fe593ea84b66223d50924bddbba252c37bc4e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3452e28df999e004bb22ce3cedc4799072810eeb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3461d71f6890f7e5ba53bf168be3945cdb16d901.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\35604e3a5d38d6794543372e4dd9ccf1d8d9cd2a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3579a2b0053bef55f93716e444173727a6c94c6d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\35ac99bfeb665ce0a2ae361dba1161e69eb8b7d5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\36521caf90f471c9da1a4e84f8562440d73ead9a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\368fb114dfd565dedb635f62b9062956b49cd475.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\36a5f6d55d7c5f67d4344e36da0a72856ad3dda0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\36e602b7081e7bb8a1cfeaa2fb92b0c43abd4fef.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3701cb3b329a07d9e6fe1986584be12f5eafe111.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\374a5295716649a80dca87097ad9ac902d344db6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\37afeaa38d66b8097cc7637bf62ac7d142e6b362.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\380a4d07d22910246c6f189fd2da93b270b96981.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\38bab3f17b186bd8ee289e5d135bb7d500ef500b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\38e796bbec2f90b1b802c14dc922102e96f6361e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\395e8569a0e6fc129029cb0b8cda1f7381954976.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3a6544f6f247366d4c12167f7fba743e9d981c63.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3a73b880acd8983f87b210ff7fee434fd6ecc8c8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3a81dbb6350d25798f401c332abcf1b49616ffef.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3afd5fba7dc182ddfa769c0d766134b525581005.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3b181a741d4beafda3ba3a8fc68239493d49f6aa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3b22eecad8a582436c52284a4db2198a98a94e18.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3b73c8abb7528c958cec0bae1890828af007b599.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3c0e6714183f9935f79f8b2e2a708833b52f00e0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3c12e50b9da4b1fde21d68040610b73b8c216bf3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3c14c4c3a176c86833794029e787ce460c9d89dd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3c476baf5542bef88b8aad92bcca6786d1afa282.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3c70c99afc7a38df3c4807857856ea258d378429.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3c94f582610453b6c1389c3a5d6ad57bcf0ba033.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3cacba057a5c55bfd6d0e821ced60cfe42e57ab6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3d498994d143a4ed7cc381700a19f2790d46bb36.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3d5093a8e079fb17262111e5ef94bf01a3007971.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3d690de86ca6e200f4f9cac61b004e47ad753aae.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3da1a03cb675f463b0d8e30744aa33512241dd29.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3ddc987158400d3867939d3594da56dd546d6ddd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3deb6393bd43f440a69c84da9ed61986b1781d36.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3e5633a9a76a8c03e4ec57e4659872a63f045117.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3e6dda036f3d57c4b1b5b318225ce2a1fccb6ab7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3ebbcdddb122f405061bb5e1ddcf94abe9258d58.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3fac0b7a9917984764a77f7fcb327e820e770d1b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\3fe2552d3d8e6c9ead2c41e30fbb69e917277f53.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\402041df2d9812dbcdae08153d0350eee170eea9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\403d885a33958731c4607ea632453e25a89e2b57.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\409c431edf54d022463f40f2d31a0dfab42f5324.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4151e9b4ea5395d73bf33309dbca26551cc7c657.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\41d412415c46baf70ca91ebf049c0352ead19f18.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\41fc7191573c2f01fdef9c8ebb072e5f54cbb598.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4380f0251c595d6cb551e643198d2dacd3c6746c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\438e9384bab6990781781719107fe187d180358a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\43f760b5cb9c3b430c6d0b31fa90d42e71a6429c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\443584f57419813b2f04054b239d4081489e80fb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4485b9462ebf52c0062692d7ab7e423c2ae05d9e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\44a440cc1c135c938d2216ea672f8ef4f9c01296.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\44b6eea3629b4e13595dc49813ee39a0d20dc5f2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\45f184e736c4a857bc0e3df0d55962f743c1493a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4602afcb8d95ebd9da583124384fd74299d20f5b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\46030e9b8451bf3b0c90833dbeacce286a263a09.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4609a2dd3568a7241548b7303e0d836e33d4d5ca.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\462cbb326ccd8587cae7a3538c8c6712d9013698.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\46634d03aa169aab9c372746ebaf3aaa65d7c7d9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\47b07518dc0f608e53198bb5f8a95caa1c164fe5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\47f65bff75b7752a38ccbb540fbd74955b3addea.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\48656efc59191537073975938f25f201524971af.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\48df5b06710467cbe5b41dd851e946fc936ecf15.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\48f486ccea11b4ba01186e287124316c962857f7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\49ac69f362c27acbc6de0c5cbb640267e7a1e797.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4a24fcf8d2a7c69aef247ea12a740145daf68407.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4a908bb4c3229f4128139e44e69529f0c2d1666d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4b1a08339bb5508f830e09f1a2c3a8022526cce0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4b56993a4d0f091e70ef5dec2e5a61c3043f752e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4b57ff228e46255a0ba7114af86ff26af4152d9b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4b5ba0d8c476c899a79cec7872f7d2287937a59d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4b8bdaf9670d7c24f952ca38bfe6c699152421b7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4bc43084ccdb8d3bb704ca577473169cd1a64f9c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4bd697fdded7ee65154e28def586a7e9b60e7078.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4c24ee4a86de1d5207d93ce39132f7b660cf82e0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4c35605124dd8011395240fac71e80b52ddbbe7e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4c84d1153daa335f5ff6d61db665024b32ac8195.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4cc91ce442f6fd13456d4e51508605fbcf891ecd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4ce668fe6eee9f59ed5ad0dfc0e9787777acd3be.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4d1090e239baa1ea54af521f4b8bd8b5ab51fd35.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4da8a87e614373d56070ed272487451266dce919.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4e2bc97a6164191ba3d53abfa49ccd973aa80326.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4e3f4e31c370281c2d721d6113265c25d9c566c5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4e618ec5d2edea031a9ff8058a9bafafe30937be.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4e7cd4e923777d6caaa76bb2b93f6121bcd5b6a3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4eb8b7fd0032816e4a29d65b06939266d6446624.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4ed084719f40fac25989dbc99d08c2c609b33289.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4f1c22de2d2e0b3f2a12bf2a557d429d3ae31a5b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4f392ef1aa2234dda52ab8e6540f275322a20697.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4f53e43ba1bfb84611eaa839994f9297cdd65dc9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4f5901d41427c10d4dafdc6d3cef2dd20bba871a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\4f8d24c531d2c334969e09e4b5aed66dcc925c4b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\500c5ea0961b573ba5355d364ca94c00a0a26e4a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\50688ca74af7fb4395c03e3cf7ec3bddc84e6165.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\50aa3305f87a409bd1601bdec5d3cd404fe825a3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\50ed5cabf8d74c2a700184f876ad15de2239adfa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\510ecab01c66bc2a911abcf67451ec482ed18a05.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5129bb31949c66ebd544658c531004046d4102da.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\518b635f78bdc65dc6666380eafc3b7a283b3af7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\522da4df2c31909547ac81763ad28d318fdb88e6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\526c800695643e303ad5671d00693dd0f9b84eb0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\52a8089242fbe484e537cd8ea5c0c9ec3bf32150.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\530f3ba1a2c044305d73123fd8d2246f63120bc3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\532dc582abd41e833303742ca40976db63fe6fc1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\53326420d7eb607538a248c0da27d3b092f651e2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\53442eacc3f233078507fa37b78267399e8c1e3b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\537009c0181c3269f40753f75ddca4d17ae8abae.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\54122e9771f67d234bdcc67172a788bc96eec34e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\541777e512aa3907a9db91cde5207a07eabde5ef.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\541e7d329fa466fe7b961263666386dddec75894.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5470929d340ca0fbb030ce8e00a885a0d730adbe.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\54ae20515fcef8417e255b19df1a5bf6f2ab40bb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\54cfc3c68e1e4832fee5b3294e5673e37978ebdf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\54d2fd5689ac3255e8041c36b5fb333a3bd3f9b2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5502de277b6ee34f29d813cc49d7cda1168c202d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5507325ad63261970f22c625abcfa59f99e24efd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\552548a4e9a576953acad9fa549ec4d796fe52c9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5570cb9ff2905fda60cfd6e94f3c32774b765265.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5585d267f9b505b068f9f72bff5f12c248472f0a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\558d318e1655da9f5e9507ccf24fc773a0d377ff.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\55e0d20b6cc56c2cdc131e1dcc51f752575cf532.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\564f8823050b52b5f5c36638ac1ae07557963f36.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\56a81431a148344381a1da80c877885f3f3d6d59.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5796fdc80a85b6e634a279c57711356055b65880.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\57a86b3acd182c955877afd4792719a4e9a0ac32.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\57e01ad2a4961cd5cc6a3733f5f8c013a8946f3c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\57eb12d93057a4bc1a043420327251b2327c9b39.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\58bc0f2712ea1ced1f886c9fdd17b8cf014e66cb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\594d23a412649df7dbfef5347a2a0c208ed4417d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\598b8f280571ace476ad56a469e7959918fa7958.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\598d3eb737dfa7701ce8c29c86bc9f6589d8a581.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\59eab95c43fdea01481fdbf9bae45dfe28ffc693.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5a425d474d13cbcfc448f36de539d4a679f7b618.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5a5a7b6d40c4b752aabf69f69bbd7c71e2658e8e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5a843f4299b04090ff76451decfe9f0b025cfe48.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5a9164999237a46d3d6aed53bd59b17d91a9e6d0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5aa0306fa361d31085f6df03d7e71307534b34c6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5b1a2b3207e3f674b94f6f271b92c29f3a8a6754.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5b27a16123f46b5aaeda3171d22e075dcbd36ee6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5b9a15ad62b0e85fcb5012e7e08e5183b4da786b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5ba386b58c8d194e8cbe581eed95f7c64d05cd53.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5bb89950ec5a06e2b7f69b2a9c4213dda19b1ab0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5bbc428de7fc2f10c24d8439edda26d84b319687.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5c31897d01f3edc7f58a0f03fceec2373fcfdc3d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5c525cc04075b7c82e52b1e80d96bb394b1a0907.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5cf2bec1530cef38cd0eea173fb90ba1a788e11f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5cfb44ecdfb61b2898b6cf7315b1e810874a9560.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5d3612cd95331a2f0c43ef9d03ce583b5ff8c995.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5d768aca47f1452fd3e3d2c246926b8532ca1c87.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5d9e490190986584d799e2828c6fd9679e5ae159.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5daceb492bad28a61fc43c4be83a164042d8c430.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5dc2c157ddcd97f36c94faab478c38f463f78ae3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5f06ba54c9208857d92abfd16488c44217f8d726.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5f6a410a4ba086b296100a8a5a10df59eb337c90.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5f72fd41d7ba8d4d7de7aa1f83fc5c90ffa50b03.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\5fb2cce255dd56b944ae52cec664bd7f4f222c66.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\610549c879c62efc42f2ad32f63c65b25d8931c7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6115e9fbaed5e154ff92cb8d02387fec72251dab.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6186828135d9e5e7e50f66567505d02e910b3ee2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\618cd102ec5051a05ca5131e21961c964152f15c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\620d69e419e3c8756de8bf96d86f6bf0de7ed919.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\622580ea34c3acc011faa7573bc549e34078edde.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\62298f90df017016d34e978befc29acbec201ccd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\63781701d3aac3b1640caae24b2c8c2a06c8e6ab.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\63b645b7d8cd2502bedca374d0418486838bf4a0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6445688d3f59cb7e02f2a8b28450cdf118a8a373.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\64478db449af8e4a5645c17f548649a0159da0c6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6499f5c865c3db3bdfc1e969ea940a80d547a5bb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\64b327001f8fa95b83dc23259a2ad617be12498c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\656ef218f7dcea5555a59cdc29d0f9ec8d5c2a82.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\661003bddb5c847237ff4a1f34a823f7971e2090.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\671b44ae9bec89cacf7ae9e7b2a79649a9b66773.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6726b273f345838661eaa39e8adb5df4e1815a9a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\672ca9b6e50e26c49fc7be2ef3b4f685badf9db1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\67a2ae408efcb6686a22ae27435c7e4fa10c4a21.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\67ff35bb162142475e1d16e78e56dbcf59ed65bc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\681c5099b2afb08fe5cb2f58d92e9c6e4873bec2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\683416e300a4f4075835852558855b07c342bd72.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\685cdc49b5db13ec22f0eda615b026bf59cd23f4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\68e509d42a7d3349550512d80eca4e2b2123613f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\68ed7bb581b0cf32116ee46767c08e3180b05c34.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\69008e4d4e158ae9484c3821e4384aa5948c3209.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\691da9d5576aa959e22114a1ebb35548efd276a0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\691f54c001708f6684ae7837adc749fe09404e61.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\69f473aee22a4ac8df15a54a6109b70b1ae9b7e7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6a8d26d75bd44c7666c9b7ed782c13a57f409122.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6aa2716c00d89f4b760a9167b15d6203303ae6b9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6ad14ceec74dc9b3d3dc6db7fc319b9dc22dc546.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6ae20454d1a9f228864de24660c2460becbc8151.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6b039b72ca7301c604c743ce7ca3e2c4f5659068.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6bdd6867205d9b09f833f01c65f1bbfdae469164.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6c3e940e2848827af17dd23005a9dee8ac0cd349.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6c91b00faa16142426820016ad3a7847fc77c8e4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6cb02c7565f6f74a3d165e14196de5e9e87d2d04.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6cbefcf4c0fbb8466db7d00b8f53e1f37a6bf8b5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6cf87a546884756094da0d300e85a061c2cc43ea.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6d0127f985edbe088bc279865bef25a31f54a066.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6d1b40bcdde63e22931509db389c6bf1949901dd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6de59b8f8122ee30e23ef152639bb402264810f7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6ede7d40a9858d89341de780a3be9e8b43291151.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6f53b9dff58818839c45410d11b58f0f3e327d8a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6f8a531b4b84d4daaadcff9d7e03ee24274baf0b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\6fba5d79bdf83322d365179fa84ca02de2a16ebb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\70c7260fc0702bd4a554421ac68972c991313e56.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\70cc2e5152d3dc4d44494124ff556c9bbe9e6f41.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7116ddab89281e7641fb659019c90ee3715f5039.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\712f1a74a5083d690832eed5fd9ff16a04434d8d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7207df5b61e93fa5a2b6e720c91aed05270e81be.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7325363d061f0f4699e29bfa544438e14e7c207d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7344c6de7ce2258e74ce131bed72e6ad465f0ce8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\734779fad93249a2f6fede6afd10eeff7b37919b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\736b4793b11d01f4b55ea53eaf59f7325493b27b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7395de48b2402e5e1561a93b99e8aae8728c6ff1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\73998a3f565b661816d48aa7c20fa5ae617864d5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\73a954bbd21a08cf0a662b37f1f97989285969c4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\73c8af41cfdbf52c0dfba37727e3b94cb56b495e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\73d80c8f5780d70bd8d343188c56e898e91557b6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\744e302e6bbeb9fc1232e37418c2f8b5f8179989.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\74b00f19c3af87d1081644f02490ba250f57b7ca.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\74bee8bf3229f4c28991a7231c02d911d24770e9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\75216c0238ddc17dec8ef66adcc84bd48922cae4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\754315299d847600d6c5d414665c728d40bf731d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\75e68869f9b65bca661e768402395d9dede6de2c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\760481ce5a46e6efa12a80202bbd240dcbefa8fd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7607a4ad84452e29998a80f44bfd6bb2f5f68a7f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7677310b4e43cfaff6fd4ad53636fa80de589fd2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7698ec48a89a682cb52c066e293567fa3e83b7dd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\77a04c5091c0457ad305691d4d4f24354d6bae79.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7808e4cd470a379b12470833f4a6fc7e1b567c96.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7837a1b529936ac78b802e41daa73ddab8bfc41a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7852aafdfb9e59e6af78a47af796325434f8922a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\78ab35770cc98632c434523df48c6ba290182d37.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\78ab74f8133c0dfe8fb414c216350dd37f69b0bf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\78e49fdb6f0aa9924a5b510341d52b618fff0ca6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7a51b191fd6575831585eee67a0ac67f209d8fcb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7aa49aa2b76ea92b9265ad9d5a9d9ef0ea585b24.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7ab9f9fcea519ebce527c3ede8091beedbb26ad9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7b40dfdf9a60cfe4a199cb967ee458b7dfaabe0c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7b5be48b7769ccad2f44a8c54efcea5281a6c633.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7b89a657dde3e5888c31d1d5ef3b90ceb0f66a62.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7bc87a77a7aba144248cfa3bc514541f09047edf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7bdceb3b947ec934199d0c97fb9ec5a0e2bac776.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7c17c6f5e434f1d8227c9d8c5f9da26200364491.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7c8d7d72fcc79fa72a8c2981ef66e09a03b5cf7d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7cdd50888f639629380fa9906d73f97010158127.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7d3dcbbc2b0741fc544693d7b3fc185f0bd50eac.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7d91bba210d98289dac830135c56c13e206837ec.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7d98a73314cd9f923aaa367f3f4de056b8cedec5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7e6bd4f9d794f87445f67c9ed9d5699706040fa4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7e80848c7b22e870b0e0a458dce96f6955576f04.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7e9cd4bbf0fba1cc0bcded40041fd01b9dfb683f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7ea2f1ca7b3c2370d413ae990eb71b2b990c91fe.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7ecc2bca3cfb5ebea94cd2601d6d3db60603452f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7ed2c0786eef3d39fa0435b461aa34ab454a4753.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7f8c673d136b9bc0aeda55e49cc72aacaa5688de.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7fb7c0c48b30e66dfaea5fdec16d999ed52d1ee0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\7fd337f16780aba8f1e599ce7516dc9b1d80a546.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\80df1ce37238d763d313bcd0e673716c7c4458c1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\815745bf1b522d33fd7371cc9a6561a2a93ef87e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\815f653470c57a95b92e3857d8f5d038b4000d99.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\81f08d610ed4c61197c816bdbe5f23b14670dd47.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\82c1bf108f88d4ec2b4e68106d9531f1c8b45a5c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\83323659de2f2d5ed3ca6ada7cf137a8a17bb014.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8349823092836fe397a59e38615d1491423dbe70.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\836190d366ba8e34c3a5df8a24d17a36fd503505.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8367c21d603a58c206ae740debdb10099f84f198.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\837c77d566651f559ed3f78c2e490213f99212e6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\83cca3779f8aeea47c5426e1d0fa65f1e9567e0c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\84c3cb4cff4551bf95b6f79bd6c52525faee0a44.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\86b6b0c1b2777541feb83116bcb7a5cb12a52310.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\874eaaeda2e803101ab952b3d9bc73426111b3d4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\875b7c463f00772fa0dc18ada678bc1ff16a4274.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8762adc9311e5803d1de438493772269acfe7fa3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\87b89b038b28d2ba0bcfd2203845bf15f922c1b5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\87bfa13221ccf5a02d4a888ef1320b57f2c2d5e2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\88e3b43319399c339186c02c44d369228c35eef4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\896cb03362facd2a670bee13dc5368b81768b2ba.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\898880683d9a962428033fd7500f74647c4a36f0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8a1fde8c65e439496ac5810504de23ef77312f28.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8aa021696a6accab9053f1d779eef6710439bfa5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8b2e50eb3ea84225580fbdbccccb4fcd3f062feb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8b71ce92ec03e066eca7792bfd17e2f210d50ce3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8b88901d0bb033af3a18ebcdf7f2aa7110b21504.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8b88d01563538e4abd29c3ba493c534af30664cc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8bb82a543720a74015bce24945dc0955b666ae7d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8bd8158e4a47c65b12a462b49b4b8b4af1488dc3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8c223fd63ba6a924ecd3df8d32bcf0556758cb95.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8c900409a679dc1f16ef13c73500cbb53605b683.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8cfb8dfc2f8b2c6e1d60ff48da02de6670694869.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8d6d3be5f9353eb3358b44e646c83f1a90861b4d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8e830dc6cb2de091920ff5b3b14f806a61a8aef4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8eb99586e9103599ab2f58cc33ca94cd0b60ae53.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8eef6fec694446e150e95dfbef3a3579ae8806d3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\8f4b98da50277e6c9516aef0227f5cca9af6b5e8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\905603e9db862f9b4b42a95f51e9c8d0015d1dff.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\906b616191eea5e5e724d47546f7d2d9d096ed7a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\907da65e13cde5d7ad71f304967a7ab9dcb5efa1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\90b5ecf991032f3918ad43b252e17d1171b4ea63.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\90e28ff462882ca7a9329beb879f73c2e99430e4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\91098f6fe46a21565bf0cb06fe960cdb2c3f5e38.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\91233a0c35364955d1fdbf2f980f7220f9bafc70.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9211b1703c76e20afdee8fcb9f7996edadef0e9e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\92413565e9d507fdf5550d5123ef420948ae993c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9261e7a9a4d3febda29d574074a79f02376249a3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\92860a0f4425d12aafb8e096d8e65d38eb67dff5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\92c020779e68e82d9a5f5898435e618caa63a099.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\93d8f1552a81587e2f88e38580cbe4753b9934f1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\93f67ffe7803061de9b19c4dfa346b3aa97aa4eb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\949feecec7c10e71532e860f7f1230bd696e8234.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\94df380f8a3bab7e30d39cc9d7e95d1f302f5f9f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9562199d3eeddcd5c1944831ea55a2a468a1bc7a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\95754ec84bdee0637765a5ee34983a5f6cc86eed.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\957e71389292e0881dff2bbe0e87fe6bf8b2d381.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\96ef1767754a53f792951ba1752440ae94e90c60.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9701a8c529cd8c18124da4cd61c1165a64b50281.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\97027e08cb413795cf19064338efd049fe5d491b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\97339c9d365437d54ea33be1a1b634d35f460208.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\975349ac1c79fc64525ee17e7d088c88c2c2071b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\97a34a062a7ad09f7633d4a0e76d013b28b9b582.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\97e0830fe6c8bf6facb896320dd1b8d31ab49a54.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\97e0efc17b5a10c75f7c83b08423d27585a31df5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\982689f92bf67e90a43fc142c605da9a5045f756.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\982a6fe2fdba30430b547741811ff4411f1485f1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9867fe1531c1541ca2b25a82d7adb62d575798be.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9896bc65559e6406d0d3cb35e9b01b953d61959d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\99a46ba28f5811c112876ec77f570d6c10778db6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\99dab788e984601ee4efcd1138926caf10c00dcf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9a077c5ff1dcbb9104feeaa832e73ccdcf5fdcc9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9a3de625119a83aa1401eeb297937d9605f0d1f1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9ae476404f7ef1ec1ede965f0b898f31a5bf5a81.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9aef9e1c22896809852ec6c94e413aff33b3d9cf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9af0b8e13e1e5ac714135dca8ba6d5216966066b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9b17a9fa6be11d8b3dbed9f3dea2516020083bd3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9b4445849937393a4b05378653521a9d0c34dc8e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9b5f5119bbfbded3245acc37859cefde967458e7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9bbfd3d34ee18ea1b9f4669331a6cee9c5992893.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9bf1677e5f822110e736076a4ba6960a80f2f348.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9c32d461dc9d4737756a990cf13bae1a03e078a9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9c33486a49de4aea64ce61c0a2c21a88c316b6a8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9ca81dfa68fcbff13166d810aca929027bb3763b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9cfdce851951c817101b36f7b292e744906474c4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9da357734a3bfc5b51dc2c63b6b4ab28930767ab.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9daed5e675ac113d1939056dcebbb1f32ec0d7ec.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9e4b67ef6613c31882bb561d28770203da19771b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9e780780dbaf35c0d537ddb290dfd484148a3c55.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9e94f9379fd74fcacc4f3a57e03cbe9035efee8e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\9fb5ca87ca47d35d87161158ae37fc0ca642c51e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a01672035e0b9d21f9a934b9c9071610513325f2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a05179c57d5e1cdddc839ff3c0b70a614efbb6cb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a09c1ad0f05bbcc256cd667ad7411e6690134370.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a0ecc9b46addbc1faab491434b5331702f7883b8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a1bff76ce360e8990b0a4ee2a5228a6e6e63d9c1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a22ceac8d5e1cfaf2e79557d3c794458d0e1cdbe.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a2a6e262098539eb875a26800d9f6d3d0d5d1875.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a2ab0a998912ed2ec93187333dca03aa22e39a71.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a2ba0a34df31fd8059ed22787331a75b65eafa74.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a3116a1ea2e157c8855023d968cc0f0bd6c9846f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a33f451d8e2caff57c1133f14198454a64993c47.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a34d2c1876ffedc4f95f3af13ac06039cd38a5b0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a36bfdd1c9a1666401269fb0c08a3c2922bd6812.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a388cca261e8af7936cf3b79c35f43442f65706b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a3dcd6b3f0551e607d0235616109ab8d506f81a2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a42ff48d50aa3a0ebfe840d46ed2204c49955442.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a44334e676e43c1889d282b0f0a8365f0f1e0c52.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a4d1861ceb0adf8dbdb8aed9465a3487fded5568.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a5039336577ac0fed15b784adb912df2a6a143f3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a555fbeeffc44dfdb5087c83395f0f743654cbba.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a5944e859b017fd4504ea042daaec613f3fba167.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a59906b732bf4a489e282c3e4f499d4166c622e7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a59a1e31a7d2edf1be120c4a13a454bed64b1593.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a625e5d6fc8956c231c500a40e887e248a69de61.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a6af28a7f0a0d8d1e4d5b06f5ae38412298bbcb4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a6b05fa393f9404c906099e179ca476294068394.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a6cba8f9a9f89a08bb87e30a209389c9f1ca8c3d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a6d2875c8b70a41ea815511417546a0689816964.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a6e5c7919fa1f35244c27ef7e31f6b5817211b66.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a726e1844e86c7c17b8b4461dd4c29ad3094c792.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a7804d77d1a75810af0148d8fea8acbe2941f1b9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a7a4a5568e5b7c921bc616eedf532a393229decb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a7ac40c0f2083c733cc714e563b21c2d606818c0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a83b9f98ccbd6dd76d47c49cfe5785f9c2ecc09e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a8852f0f854746663fd96d09629d8b0db0d55a4a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a891d059692c951d1fb0fb36914fe451f973684a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a8be6ba53508e988c2c8c153478ff615b0d77687.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a8e53c77bc72f467723a7d5b8020875c0e96f8aa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a9a978c056422f979395ce56c96738d33b104a3e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\a9c58042555c0d56b8e94bc5558db0c3dd697dcc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\aa507fcd3876f8c422993c8dd13847b1b8e77471.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\aa5475eb5aa3a1091fa01df00aca2f3def543f70.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\aa5a99e67d3e887b3fa88575fb0d4a662d0a60ce.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\aacc3acdd6a928fd820d51a2d1ea98c164b6abd4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\aad76905ce54679c80b75e4ee35717c30e7e1099.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ab1692d56f32fdb0922e5f323f6ca385fb1e0808.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\abcfffafab399149d4adadd6bb458c4994e2025d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ac35eeb6eaaf382fe0e123719f7dc409046e5d87.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ac3e33b6ed12e43208d76bf455f3b81067e6ac6d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ac4a3bbe27272cd6780e7168906b0bf1413fc749.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ac81102667b0d56edeb8ab0044765dc49a19f374.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\acb0e11e5763dbf7588ec435751a2ab705a783f1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\acd5cb2ba08da5c0f6bc310d74d497d38aa86be2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\acd8d28ae75dc1ec5fea2d767e3158545011c1f7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ad146e228bda4e5a3d56b28a3a1acbd01e0b323e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\adb09e4e5c7331b2aa661b3d3bb0a643e00d11bc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\adb3a6501ad731eda95d0a1a182a793a0dcd58b0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ae4f1ac93bb55da1231f08eb1654f19352ab1a3f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ae61300f4e7a12a68baf114f199eb51940c7aad3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ae66f22f886d3b019934ae237c11a5fb53dbefb2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ae6fc64042b050df93ebb8f8045892952f18510f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\aef0734eb2538d3624d650a4ae20c3f66cace5f2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\af266fac8970a7960e96630a67d91bec5dda0335.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b0746b57a6c8f3cd9e52a27b0d708d069134b502.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b0899a264af548a89d649154aec569889717b295.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b117016a887f05386acc4387dab5bf8c769cf90e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b1a5096c2bfbb4ab9826c36e3961929a744134ca.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b1b9d81726e56c2781ce6b63ae5015cf2f314c6f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b1ce6ca5960cb799403188ca303a364428f812d8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b1e7752cbf528db99723a3a67b49b9b5c6e449af.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b3cb1d91dcd7f78dd9bbc88b93d2403eb6869746.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b4d9a00412c65725de441654454803bb20ecb931.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b4edd81e0d2770b0d6e0e3c93c7fffc43e3caa80.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b4fc44eabb1f640124fff98990c85dc5c9984aee.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b5161b031c7f720562e94735a018d1c3c8be3ae5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b51d1c7287ac75790a52752489577d33d522898b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b53dd68cad40cbcd698a848f53853e5f03d44a1f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b579c3547bba2a33057373d57b7c05f37dd4cfc3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b5b029e65b963ae092dd88b21911e74b0cf557cc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b5be10942198c0a79dff8aea2253b77ae2401882.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b5e1d36298a78690140d5ccb9673dbd48a0ddcd3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b5f172f9c5db90d48f5f4f281bab76e1401f1a36.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b696d208705fcb1925693c5f0d118733bb557ea6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b6d2052bb9aef4f3b1d21e86ca164cd43d020857.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b6e92eda6c26b456165422d4e5552b81754c4a82.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b705170c7527f5bc6842d90bf443fa9f9ee00bf9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b7f94d0b929301cc7a5d8f68a74d4f0270c74649.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b801b7f92cff2155d98f0e3404229c67b60e2f9f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b80d9de7750d627a0189fcd2dfd82c7efcb1c079.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b8bb4db131a25b1bbb30d4205b16dc9ef988d22f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b9548530d954b407b35fb266e2b4192402c982a1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b970b7a807f37678f91d4b989ffdb7d7606d754d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\b9b0cad2db02479624a3d087626811e4d2fbcdb9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ba1c4c0de19352d0b9ed7db6199803b621f0a8f2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ba3efcd6b74e55327fd7db470d824fc18943f30e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ba92c589f077b55e9cd13264d619c2ffe9d3ce7a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\baabfb35a321ea12028160e0d2c1552a2fda2dd5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bab88fad90f0f0448f75af01145e2f1b90412527.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bad4192bbc6d41c066bc947eaeeafc53024c98be.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bb53a02a8468fa9b33b082328580378680aa0838.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bb6317ba4c887c1e5d5409d3727c908234f307ea.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bb8e3d331bb2975e1c644c8729b842797da2d626.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bb97f90ff5bfd41f2f1148c5cbdb257fdba535df.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bbba5ff9ab9eccef99a0bbc6ea557a373158bbcd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bca15ad98e1d08c6a62c9c0486de84bc15a9f661.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bcef0894cf382a83c2aa4d9c803978780c2809db.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bd872a86902d848bd94694469bb0c5cb05bea483.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bd8f57ea4aadeda075d557f566a1909d68e658c4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bf04c9ddc80690a5e2d3c74d79a797319e36e5b6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bf39a81c9aab9e0569f33d93d5e203d320dac024.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bfb0852ef4dd124ba671303aee7a71a08c7c21a7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\bfed3ce6a6df2d442d96d0cab07d4bc1f2347255.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c04c7fb330a409a00f67040dde0f83b3da88eacb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c04e10858012bdd666d50c0d69c3d1e7224ccbea.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c08c3b98a956287a6eaf99f6d90969d7dd1410d1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c10661c9b35e068691b879661bcebeba3bd6aad9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c124701d09545e0117cca7a3a5f449584223899f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c12ad8b411c24fe215be91d1f990645743e55ab5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c14d50924f959d38ff7857e6db3a60c929448956.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c15f8c23eedbedf8d48bbc88b8225ae9c9d234bf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c1a08ce8f2c5d5311b19822c3df5aa9d25c4d93c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c1ae608c7ffb926a0f50a6a34c0780983274ea74.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c2298fc67d299e45f2458eb2b2c4cd37b7a1ef64.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c2510fbc213aa74c6c3195cbc8033c7d0f11b248.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c2a84d150958eb9c206a388660a12e4f5fa6fa1c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c2af4db8591a8ef84229eb08359c0a50efa24544.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c3e5af129cd3332c6a01776cbb9534bceb46e21a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c41fdb2efd6d61384a92a84cbba3f8233629a41b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c42a617a00afe6a36bde0a8e3638e0f55bfee4f7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c4ce14ce42fa4360dfe3515ec9d1584847381c27.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c4dc1c0360d3fe9ceddc4650aea022f24d88cd99.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c5bf5beb177c16aa29255e89e0cf47e4e9467cce.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c6012a6141f5ad05ac983aa9f1a7c57938363b16.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c6039f8933305c9f44a44c81a15b321b6c2848dc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c672f5b0d7c0a40f7cc1bdf079066d72eb8f64ec.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c6ec1380338cb2467dc6fc14add09ca5bea18411.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c72595eda8a721e9bff19fcc18ecbb8510054f37.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c8437a45bfb84fb206fe03fd18d28858bae32651.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c8559b6ddd6213bc91524397933bce33aea7d50f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c85f571a674c7fed0ccb9176e9cf9f3d3659ca32.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c8b6f2752a842dd9eb9c50a82112748ef10ba259.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c8c052d27aaf8015316dcd2644fa5e0b3870cea1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c8d206a4f9af0709b6e9ee90c4d854d482cb0784.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c8df44a3612e85e267351e936ddeb8fc5867afa1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c8e940f25e802de4fb2abd917b76b8dffc5a23ae.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c91b581d38243c502118d93fc6f71ee02ec395e4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c952f4000c5163f168b943fcee51e0634066d9fc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c9533866dd77deaa65ee1015b57bca5486f93326.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\c954675ee859e2f7b8f352a398a67469b50f05de.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ca88735399ff43d0e673876200655099f06f5567.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\cb346b64e20934bc01f7727cf51494b22f172de0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\cb3974f33a1d54369d630e62052d603148bb12e7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\cbc05d14c57b91081970a232ab83bc993f998fe2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\cc41c51293ed24a57ddf467e462ec9399463ee33.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\cc5e36a8d22f6b1708b1c43764806b1722aa1754.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\cdfc8be07677f50c91106e5b56379acb0d0eb289.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ce16fdcdc9f5932ac2e66646317a33f142ee1eb6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ce97f5b8be9e2d7eeb169762fc07a08bacb3e8cf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d010cc0209bc5a0eee6259f72c42590c448da1cf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d01e580a45c457c7c200cd884e746b4d836e2ea9.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d069dfb7f0aefcdc2c890a1bbe773ebd26b01a55.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d070dc3ef2125fd891f3e023598831e23a145880.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d0b33d173f1d32662e2ad8d7e85181f9b23f010c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d10525ee12ac8eb0941e0b21d04b3552baaeb79d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d12b31daa935e9043a9f526a2547b6216ada7cde.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d14f165645f21a60d224c6286c5e51799797eceb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d1696944ff45daaa2f305405c54ba45b16f3c86c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d1b26388670660f66249b55ece2beccee4c18d1a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d1c77a93f69b5dbcf3806674bff9fe4677209ae5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d23c6a066a58dbe5ec5fa57e67f1b795337299f7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d2609c4d8837558f4b8f32170422ce1cd12425a6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d2616cd5feadf2f63508144580ce2b9e172decf5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d29c253b473969c4d92ea9911d3badb9a7ee63f2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d2c7ac58cc309df9d3da8e766184d09c32d16dca.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d34a47b592e9c6e84b5b4115474349acbc0fb925.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d38c890fa2f9a9325e663e55a43cfbef262e86d8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d3c2e2839498c613ee95739dce7052109750362c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d3de3a027f59c0acc13afa851e270c7fef849db5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d418cdee18b07a0adfb553d43b4a08f7736fa9f3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d47377d0da60ea7aab2522c204ad65753b0545fe.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d485fce0c7ad92a1e1606eb81043bf9e12714562.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d49f8ba7212183bc1b7f2cb6dd709dd197303b3d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d4cfb1fc4fc53abbc1d4b0ca60276c6af6632c3c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d4dde806ec1cfa2209b422f67ed42bcaf6bd7201.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d4e1b8705646c89433773a9f67bdb8c12fcf4cef.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d5a1fed3fc7baf4e31c897ec639995106e9fae78.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d5efae48a9bbfccac8d9e6c1a80c8c605d80ee21.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d5f14e63b86239f491c9dd1749cff01dc4d633cb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d5f7332f6dba1d990d54b5887a175e530d144130.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d618a596def847bdfc19991c694e09bdeccf2287.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d7111b44f2d13b24a11d55c8be2d2bbe6b245eac.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d75c48f51b2a5c26a41fe615339e5947d5e9cb30.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d7701711485f822c0f43df58b48faf6a433028c4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d810f78f8850d9386abc6fdd8d652f2f85191753.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d82fe700418fb6494ffb9a2c0cf4f6b8012b3824.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d83e3c028de1950c4f8dedae21eb90f90c4ed6c3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d85d142716f7f85cd27b54fc38a9e2968cf14346.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d8c2466863d8ff87bc1aaf5218ce6f3f7fb29c3b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d91120d80916351170b34715ea4ab141f02c51fb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d92e6ddc127c4dd588cfa8fd584a9a1adb4bbe86.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\d9eeeb81d17be6a722b11081f13576de197d249f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\da3aa20131ac2805c0d9e1b29f094683479ab5b7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\da5ecffba8c5d1b7afee057ff61149fb976501ce.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dac1b1607ae72b9509ab26367e0d55016e8132a8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\daf32e013d325a6feb80e83d15aabc64a48fae33.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\daf507bcd9337e05d4bdc3da73e05639837f2031.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\db142fe07544dcf0e454f8a9f39452f0aa92a6ca.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dbefc8ad2a3de5d1696b7e604de8bce1da2ea8cd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dc3d8b94ae9acb1ec585507d0d77164a7f688d10.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dca8ced82157924ed86c698a7dd482be81b4b266.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dcd7a1235ea74e3ef71d051103bf8a64c3c8f457.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dd2cd004a197872cfce1c31a9685d6972f838efa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ddc40ffc47069f3714c8da65843f36f41ed0f8bb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ddefea974fdb2730133e762ece41335b3d3cb94d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\de055f09fef2776bc78bac5d58c4131301b2025f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\de3ae7971891058db2f4b36cd216531e6f533332.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\df6013a1630aa41627cc84c8e780d40ff93785cc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dfb0fedbeed56bd2b795a67faab28295afc14c96.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dfb2f07149617b0c2911899ac61563e808e2cee1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\dff716159738acd6bd53b4be12af45358a58752e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e01948b342544a5fcd8c14d757501779029c6d6d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e046a06f262d74d025272ad697d358c4afb5d9bd.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e05c39ce913edc81f93c9247f9714d6cddd30073.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e05ece9c6fdb3e9942598685bb94748b97d88e16.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e066fdbb4a9338a57477424cb66fd4d75b2fd66d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e08d308e5923cb4f741d34a6cd3548a17a04a2b1.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e0a53c258ee0483666329ccab591c0753ff49531.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e0fd364797eb02a280e25e2649dfe6d6c0d33e05.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e1632ff25e6c30d4d89828154be1389a90109db8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e172b0d49bd083dca5cd1021900477dd97774b84.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e18620d5cff022d18aa23da2fa4634331fa54860.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e226beab8af4202a5182f7603fe468dedd5dee6a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e29b287d326be9c2c0a7dcd7a81cb49cc2125360.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e350e1b4aab99c73d0e0d03121b50c059c04fd61.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e3f19f88e7e3d1279b4d93e4db041e44a8d35dbb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e4701bdf5d414ea8ca38a4d8d3f69540805293b0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e476334b4f415f8fde5e2f560fd1ad0fa314af90.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e49c9e5b4f77c5cdbf6fa77343d767d909548dea.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e4b62ae61f0135ded00dcefd923c75ba78630ced.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e4cb4af2f4ec6f023096afcf7b3d4410584e1393.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e4d752fd72eb7b1089e4b820598b1f69f50bc636.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e4e5e67fd4f75548dffab7f75484bc14fa15c0c2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e4f66cfdaf836895e7d37941223f11ebc41dbb57.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e5163021a1b88e2c2335cca27fbfcf883f870830.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e560eb28bb3f35099d2632f80adaa14436516474.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e5f19b6daf956e815c779228cc0cad1293d65bbb.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e60640bfc445db0f53b5d92cb9f69cd943f1b73a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e62e453735dc2022f4d81aa9b22769de30d0bb90.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e6fe63419fe399f9c77513c1c11a043db006a2f3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e7bdd52badd2d2ca8ce7bd838ca42d02ca3396ee.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e7c3ec3dcb3469ee4d608029e7aa3068a4c90c54.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e7ff9455c483c237c3ee08b9d230b7be65c064bf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e8b2e667f2145ec96fbb6400fb41b05cafa26e37.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e8b8e0f027881d2480d808b05d9281a7ff4e9b4d.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e901cf35efbad09a224f2c500f7aef1f6d2d3b6a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\e93ba7d8a047795d5ec114741f32f5b18e8567c7.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ea17aad7692bb5040f3b2ba1d01b8249e4969469.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ea7cf2347337e379e62c9bf09b2db9b44fb2e812.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ea84759e67802b696a5c60503c024546e542d572.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ea94779ce166bcbf3813ccaab028524ff397089b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eae11a3f01b79e76a968bfb84886432f04584596.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eaf0b485f290fa884dd18de71be79d86de20eb31.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eb5e5088c41f976387784acd57dbb683932c6002.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eb7fc5004191e8fce0d6e4ff98de6865872e665f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eb8ac60527db35b10881cb4fd86b8a6e21983d02.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ebb553a859d31ebcfc57d489ca84af0df894dab8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ebed882da10cbf669cbb86802e9fa07a4a33ef91.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ec223563f366aa8ed21ec84d1b0d21a911362cfa.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ec6adb9dc121bcbb8215cb16b880d6dc6497d209.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eca1cf50bd67a7efbcaf78864f5f5b3b850243d6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eccef80cfbe078235df22398f195d5db462d8000.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ed0a318f34fe2d65cce6127be009ad7d5f3c7cae.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ee3aa97a0ad91231296bd0643136c83bb97af88a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ee7f56d556c2063d71905df34a589bbf5f1ff051.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ee986ee9617cddea4f905898fc76512afd10ae1f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eeae21c913759e8cf4c345e561457fe7ef84480f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\eec5b4ce2de8d9d00be664e6af79cac476991bbf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f056da9c64fbf00a4645ae326e8a4339d015d155.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f08249cbb9bdfc3b7f103470537a52190c81b974.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f0cd43222c9a51d619357babc4d0ac984e8e45a0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f0e75c4697317cbd0e3a5cdd722fe0526595db64.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f0f3f4ab41fdd2d12c3fa45dc53a17d7c58a4068.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f122e125c2cd3580b1a38e27c780b99f671fd78f.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f126b7f9c8ba02a0f872b51cf95d08086e0bd4ac.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f14434d7d3b68f4fe7f68adba747b7700ccdd6ac.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f162829ffeba4cedd16aaf7f09902182c5cf903c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f17c6ac4986aecba2dc22f67b32f2d16b369117b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f19e2df932f90f387fe71533f3802875d9042160.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f24d3b4b4af138be06b7452b7acefc8948bc1056.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f2f114ba9f0b618a8b2396ab87f2d861ba9fd483.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f32841f22efe8c2bccd0d04a0c0b47e6ecf86b61.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f339a22cebc1bc0099beb7aea41be09cc916e5dc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f33c6d94b0efaa198f8f3f20e644625fa3fe10d2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f3ff1ecae96700f41b83d2a034a3a959428388b0.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f40eaec892778166e4f8f8eeb576893e82a591be.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f47af9fe364fd9bac5e061142208d801667c5aca.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f49a9b36227a498495c9788a4e5e4d6429b71eab.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f4d76fb8161f4cee6987d7d412798e1276348858.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f4f4b045dce8ff5ff1f5aaa48a6bdbaaa7b05179.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f4fecacaff35a0498dee4231408c6b9aa049c51b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f53e10f9cd4b97134ded58b3de4bcc36a55304d3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f5d9ab798adf8face1655d476644ebb75fbfef18.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f628a6163a739a85fd972c00b7d274823c41bb66.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f6312e7233ad4c0780492db3c05a61abe4fb6b22.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f634f2b73197fdda1d444dcdd6bd84a96ea148ad.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f6f1ec12291f743de1504003929ea654217f2af6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f734d47a423cbe54ec0cc9b2dee39470cf74fd9b.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f7891683eecd12b3908862fc7ef1d341ca7a293c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f7b069911c90d9deab85becbad3e9a633e0bf57c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f810de29894bfa40d93bc6dde3efc22c9d0d3f4c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f82372b109168b411344f537210c63cc6ed323ed.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f8360b0e64d444215d21a61e056b070f12417e53.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f854d7b750541d65c48df953a65028a50ce57881.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f863247dc84916a96448088399badefd09d54fb8.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f879c69d61f14d7efc51023258d1361bc681d403.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f87eec8279bc79900914c5689bb42f49f41aace5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f89618a7949b8ff9b92c87ca5b839a3107957b5c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f8e8a9c6f5662c1739be031e6134284ef08ea8b2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f8eee21acf147a7088fa376af6c3d85e606b47c4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f905f78b32f63c6d14a79984dfb33f1b358b8ab4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f93daa3d8d53761cfcf4c9fb8ff03dace02a858c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f9a5880da2dbac60b06c520c9af75985d5d6c24c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f9b108c052ded463773a9838a8157ac7d5ba6d1a.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\f9c6a656d0ce352cc1fa0df8d83e6d4a1f53d7c2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fa15a1608185e44b0bf4e66354533e1a05316535.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fa6c14304fc37470639d55f97f7903035d768bd2.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fb7371ad31cd501854660876a282060402082537.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fbc41a8e025cd6eb2d7c156aad3b6923af2349c4.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fbebe4b66073c44cface2e842754bce26e3e2913.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fc15f433048e4d9464964930c36a1e7057ba1088.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fc7a6b5d1852c5ecce2d20fd0d73d5f957ed7055.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fcb76f0907f67a850fd4560218034f214fa28bcc.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fccf6791aac676cf26c0be7745e217f9b8d6336c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fd14ed7c073b7ff03afa517e9c0fd1e849878252.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fd3fc2c49f5cc27e4262261d0c9045911d65cb6e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fd6163a04c4ac0100a7c9be19def78d2171eac47.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fd847a97e6fce134345b25ae9240c4ebb680cca5.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fd8d54106cd0cc2d09b1fda70f668736b5d0bd5c.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fd96fc1859201e13e35526c188e022fa6e335327.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fda076d8d8b2fb52990e3db1982e4af2ff0fb08e.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fda16c1ee2fbae3368f64a138aa27d972a83e020.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fe09ebcf10de441c0ad0a90f0926ec0aed5a0577.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fe66306680e5be9820e2cc3742b663a782eb1b31.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fe685aa676e739bd52ba2585a7e5b27c55e2d0d6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\feb8807e8418fdd6c4b8816ea761c9a68767a407.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\fec0c97b9bdc012b76f082b8bcd6ba0efbb4f5c3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ff067164497bcfbd9145be223dcd2b05f159dd63.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ff54e3e961a72eb1d2500166809b3651b2f98cf6.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ff6bd29c8ecd325c4edddb2b050d5f4919db6112.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ff6c0b549e3f20ec78845f981f19687eece4addf.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ffbd7555a337706238c211197b221795e4e35146.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ffbf8ea9948d73572fd052a74afa01b19e6758a3.json',\n",
       " 'biorxiv_medrxiv\\\\biorxiv_medrxiv\\\\ffc9bc4ac6548cb1c03c2c6f088ab1c1c2067c6f.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\000b7d1517ceebb34e1e3e817695b6de03e2fa78.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00142f93c18b07350be89e96372d240372437ed9.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0022796bb2112abd2e6423ba2d57751db06049fb.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00326efcca0852dc6e39dc6b7786267e1bc4f194.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00352a58c8766861effed18a4b079d1683fec2ec.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0043d044273b8eb1585d3a66061e9b4e03edc062.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0049ba8861864506e1e8559e7815f4de8b03dbed.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00623bf2715e25d3acacb3f210d6888ed840e3cb.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0072159e1ebecc889e9bcabb58bb45c47e18a403.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\007618ad76a3548195ab5d11c1e2459931c91cd1.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\007bf75961da42a7e0cc8e2855e5c208a5ec65c1.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0080d3bd9fb92e022c27715c2d1249042aa998b8.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0089aa4b17549b9774f13a9e2e12a84fc827d60b.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\008c1ceaeffe7abc87b031af39fae2632fa72897.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\008d980cbcc283a9b707de3d9a02573dde8528ac.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\009002e8a66b8c1df088cf04069629fd76b13bb9.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0093f9ae0861afc0d29fff935ae6a3af898cea00.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0094b25e2500306fadbdfb41d520f2970bb086d3.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00951716e01c8e0cc341770389fc38d1b5455210.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\009892e02bc1a4c9abf6f547b979e68ecbde8087.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00acd3fd31ed0cde8df286697caefc5298e54df1.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00ae0041374cbbf28df0d2cbeb08c1396a4f7878.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00af80743cef9bd8c04c532d74e9c67f0c9312e4.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00b88130d2a7c8489e209742494303b6731d7544.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00c19aff3efc6e6430f77d81580ac927990ad5a2.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00c88aab2bfcca0db253c972382e5eee245d071e.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00cb1a95986171a256f5ef14701bd8f571221a28.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00ccc4ef2b6c6f238836b7feb5123bd6a822cc08.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00cf719db65d07c0c7bdea9defaea448fc5f5786.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00dce8da5fcee849b30c9d7a7e358c5855b14ca9.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00f1ef126735474ccaf3a8d1cce9a4823d7f1228.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00f7d667e3362066c08f7d6594d1d3b0cb16e08e.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\00fcc6277c19f6b180232d7a6fe0b0abfafb21f7.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\010ad052b58172981e089bc73b9ad16a732960f2.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01110a24b6ce133ae80d21a36c7994e364ba296c.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0113b9a8d82039e8474e887038d643853b5fdfeb.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0116dea9b6d4c58748d1a38b69b4fe6e06868aa4.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\011b8a7002da6c0fae7ca76c127567a6462daf5d.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01234cee305e37a78cedea8148bf6177d47fe66b.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01285aff189f375f52b762256d8f419ccad3077c.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\012debf5a240a496518af146ddfc16c958339c2b.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0132b8c2721692d9f53bd33a738fa94618bb4e91.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\013ff749a58f92db7e227f7acbe47d050175851c.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\014336e4855d194c1833ba91c8ae3b302cbeafe2.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\014d7df59dc6c5ed456bf19b76b394e920d3a9dc.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\015729be3f439e9ef2033276a4ef14df8873dda9.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\015a5be4701f244a51d6c244c2ba5b69ac68dd9d.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\015c335c8cb9e5eabb352e81c0f38bf6f781b202.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0162b627476379a84d5a92480a881efdd3063f66.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01669153236899edec9fff5fc4fb19a742c3dfef.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0180abece22ace6548f4601591c98059808d0509.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0181010a40e179c9b66d8b1f0eaac4fbf0ab6341.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\018269476cd191365d6b8bed046078aea07c8c01.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01880ef487dfed3681dd6d296a0a82417b8f38a0.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0188acc1eba8c0a3adef55aeada7de552cf855be.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\018b618ea132d47ffb43b003a6c78cb9eeadc017.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01a462bf8cf2cd973d1651b991d27c9556d2ae3c.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01b5ae0cce289ca9cdcebc4079f960e833cfbd79.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01be6aef7ae6555eef2062bddb57f8643d1244e9.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01bf19ea2cb2baab425377eaae3286190e9bf975.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01c25f9838ef8e005bf720c4f76e67a9f1038ff0.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01c6c44650ad5466cdb134a28d0b14700fa3906d.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01c6ed4b3dee404e4d908af40ab93183b573e9ec.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01cadfe2bb23b6989e57fce2ba744691557faa93.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01cff21366d475b5bc294c90da89a521b510234a.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\01e22956c79df3220a67bce71cf22de95d29b723.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02049ffbdb20997055c4def3516af0e6e40dd427.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\021144f6c1945b9bff9ea9e4b04b24dcf53330b4.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\021af77daa9633d212e816cdbaf9f89240bfa3a0.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\021fba0f252408868fa29b65686dedb3b3d6bf02.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02276a07ed2aa49afe07d97e7d7286b2b6fea532.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\022efb5d64e88419546d697b6a5a96e6bc8a9fab.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0234207cba8165f1e1852b92ae0fa0d9e2f870fb.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\023858d3191988a37587ab951db561a3db3ae2cd.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\023aa7bdf9b73b7453051e8fd33f9e80df951001.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\023af2216525c16fa52a75b99cc91ebb03e93107.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\023b805a96c852619186f55af4bbbdd5f9fa8156.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\023dff96542ca1e77e3a1d08a2f3893ca1ebc057.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0241ada1b12eaacadc56a59586bc1728a580583d.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02480623218229a12cdbc4800defc262f7cc175c.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\024d0bef5417a110fb52607e85ec80eb4d7d92a3.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\025509677d6507aaf63c46fc8efbe60b6d63a820.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0257dffb6a2cc8bdf6d252c400a38419ef256505.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0257e6797e12beb91a4a4fcd7130f90d84260718.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0263ef539a1344a642b1f4ff11aee6bc6ca84a34.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\026447f6e8f25d774926c5f38f8c8ae1f62aa54d.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02652961663ca435c195fb0ed3e43642e04cfab3.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0266337807a2aab25e7dc43967cba01860fe5d86.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\026c59766e222f22b416a6525ae3194a140b5faa.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02753048c4dc699d155288935b9dc6517a79f279.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02766b20eeb39ff051a2f110def2e4bf8b9b010b.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02783b0989ab6a54fd4af139d787b69b9d9d009e.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\027f86c44365e0ba6a6c26e24134c4c3e98a72ef.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\028382edbe67a35f1e099fa0607fa9a4922a45d1.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02843e9be924677d6541e0221a26501ce2111aaa.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0293ec2b1cf8097392297cffd06854d2f37348f4.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02974b9466bf593193d05ada669d7ff0567bb657.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02998402c64a65e3ea2fd0fdd8f688aeebda6192.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02a009e42054081b441d0f4b203679c4b0cae38d.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02a29bd2f39ecdbbb5e51a003c0bc7818029f384.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02a9fd5f564cb3a5b6b1fdbcd9df585b8e24a2de.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02aabb0b8eb8a3f3692e162442bbed19c5f915b6.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02b20ad26d6b2f05b38712292186edbf3aa05862.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02b8dea56378d11fe92d6a60ff37a34b0d6ea63e.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02bfb4c09489a5f48c23e8441e37b055bf7228ee.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02c4e304d5e063e9bd66d293a8c1472a5865c173.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02c9b77cb2898b1597278dbf32a0c61be93d1983.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02d808df510466b74102dd922486bf922a5ed947.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02da3b3b4a31f4e15f06a0f076524bea9ab0f92a.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02e34b2fe0968cb8a46313fe767858da7e2691b1.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02fdaf97dd0af539eada2af2f1563e0d77a28f92.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\02fe68e2ba45e7f0c6f746d4b3db36f8f99ae4f5.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\0304ed96b29c38014a17bacefcc294a02c04d0a0.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\03148d1ba3c986ec7306ab2a90bcd6082c512f49.json',\n",
       " 'comm_use_subset\\\\comm_use_subset\\\\031fd47b33af0e00eeb5e76c0d010df92df6bef6.json',\n",
       " ...]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get all the json files within the dataset into a list of paths\n",
    "all_json = glob.glob(f'**/*.json', recursive=True)\n",
    "all_json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "    \"paper_id\": \"0015023cc06b5362d332b3baf348d11567ca2fbb\",\n",
      "    \"metadata\": {\n",
      "        \"title\": \"The RNA pseudoknots in foot-and-mouth disease virus are dispensable for genome replication but essential for the production of infectious virus. 2 3\",\n",
      "        \"authors\": [\n",
      "            {\n",
      "                \"first\": \"Joseph\",\n",
      "                \"middle\": [\n",
      "                    \"C\"\n",
      "                ],\n",
      "                \"last\": \"Ward\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Lidia\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"Lasecka-Dykes\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Chris\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"Neil\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Oluwapelumi\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"Adeyemi\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Sarah\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"Gold\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Niall\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"Mclean\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Caroline\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"Wright\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Morgan\",\n",
      "                \"middle\": [\n",
      "                    \"R\"\n",
      "                ],\n",
      "                \"last\": \"Herod\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"David\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"Kealy\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Emma\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Warner\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Donald\",\n",
      "                \"middle\": [\n",
      "                    \"P\"\n",
      "                ],\n",
      "                \"last\": \"King\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Tobias\",\n",
      "                \"middle\": [\n",
      "                    \"J\"\n",
      "                ],\n",
      "                \"last\": \"Tuthill\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"David\",\n",
      "                \"middle\": [\n",
      "                    \"J\"\n",
      "                ],\n",
      "                \"last\": \"Rowlands\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Nicola\",\n",
      "                \"middle\": [\n",
      "                    \"J\"\n",
      "                ],\n",
      "                \"last\": \"\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            },\n",
      "            {\n",
      "                \"first\": \"Stonehouse\",\n",
      "                \"middle\": [],\n",
      "                \"last\": \"A#\",\n",
      "                \"suffix\": \"\",\n",
      "                \"affiliation\": {},\n",
      "                \"email\": \"\"\n",
      "            }\n",
      "        ]\n",
      "    },\n",
      "    \"abstract\": [\n",
      "        {\n",
      "            \"text\": \"word count: 194 22 Text word count: 5168 23 24 25 author/funder. All rights reserved. No reuse allowed without permission. Abstract 27 The positive stranded RNA genomes of picornaviruses comprise a single large open reading 28 frame flanked by 5\\u2032 and 3\\u2032 untranslated regions (UTRs). Foot-and-mouth disease virus (FMDV) 29 has an unusually large 5\\u2032 UTR (1.3 kb) containing five structural domains. These include the 30 internal ribosome entry site (IRES), which facilitates initiation of translation, and the cis-acting 31 replication element (cre). Less well characterised structures are a 5\\u2032 terminal 360 nucleotide 32 stem-loop, a variable length poly-C-tract of approximately 100-200 nucleotides and a series of 33 two to four tandemly repeated pseudoknots (PKs). We investigated the structures of the PKs 34 by selective 2\\u2032 hydroxyl acetylation analysed by primer extension (SHAPE) analysis and 35 determined their contribution to genome replication by mutation and deletion experiments. 36 SHAPE and mutation experiments confirmed the importance of the previously predicted PK 37 structures for their function. Deletion experiments showed that although PKs are not essential 38\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"Abstract\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"for replication, they provide genomes with a competitive advantage. However, although 39 replicons and full-length genomes lacking all PKs were replication competent, no infectious 40 virus was rescued from genomes containing less than one PK copy. This is consistent with our 41 earlier report describing the presence of putative packaging signals in the PK region. 42 43 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"Abstract\"\n",
      "        }\n",
      "    ],\n",
      "    \"body_text\": [\n",
      "        {\n",
      "            \"text\": \"VP3, and VP0 (which is further processed to VP2 and VP4 during virus assembly) (6). The P2 64 and P3 regions encode the non-structural proteins 2B and 2C and 3A, 3B (1-3) (VPg), 3C pro and 4 structural protein-coding region is replaced by reporter genes, allow the study of genome 68 replication without the requirement for high containment (9, 10) ( figure 1A ).\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [\n",
      "                {\n",
      "                    \"start\": 351,\n",
      "                    \"end\": 360,\n",
      "                    \"text\": \"figure 1A\",\n",
      "                    \"ref_id\": \"FIGREF50\"\n",
      "                }\n",
      "            ],\n",
      "            \"section\": \"\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The FMDV 5\\u2032 UTR is the largest known picornavirus UTR, comprising approximately 1300 71 nucleotides and containing several highly structured regions. The first 360 nucleotides at the 5\\u2032 72 end are predicted to fold into a single large stem loop termed the S-fragment, followed by a The PKs were originally predicted in 1987 and consist of two to four tandem repeats of a ~48 86 nucleotide region containing a small stem loop and downstream interaction site (figure 1B) 87 (12). Due to the sequence similarity between the PKs (figure 1C), it is speculated that they 88 were formed by duplication events during viral replication, probably involving recombination. 89 Between two and four PKs are present in different virus isolates but no strain has been 90 identified with less than two PKs, emphasising their potential importance in the viral life cycle 91 (19, 20) . The presence of PKs has been reported in the 5\\u2032 UTR of other picornaviruses such as 92 author/funder. All rights reserved. No reuse allowed without permission. can occur in the absence of PKs at least one is required for wild-type (wt) replication. 104 Furthermore, competition experiments showed that extra copies of PKs conferred a replicative 105 advantage to genomes. Although replicons and full-length genomes lacking PKs were 106 replication-competent, no infectious virus was rescued from genomes containing less than one 107 PK copy. This is consistent with our earlier report describing the presence of putative 108 packaging signals in the PK region (22). 109 110 author/funder. All rights reserved. No reuse allowed without permission. Plasmid construction. 117 The FMDV replicon plasmids, pRep-ptGFP, and the replication-defective polymerase mutant 118 control, 3D-GNN, have already been described (10).\",\n",
      "            \"cite_spans\": [\n",
      "                {\n",
      "                    \"start\": 469,\n",
      "                    \"end\": 471,\n",
      "                    \"text\": \"87\",\n",
      "                    \"ref_id\": null\n",
      "                },\n",
      "                {\n",
      "                    \"start\": 662,\n",
      "                    \"end\": 664,\n",
      "                    \"text\": \"89\",\n",
      "                    \"ref_id\": null\n",
      "                },\n",
      "                {\n",
      "                    \"start\": 857,\n",
      "                    \"end\": 861,\n",
      "                    \"text\": \"(19,\",\n",
      "                    \"ref_id\": null\n",
      "                },\n",
      "                {\n",
      "                    \"start\": 862,\n",
      "                    \"end\": 865,\n",
      "                    \"text\": \"20)\",\n",
      "                    \"ref_id\": null\n",
      "                },\n",
      "                {\n",
      "                    \"start\": 1117,\n",
      "                    \"end\": 1120,\n",
      "                    \"text\": \"104\",\n",
      "                    \"ref_id\": null\n",
      "                },\n",
      "                {\n",
      "                    \"start\": 1637,\n",
      "                    \"end\": 1640,\n",
      "                    \"text\": \"117\",\n",
      "                    \"ref_id\": null\n",
      "                }\n",
      "            ],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"70\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"To introduce mutations into the PK region, the pRep-ptGFP replicon plasmid was digested 121 with SpeI and KpnI and the resulting fragment inserted into a sub-cloning vector (pBluescript) 122 to create the pBluescript PK. PKs 3 and 4 were removed by digestion with HindIII and AatII 123 before insertion of a synthetic DNA sequence with PK 3 and 4 deleted. PKs 2, 3 and 4 were 124 deleted by PCR amplification using \\u0394PK 234 Forward primer and FMDV 1331-1311 reverse 125 primer, the resultant product was digested with HindIII and AatII and ligated into the 126 pBluescript PK vector. Complete PK deletion was achieved by introduction of an AflII site at 127 the 3\\u2032 end of the poly-C tract by PCR mutagenesis to create the sub-cloning vector, pBluescript 128 C11, which was then used to remove all the PKs by PCR mutagenesis using \\u0394PK 1234 forward 129 primer and FMDV 1331-1311 reverse primer. The modified PK sequences were removed from 130 the sub-cloning vectors and inserted into the pRep-ptGFP plasmid using NheI-HF and KpnI-131 HF.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"120\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"132 133 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"120\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint 7 Mutations to disrupt and reform PK structure were introduced using synthetic DNA by 134 digestion with AflII and AatII and ligation into a similarly digested pBluescript PK vector.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"120\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"Mutations were then introduced into the replicon plasmid as described above.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"135\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"To assess the effects of truncation of the poly-C-tract on replication the entire sequence was 137 removed. This was performed by PCR mutagenesis using primers C0 SpeI, and FMDV 1331- In vitro transcription. 143 In vitro transcription reactions for replicon assays were performed as described previously (28).\",\n",
      "            \"cite_spans\": [\n",
      "                {\n",
      "                    \"start\": 208,\n",
      "                    \"end\": 211,\n",
      "                    \"text\": \"143\",\n",
      "                    \"ref_id\": null\n",
      "                }\n",
      "            ],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"136\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"Transcription reactions to produce large amounts of RNA for SHAPE analysis were performed 145 with purified linear DNA as described above, and 1 \\u03bcg of linearised DNA was then used in a 146 HiScribe T7 synthesis kit (NEB), before DNase treatment and purification using a PureLink FastQ files were quality checked using FastQC with poor quality reads filtered using the 225 Sickle algorithm. Host cell reads were removed using FastQ Screen algorithm and FMDV 226 reads assembled de novo into contigs using IDBA-UD (35). Contigs that matched the FMDV 227 library (identified using Basic Local ALighnment Search Tool (BLAST)) were assembled 228 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [\n",
      "                {\n",
      "                    \"start\": 368,\n",
      "                    \"end\": 371,\n",
      "                    \"text\": \"225\",\n",
      "                    \"ref_id\": null\n",
      "                }\n",
      "            ],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"144\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint into consensus sequences using SeqMan Pro software in the DNA STAR Lasergene 13 229 package (DNA STAR) (36). The SHAPE data largely agreed with the predicted structures with the stems of PK 1, 2 and 3, interacting nucleotides showed little to no reactivity, suggesting NMIA could not interact with 300 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"144\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint 14 these nucleotides either due to the predicted base pairing or steric hindrance (figure 2B). The\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"144\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"NMIA reactivity for the interacting nucleotides in the stem-loops with downstream residues of 302 PK 1, 2 and 3 again largely agreed with the predicted structure, although the SHAPE data 303 suggests that there might be fewer interactions than previously predicted. However, differences 304 here could be due to heterogeneity in the formation of PKs in this experiment. The evidence 305 for loop-downstream interaction was weaker for PK4. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint\",\n",
      "            \"cite_spans\": [\n",
      "                {\n",
      "                    \"start\": 187,\n",
      "                    \"end\": 190,\n",
      "                    \"text\": \"303\",\n",
      "                    \"ref_id\": null\n",
      "                }\n",
      "            ],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"301\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"orientation. 351 Since removal of all four PKs resulted in a significant decrease in replication, the minimal 352 requirements to maintain wt levels of replication were investigated. As near wt level of 353 replication was observed when only one PK was present, all further mutagenesis was 354 performed in a C11 replicon plasmid containing only PK 1. In addition, the orientation of PK 1 was reversed by \\\"flipping\\\" the nucleotide sequence to 367 potentially facilitate hybridisation of the loop with upstream rather than downstream sequences.\",\n",
      "            \"cite_spans\": [\n",
      "                {\n",
      "                    \"start\": 13,\n",
      "                    \"end\": 16,\n",
      "                    \"text\": \"351\",\n",
      "                    \"ref_id\": null\n",
      "                }\n",
      "            ],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"Function of the PKs in replication is dependent on downstream interactions and 350\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"Changing the orientation of the PK reduced replicon replication to a similar level seen in the replication decreased until at passage three there is a 2.5 fold reduction compared to that of 398 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"368\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint passage 0 (figure 5B). Therefore, it appears that replicons with a single PK are at a competitive 399 disadvantage compared to those with two or more. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint 20 of infectious virus despite being able to replicate after transfection into cells, is consistent with 448 a requirement for RNA structure within the PK region being required for virus assembly. The 5\\u2032 UTR of FMDV is unique amongst picornaviruses due to its large size and the presence 454 of multiple RNA elements, some of which still have unknown function. One of these features 455 is a series of repeated PKs varying in number from 2-4, depending on virus strain. In this study, 456 we sequentially deleted or mutated the PKs to help understand their role in the viral life cycle. 457 We also confirmed the predicted PK structures by SHAPE mapping, although there may be Although all viruses isolated to date contain at least two PKs, replicons or viruses containing a 464 single PK were still replication competent. However, replicons with more than a single PK 465 were found to have a competitive advantage over replicons with a single PK when sequentially 466 passaged. Replicons lacking all PKs displayed poor passaging potential even when co-467 transfected with yeast tRNA, reinforcing the observation of a significant impact in replication.\",\n",
      "            \"cite_spans\": [\n",
      "                {\n",
      "                    \"start\": 920,\n",
      "                    \"end\": 923,\n",
      "                    \"text\": \"456\",\n",
      "                    \"ref_id\": null\n",
      "                },\n",
      "                {\n",
      "                    \"start\": 1022,\n",
      "                    \"end\": 1025,\n",
      "                    \"text\": \"457\",\n",
      "                    \"ref_id\": null\n",
      "                }\n",
      "            ],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"368\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"Moreover, viruses recovered from genomes with reduced numbers of PKs were slower growing 469 and produced smaller plaques. In addition, these differences were more pronounced in more PKs is functionally competent as no differences was seen between replicons congaing a single 472 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"468\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint 21 copy of PK1 or PK4. This observation is consistent with a previous report of deletion of PK1, 473 along with the Poly-C-tract, with no adverse effect in viral replication (37). This also supports 474 our findings that the truncation of the Poly-C-tract to create the C11 construct had no effect on 475 replicon replication in the cell lines tested. As has been described with Mengo virus, it is 476 possible that the role of the poly-C-tract is essential in other aspects of the viral lifecycle which 477 cannot be recapitulated in a standard tissue culture system (39).\",\n",
      "            \"cite_spans\": [\n",
      "                {\n",
      "                    \"start\": 443,\n",
      "                    \"end\": 446,\n",
      "                    \"text\": \"475\",\n",
      "                    \"ref_id\": null\n",
      "                }\n",
      "            ],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"468\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The presence of at least two PKs in all viral isolates sequenced so far suggests that multiple 480 PKs confer a competitive advantage in replication. Here we showed by sequential passage that 481 replicons containing at least two PKs were maintained at a level similar to wt, but replicons 482 containing only one PK showed a persistent decline. It is unclear why some viral isolates 483 contain two, three or four PKs is still unknown, but this may be stochastic variation or may 484 reflect subtle effects of host range or geographical localisation. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"479\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint Significance is shown comparing the replication of C11 PK disrupt and C11 PK restore (Aii). Significance shown is compared to wt replicon. Error bars are calculated by SEM, n = 3, * P 673 < 0.05, **** P < 0.0001. 674 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"479\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint 33 675 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"479\"\n",
      "        },\n",
      "        {\n",
      "            \"text\": \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint \",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"479\"\n",
      "        }\n",
      "    ],\n",
      "    \"bib_entries\": {\n",
      "        \"BIBREF0\": {\n",
      "            \"ref_id\": \"b0\",\n",
      "            \"title\": \"Genetic economy in 598 picornaviruses: Foot-and-mouth disease virus replication exploits alternative precursor 599 cleavage pathways\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"T\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Jackson\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"T\",\n",
      "                    \"middle\": [\n",
      "                        \"J\"\n",
      "                    ],\n",
      "                    \"last\": \"Tuthill\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"D\",\n",
      "                    \"middle\": [\n",
      "                        \"J\"\n",
      "                    ],\n",
      "                    \"last\": \"Rowlands\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"N\",\n",
      "                    \"middle\": [\n",
      "                        \"J\"\n",
      "                    ],\n",
      "                    \"last\": \"Stonehouse\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 2017,\n",
      "            \"venue\": \"PLOS Pathog\",\n",
      "            \"volume\": \"13\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF2\": {\n",
      "            \"ref_id\": \"b2\",\n",
      "            \"title\": \"A universal protocol to 602 generate consensus level genome sequences for foot-and-mouth disease virus and other 603 positive-sense polyadenylated RNA viruses using the Illumina MiSeq\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"N\",\n",
      "                    \"middle\": [\n",
      "                        \"D\"\n",
      "                    ],\n",
      "                    \"last\": \"Sanderson\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"N\",\n",
      "                    \"middle\": [\n",
      "                        \"J\"\n",
      "                    ],\n",
      "                    \"last\": \"Knowles\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"D\",\n",
      "                    \"middle\": [\n",
      "                        \"P\"\n",
      "                    ],\n",
      "                    \"last\": \"King\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"E\",\n",
      "                    \"middle\": [\n",
      "                        \"M\"\n",
      "                    ],\n",
      "                    \"last\": \"Cottam\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 2014,\n",
      "            \"venue\": \"BMC Genomics\",\n",
      "            \"volume\": \"604\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF3\": {\n",
      "            \"ref_id\": \"b3\",\n",
      "            \"title\": \"Library preparation for highly accurate population 606 sequencing of RNA viruses\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"A\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Acevedo\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"R\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Andino\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 2014,\n",
      "            \"venue\": \"Nat Protoc\",\n",
      "            \"volume\": \"9\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"1760--1769\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF4\": {\n",
      "            \"ref_id\": \"b4\",\n",
      "            \"title\": \"IDBA-UD: a de novo assembler for 608 single-cell and metagenomic sequencing data with highly uneven depth\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"Y\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Peng\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"Hcm\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Leung\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"S\",\n",
      "                    \"middle\": [\n",
      "                        \"M\"\n",
      "                    ],\n",
      "                    \"last\": \"Yiu\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"Fyl\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Chin\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 2012,\n",
      "            \"venue\": \"\",\n",
      "            \"volume\": \"\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF6\": {\n",
      "            \"ref_id\": \"b6\",\n",
      "            \"title\": \"Basic local alignment 611 search tool\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"S\",\n",
      "                    \"middle\": [\n",
      "                        \"F\"\n",
      "                    ],\n",
      "                    \"last\": \"Altschul\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"W\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Gish\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"W\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Miller\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"E\",\n",
      "                    \"middle\": [\n",
      "                        \"W\"\n",
      "                    ],\n",
      "                    \"last\": \"Myers\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"D\",\n",
      "                    \"middle\": [\n",
      "                        \"J\"\n",
      "                    ],\n",
      "                    \"last\": \"Lipman\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 1990,\n",
      "            \"venue\": \"J Mol Biol\",\n",
      "            \"volume\": \"215\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"403--410\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF7\": {\n",
      "            \"ref_id\": \"b7\",\n",
      "            \"title\": \"Genetically engineered foot-and-613 mouth disease viruses with poly(C) tracts of two nucleotides are virulent in mice\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"E\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Rieder\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"T\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Bunch\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"F\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Brown\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"P\",\n",
      "                    \"middle\": [\n",
      "                        \"W\"\n",
      "                    ],\n",
      "                    \"last\": \"Mason\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 1993,\n",
      "            \"venue\": \"J 614 Virol\",\n",
      "            \"volume\": \"67\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"5139--5184\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF9\": {\n",
      "            \"ref_id\": \"b9\",\n",
      "            \"title\": \"Both cis and trans Activities of Foot-and-Mouth Disease Virus 617 3D Polymerase Are Essential for Viral RNA Replication\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"N\",\n",
      "                    \"middle\": [\n",
      "                        \"J\"\n",
      "                    ],\n",
      "                    \"last\": \"Stonehouse\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 2016,\n",
      "            \"venue\": \"J Virol\",\n",
      "            \"volume\": \"90\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"6864--6883\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF10\": {\n",
      "            \"ref_id\": \"b10\",\n",
      "            \"title\": \"Mutational analysis of the 619 mengovirus poly(C) tract and surrounding heteropolymeric sequences\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"L\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Martin\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"G\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Duke\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"J\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Osorio\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"D\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Hall\",\n",
      "                    \"suffix\": \"\"\n",
      "                },\n",
      "                {\n",
      "                    \"first\": \"A\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Palmenberg\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": 1996,\n",
      "            \"venue\": \"J Virol\",\n",
      "            \"volume\": \"620\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"2027--2031\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF11\": {\n",
      "            \"ref_id\": \"b11\",\n",
      "            \"title\": \"No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the\",\n",
      "            \"authors\": [],\n",
      "            \"year\": null,\n",
      "            \"venue\": \"\",\n",
      "            \"volume\": \"\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {\n",
      "                \"DOI\": [\n",
      "                    \"10.1101/2020.01.10.901801\"\n",
      "                ]\n",
      "            }\n",
      "        },\n",
      "        \"BIBREF12\": {\n",
      "            \"ref_id\": \"b12\",\n",
      "            \"title\": \"Figure 3. The poly-C-tract is dispensable and only one PK is required for wt replication\",\n",
      "            \"authors\": [],\n",
      "            \"year\": null,\n",
      "            \"venue\": \"\",\n",
      "            \"volume\": \"\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF13\": {\n",
      "            \"ref_id\": \"b13\",\n",
      "            \"title\": \"A replicon 650 with entire poly-C-tract removed (C0) was transfected alongside wt, 3D-GNN and C11 651 replicons into BHK-21 cells (B). Replicons with sequentially deleted PKs (\\u0394PK 34, \\u0394PK 234 652 and C11 \\u0394PK 1234) were assayed for replication in BHK\",\n",
      "            \"authors\": [\n",
      "                {\n",
      "                    \"first\": \"3d\",\n",
      "                    \"middle\": [],\n",
      "                    \"last\": \"Wt\",\n",
      "                    \"suffix\": \"\"\n",
      "                }\n",
      "            ],\n",
      "            \"year\": null,\n",
      "            \"venue\": \"\",\n",
      "            \"volume\": \"\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF14\": {\n",
      "            \"ref_id\": \"b14\",\n",
      "            \"title\": \"All replication assays were measured by counting the number of GFP 655 author/funder. All rights reserved. No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the\",\n",
      "            \"authors\": [],\n",
      "            \"year\": null,\n",
      "            \"venue\": \"Replication of replicon with PK 4 as the sole remaining PK (C11 PK 4)\",\n",
      "            \"volume\": \"\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {\n",
      "                \"DOI\": [\n",
      "                    \"10.1101/2020.01.10.901801\"\n",
      "                ]\n",
      "            }\n",
      "        },\n",
      "        \"BIBREF15\": {\n",
      "            \"ref_id\": \"b15\",\n",
      "            \"title\": \"Error bars shown are calculated by SEM, n = 3\",\n",
      "            \"authors\": [],\n",
      "            \"year\": null,\n",
      "            \"venue\": \"\",\n",
      "            \"volume\": \"\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {}\n",
      "        },\n",
      "        \"BIBREF17\": {\n",
      "            \"ref_id\": \"b17\",\n",
      "            \"title\": \"No reuse allowed without permission. The copyright holder for this preprint (which was not peer-reviewed) is the\",\n",
      "            \"authors\": [],\n",
      "            \"year\": null,\n",
      "            \"venue\": \"\",\n",
      "            \"volume\": \"\",\n",
      "            \"issn\": \"\",\n",
      "            \"pages\": \"\",\n",
      "            \"other_ids\": {\n",
      "                \"DOI\": [\n",
      "                    \"10.1101/2020.01.10.901801\"\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    },\n",
      "    \"ref_entries\": {\n",
      "        \"FIGREF0\": {\n",
      "            \"text\": \"and-mouth disease virus (FMDV) is a single stranded positive sense RNA virus of the 45 genus Aphthovirus in the family Picornaviridae. It occurs as seven, antigenically diverse 46 serotypes; A, O, C, Asia 1, South African Territories (SAT) 1, 2 and 3. It is the causative agent 47 of foot-and-mouth disease (FMD), a highly contagious disease of cloven-hooved animals 48 affecting most notably cattle, pigs, sheep and goats in addition to wild species such as the 49 African buffalo. Disease outbreaks have serious economic implications resulting from trade 50 restrictions, reduced productivity and the slaughter of infected and at-risk animals (1). The 51 2001 outbreak in the UK caused economic losses of over \\u00a38 billion to the tourism and 52 agricultural sectors. Inactivated virus vaccines are used in countries in which FMD is endemic, 53 but these are often strain-specific and provide little cross protection between serotypes (2). 54 Antigenic variation together with the relatively short duration of immunity following 55 vaccination combine to complicate control of the disease (3). In addition, the carrier state, in 56 which asymptomatically infected animals continue to shed virus, contributes to the spread of 57 FMDV (4). An improved understanding of the viral life cycle may be important for the 58 development of improved vaccines and other control measures. 59 60 The FMDV genome (approximately 8.4 kb) consists of a single open reading frame flanked by 61 5\\u2032 and 3\\u2032 untranslated regions (UTRs) (figure 1A) (5). The translated region encodes both 62 structural and non-structural proteins. The P1 region encodes the capsid structural proteins VP1, 63\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF1\": {\n",
      "            \"text\": \"73 large poly-C tract of variable length (which can be up to 200 nt), a region containing two to 74 four tandemly repeated pseudoknots (PKs), the cis acting replication element (cre) and the 75 internal ribosome entry site (IRES) (5, 11, 12). Of these five structural domains, functions have 76 been ascribed to only two, the cre and IRES. The cre region is involved in uridylation of the 77 RNA primer peptide, VPg (also known as 3B), and the IRES determines the initiation of 78 translation of the viral polyprotein (13, 14). The roles of the S-fragment, the poly-C tract and 79 the PKs in viral replication are not fully elucidated, however recent studies have shown that 80 truncations to the S-fragment can play key roles in the innate immune response to viral 81 infection (15-17). It has also recently been reported that viruses containing a deletion within 82 the pseudoknot region showed an attenuated phenotype in bovine cell lines while remaining 83 unchanged in porcine, suggesting a role for the pseudoknots in viral tropism (18).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF3\": {\n",
      "            \"text\": \"138 1311 as forward and reverse primers respectively. The PCR product was digested with SpeI 139 and KpnI before ligation into a NheI and KpnI digested wt pRep ptGFP replicon. Sequences of 140 all primers are available upon request.141 142\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF5\": {\n",
      "            \"text\": \"prepared as above and a sample (12 pmol) was heated to 95 o C for 2 minutes before 151 cooling on ice. RNA folding buffer (100 mM HEPES, 66 mM MgCl2 and 100 mM NaCl) and 152 RNase Out (Invitrogen) was added to the RNA and incubated at 37 o C for 30 minutes. Once 153 folded, RNA was treated with NMIA compound at a final concentration of 5 mM or DMSO as 154 a negative control for 50 minutes at 37 o C. Following incubation, labelled RNA was ethanol 155 precipitated and resuspended in 10 \\u03bcl 0.5 x TE buffer.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF6\": {\n",
      "            \"text\": \"10 minutes in a thermocycler. A reverse transcription master mix containing 4 \\u03bcl first 161 strand buffer, 1 \\u03bcl 100 mM DTT, 0.5 \\u03bcl RNase Out, 1 \\u03bcl Supsercript III (Invitrogen), 1 \\u03bcl 10 162 mM PCR dNTP mix (Promega) and 0.5 \\u03bcl RNase free water, was then added to the 163 RNA/primer complex and extension carried out by incubation at 52 o C for 30 minutes.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF7\": {\n",
      "            \"text\": \", cDNA:RNA hybrids were disassociated by incubation with 1 \\u03bcl 4M NaOH at 166 95 o C for 3 minutes before neutralisation with 2 \\u03bcl 2M HCl. Extended cDNA was ethanol 167 precipitated and resuspended in 40 \\u03bcl deionized formamide (Thermo Fisher). Sequencing 168 ladders were made similarly using 6 pmol of RNA with the inclusion of 1 \\u03bcl 10 mM ddCTP in 169 the reverse transcription mix and using a differentially labelled fluorescent primer (either Hex 170 or FAM). 20 \\u03bcl of sequencing ladder was combined with NMIA or DMSO samples and 171 dispatched on dry ice for capillary electrophoresis (Dundee DNA seq).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF8\": {\n",
      "            \"text\": \"was analysed using QuShape and reactivity overlaid onto the RNA 174 structure using VARNA (29, 30).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF9\": {\n",
      "            \"text\": \"in all cell lines was assessed in 24-well plates with 0.5 \\u00b5g/cm 2 of RNA 178 using Lipofectin transfection reagent (Life Technologies) as previously described (28). For 179 complementation assays, BHK-21 cells seeded into 24-well plates were allowed to adhere for 180 16 hours before transfection with 1 \\u00b5g of replicon RNA using Lipofectin. Each transfection 181 was performed in duplicate and experiments were biologically repeated. Replicon replication by live cell imaging using an IncuCyte Zoom Dual colour FLR, an automated 183 phase-contrast and fluorescence microscope within a humidifying incubator. At hourly 184 intervals up to 24 hours post transfection, images of each well were taken and used to count 185 the number of ptGFP positive cells per well.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF10\": {\n",
      "            \"text\": \"competition assays was performed by co-transfecting BHK-21 cells with in vitro 188 transcribed replicon RNA and harvesting total cell RNA at 8 hours post transfection using 189 TRIzol reagent (Thermo Fisher Scientific). The harvested RNA was then purified using the 190 Direct-zol RNA MiniPrep kit (Zymo Research) with on-column DNase I treatment and eluted 191 in DEPC treated water. The purified passaged RNA (1 \\u00b5g) was transfected onto the na\\u00efve BHKhere are based on plasmid T7S3 which encodes a full length infectious copy 196 of FMDV O1 Kaufbeuren (31). The reporter was removed from replicons by digestion with 197 PsiI and XmaI restriction enzymes and replaced with the corresponding fragment from pT7S3 198 encoding the capsid proteins. Full length viral RNA was transcribed using a T7 MEGAscript 199 kit (Thermo Fisher Scientific), DNase treated using TurboDNase (Thermo Fisher Scientific) 200 and purified using a MEGAclear Transcription Clean-Up kit (Thermo Fisher Scientific). 201 RNA quality and concentration were determined by denaturing agarose gel electrophoresis 202 and Qubit RNA BR Assay Kit (Thermo Fisher Scientific).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF11\": {\n",
      "            \"text\": \"-transfection cell lysates were freeze-thawed and clarified by centrifugation.208    Clarified lysate was blind passaged onto na\\u00efve BHK-21 cells, this was continued for five 209 rounds of passaging.210 211 Sequencing of recovered virus. 212 Recovered viruses at passage 4, were sequenced using an Illumina Miseq (illumine) using a 213 modified version of a previously described PCR-free protocol ((32, 33)). Total RNA was 214 extracted from clarified passage 4 lysates using TRizol reagent (Thermo Fisher Scientific) 215 and residual genomic DNA removed using DNA-free DNA removal Kit (Thermo Fisher 216 Scientific). RNA was precipitated using 3 M sodium acetate and ethanol, 10 ul of purified 217 RNA (containing 1 pg to 5 \\u00b5g) of RNA was used in a reverse transcription reaction as 218 previously described (33, 34). Following reverse transcription cDNA was purified and 219 quantified using a Qubit ds DNA HS Assay kit (Thermo Fisher Scientific) and a cDNA 220 library prepared using Nextera XT DNA Sample Preparation Kit (Illumina). Sequencing was 221 carried out on the MiSeq platform using MiSeq Reagent Kit v2 (300 cycles) chemistry 222 (Illumina).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF13\": {\n",
      "            \"text\": \"of recovered virus. 232 Confluent BHK-21 cell monolayers were infected with 10-fold serial dilutions of virus stock, 233 overlaid with Eagle overlay media supplemented with 5 % tryptose phosphate broth solution 234 (Sigma Aldrich), penicillin (100 units/ml and streptomycin (100 \\u00b5g/ml) (Sigma Aldrich) and 235 0.6 % Indubiose (MP Biomedicals) and incubated for 48 hours at 37 o C. Cells were fixed and 236 stained with 1 % (w/v) methylene blue in 10 % (v/v) ethanol and 4 % formaldehyde in PBS. 237 238 Fixed plaques were scanned and images measured using a GNU Image Manipulation 239 Program IMP (GIMP, available at https://www.gimp.org). For each plaque, horizontal and 240 vertical diameter in pixels was taken and an average of these two values was calculated. All 241 plaques per well were measured. 242 243 Cell killing assays. 244 Virus titre was determined by plaque assays. BHK-21 cells were seeded with 3 x10 4 245 cells/well in 96 well plates and allowed to settle overnight. Cell monolayers were inoculated 246 with each rescued virus at MOI of 0.01 PFU for 1 hour, inoculum was removed and 150 \\u00b5l of 247 fresh GMEM (supplemented with 1 % FCS) was added to each well. Appearance of CPE was 248 monitored every 30 minutes using the IncuCyte S3.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF14\": {\n",
      "            \"text\": \"protein 3A, and a goat anti-Mouse IgG (H+L) highly cross-adsorbed 254 secondary antibody, Alexa Fluor 488 (Life Technologies). Each transcript was transfected in 255 triplicate and the experiment biologically repeated three times. BHK-21 cells were seeded 256 into T25 flasks 16 hours prior to transfection with 10 \\u00b5g RNA. The transfection mix was left 257 on the cells for 1 hour before the media was changed to VGM (Glasgow Minimum Essential 258 Medium (Sigma-Aldrich), 1% Foetal Bovine Serum -Brazil origin (Life Science Production) 259 and 5% Tryptose Phosphate Broth (Sigma-Aldrich).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF15\": {\n",
      "            \"text\": \"After a further 3 hours, cells were dissociated using trypsin-EDTA 0.05% phenol red (Life261 Technologies), pelleted at 200 g for 3 minutes and fixed in 4% paraformaldehyde for 40 262 minutes. Cells were then transferred to a 96-well u-bottom plate and pelleted; this and all 263 subsequent pelleting steps were done at 300 xg for 5 minutes. Cells were resuspended in 0.5% 264 BSA in PBS blocking buffer (Melford), pelleted and resuspended in 1/1000 2C2 antibody and 265 left shaking at 500 rpm at 4 o C for 14 hours in an Eppendorf Thermomixer C plate shaker. The 266 cells were pelleted and subsequently resuspended in blocking buffer three times to wash, 267 resuspended in 1/200 anti-mouse fluorescent secondary antibody and rotated at 500 rpm at 268 24 o C for 1 hour before washing a final three times. Cells were then resuspended in 500 \\u00b5l PBS 269 and data were collected on the LSR Fortessa (BD Biosciences) using BD FACSDivaTM 270 software. Data were exported as flow cytometry standard (FCS) files, and were analysed in 271 FlowJo 10 using the gating strategy shown in Figure 7.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF16\": {\n",
      "            \"text\": \"of PKs was initially predicted in 1987 by computational and visual analysis of 279 the 5\\u2032 UTR sequence (12). The prediction of the presence of multiple PKs was strengthened by 280 the observation that variation in the length of this region between different virus isolates 281 equated to the gain or loss of PK-length sequence units. However, the definitive demonstration 282 of PK structure remains a challenge. Here, we used selective 2\\u2032 hydroxyl acylation analysed by 283 primer extension (SHAPE) to investigate the secondary structure of the PK region.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF17\": {\n",
      "            \"text\": \"representing FMDV UTRs were folded prior to treatment with NMIA, a 286 compound that forms 2\\u2032-O-adducts when interacting with non-paired nucleotides, or DMSO as 287 a negative control. Labelled RNAs were purified and used as templates in reverse transcription 288 reactions using fluorescently labelled primers. Elongation of the reverse transcription products 289 terminates at adducts, resulting in cDNA fragments of different lengths, which were analysed 290 by gel electrophoresis alongside a sequencing ladder to identify sites of NMIA interaction. The 291 whole PK region was surprisingly reactive suggesting that it was largely single stranded or 292 highly flexible (figure 2A). To investigate if the SHAPE data agreed with the predicted 293structure, the NMIA reactivity was overlaid onto the previous PK structure prediction (12).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF19\": {\n",
      "            \"text\": \"296 being unreactive, suggestive of base-pairing. Formation of the stem of PK4 was less convincing, 297 although the stem nucleotides still had relatively low reactivity in agreement with the other PK 298 models. For all the PKs, the nucleotides in the loop regions and the predicted downstream299\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF20\": {\n",
      "            \"text\": \"the NMIA reactivities with the original predicted structure the SHAPE data were 308 compatible to the PK models and potentially shed new light on the requirements of the loop 309 interactions.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF21\": {\n",
      "            \"text\": \"PK is sufficient for efficient replication.312    The replicon system was based on the O1K FMDV sequence which includes four similar but 313 non-identical PKs (figure 1). The PKs were sequentially deleted from the 3\\u2032 side (i.e. PK 4-PK 314 1), and replication of the resulting modified replicons assessed.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF22\": {\n",
      "            \"text\": \"complete removal of all PKs, an AflII site was inserted into the ptGFP replicon 317 plasmid which resulted in reduction of the poly-C-tract to 11 cytosine residues. This C11 318 replicon was investigated alongside a wt replicon and one with lethal polymerase mutations 319 (3D-GNN). These controls were used to confirm that truncation of the poly-C tract had no 320 measurable effect on replication the two cell lines tested, as previously reported (37) (figure 321 3A). For completeness, we further removed the entire poly-C-tract (C0) and showed that this 322 had no observable negative effect on replication of the replicon (figure 3B). The C11 construct 323 was then used as the \\\"backbone\\\" for removal of all four PKs.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF23\": {\n",
      "            \"text\": \"measuring ptGFP reporter expression, in parallel with transfection of a wt and 327 3D-GNN replicon, where the 3D-GNN replicon is used to monitor ptGFP expression resulting 328 from translation of input RNA in the absence of replication. Reporter expression was recorded 329 using an IncuCyte Zoom automatic fluorescent microscope and is shown at 8 hours post-\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF25\": {\n",
      "            \"text\": \"\\u2206PK 234 respectively) replicated at similar levels to the wt replicon (figure 3C-D). 334 However, a replicon containing no PKs (C11 \\u2206PK 1234) showed a significant (~ 4 fold) 335 reduction in replication in BHK-21 cells compared to the wt C11 replicon. A larger reduction 336 in replication (28 fold) was seen in the MDBK cell line, supporting previous publications on 337 the potential role in host cell tropism (18). Replication of the C11 \\u2206PK 1234 replicon in MDBK 338 cells was however still significantly above that of the 3D-GNN negative control. These data 339 suggest that although the PKs are not essential for replication at least one PK is required for wt 340 levels of replication. 341 342In the experiments above PK1 was the sole remaining PK and we therefore investigated 343 whether other PKs could similarly support wt replication. We deleted all the PKs to create the 344 C11 construct and re-inserted PK4 as the only PK (C11 PK4). Near wt levels of replication 345 were observed following transfection into both cell types suggesting that there is no functional 346 difference between PK1 and PK4 (figure 3E).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF27\": {\n",
      "            \"text\": \"to interrupt base pairing and abrogate formation of the PK structure were 357 made in the loop of PK 1 and the corresponding downstream nucleotides. The substitutions 358 (shown in red) created a GAGA motif both in the loop and downstream regions and reduced 359 the replication of the mutated replicon (C11 PK disrupt) equivalent to that of the replicon 360 containing no PKs, thereby supporting the predicted structure (figure 4A). Base pairing 361 potential was then restored by mutation of the relevant nucleotides in the loop and downstream 362 region to GGGG and CCCC respectively. Restoring the interaction using an alternate sequence 363 increased replication significantly compared to the disrupted PK replicon (~ 4 fold), although 364 this was still slightly below that of the wt (~ 0.7 fold decrease) (figure 4A).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF29\": {\n",
      "            \"text\": \"369absence of PKs (figure 4B). This suggests that the role of the PKs in genome replication is 370 dependent on both sequence, structure and orientation.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF30\": {\n",
      "            \"text\": \"studies above suggested that removal of up to three of the four PKs present in the 375 wt sequence had no clear effect on replicon replication, although deletion of all four was 376 significantly detrimental. To investigate whether multiple PKs conferred more subtle 377 advantages for replication than were evident from single round transfection experiments we 378 carried out sequential passages of replicon RNA following transfection of the PK deleted forms 379 in competition with a wt replicon. Different reporter genes (ptGFP or mCherry) were used to 380 distinguish the competing replicons.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF31\": {\n",
      "            \"text\": \"ptGFP; wt, \\u2206PK 34, \\u2206PK 234 and C11 \\u2206PK 1234 were co-transfected into 383 BHK-21 cells together with either a wt mCherry replicon or yeast tRNA as a control. The 384 replication of each of the co-transfected replicons was compared by observing ptGFP and 385 mCherry expression over three sequential passages. Passaging was achieved by harvesting total 386 RNA using Trizol-reagent 8 hours post-transfection. Harvested RNA was purified and then re-387 transfected into na\\u00efve BHK-21 cells.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF32\": {\n",
      "            \"text\": \"transfection of the wt, \\u2206PK 34 or \\u2206PK 234 with yeast tRNA as controls showed no 390 differences in replication as expected (Figure 5A). Likewise, when PK mutants were co-391 transfected with a wt replicon after three passages, the number of green fluorescent cells 392 produced by the \\u2206PK 34 replicon was comparable to that of the wt, suggesting no competitive 393 advantage of four PKs over two. For both, there was a reduction in replication after the first 394 passage but recovery to near that of the original transfection by the third passage. However, 395 when co-transfected with the wt replicon, the \\u2206PK 234 replicon showed a similar drop in 396 replication in passage two, but showed no subsequent recovery following each passage and397\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF33\": {\n",
      "            \"text\": \"transfection with the wt mCherry replicon reduced the replication of the C11 \\u2206PK 1234 402 replicon to background levels as seen when comparing to the yeast tRNA control. By passage 403 two the ptGFP signal of the C11 \\u2206PK 1234 was no longer detectable, suggesting that this 404 replicon has been out competed (figure 5C). Although the initial replication of C11 \\u2206PK 1234 405 was greater when co-transfected with yeast tRNA than when in competition with wt mCherry 406 replicon, the ptGFP signal was reduced at passage two and was at background level by passage 407 three (figure 5C). Replication of the mCherry wt replicon was not influenced by co-transfection 408 with the ptGFP constructs (figure 5D), as expected. Together these data suggest that the minor 409 replicative advantage conferred by multiple PKs are quickly compounded over multiple 410 replication cycles to provide a replicative advantage.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF34\": {\n",
      "            \"text\": \"a PK is essential for the production of infectious virus413    As replicons lacking all PKs could replicate and replicons with reduced numbers of PKs414 appeared to be at a competitive disadvantage compared to the wt construct, we investigated the 415 consequences of PK manipulation on the complete viral life cycle. The \\u2206PK 34, \\u2206PK 234 and 416 C11 \\u2206PK 1234 mutations were introduced into an FMDV infectious clone by replacement of 417 sequence encoding ptGFP with that encoding the O1K structural proteins. RNA transcripts 418 were transfected into BHK-21 cells alongside a wt O1K viral transcript and blind passaged 5 419 times by transferring the cell supernatant at 24 hours post transfection onto na\\u00efve BHK-21 cells.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF36\": {\n",
      "            \"text\": \"427 rate of CPE (figure 6A) and plaque size (figure 6B-C) of \\u2206PK 34 and \\u2206PK 234 when compared 428 to the wt O1K virus. Rate of CPE was monitored by infecting BHK-21 cells with a known MOI 429 (0.01) of recovered virus, cells were then monitored for signs of CPE (shown as a decrease in 430 cell confluency) as measured by an automated imaging platform (Incucyte Zoom). Both \\u2206PK 431 34 and \\u2206PK 234 showed delayed onset of CPE with \\u2206PK 34 being the slowest, initial CPE 432 occurring at approximately 39 hours and 29 hours post infection respectively, compared to the 433 22 hours seen in the wt control. This mirrored plaque assay data where \\u2206PK 34 displayed a 434 significantly smaller plaque phenotype when compared to the wt control (average of 13.8 pixels 435 compared to 37.4), the slower rate of CPE seen in \\u2206PK 234 made a small, but not significant 436 difference (average 31.9 pixels).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF37\": {\n",
      "            \"text\": \"\\u2206PK 1234 produced no infectious virus the ability of the full-length genome lacking 439 PKs to replicate was investigated. BHK-21 cells were transfected with the same RNA 440 transcripts as above alongside additional controls, mock-transfected and transfected with wt 441 and treated with 3 mM GuHCl (a replication inhibitor) as negative controls. Six hours post-442 transfection, cells were harvested, fixed and labelled with an anti-3A antibody and fluorescent 443 secondary antibody. Cells were then analysed using flow cytometry and anti-3A antibody 444 signal used as an indirect measure of genome replication (figure 7). The results were similar to 445 those of the replicon experiments and showed that all the modified virus genomes were able to 446 undergo robust replication. The inability of the C11 \\u2206PK 1234 genome to support production 447 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF39\": {\n",
      "            \"text\": \"458fewer strong interactions maintaining the PKs than was previously predicted. This may indicate 459 high conformational flexibility of this region of the genome. SHAPE mapping was also 460 supported by mutation of predicted key interactions between nucleotides in the loop and 461 downstream, disruption of which reduced replication to that of the C11 \\u2206PK 1234 replicon.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF41\": {\n",
      "            \"text\": \"470 relevant cells lines (i.e. in MDBK cells compared to BHK 21 cells). It is likely that each of the 471\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF42\": {\n",
      "            \"text\": \"although removal of all four PKs resulted in a significant decrease in replicon and 487 viral genome replication, replication was not abolished, showing that PKs are not essential to 488 support genome replication. However, deletion of all PKs from an infectious clone completely 489 abolished the ability to recover infectious virus. This suggests that the genome lacking all PKs 490 is defective in a function associated with virion assembly and is compatible with our evidence 491 for the presence of a packaging signal in a similar location on the genome to PK1 (22). It is 492 possible that structural flexibility at this site in the genome allows the RNA to adopt alternate 493 conformations with different roles in genome replication and virion assembly. A functional 494 requirement for multiple RNA conformations may explain the relatively weak interactions 495 between nucleotides involved in stabilising the PK motif as observed by SHAPE analysis or 496 by structural prediction.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF43\": {\n",
      "            \"text\": \"was supported by funding from the Biotechnology and Biological Sciences Research 503 Council (BBSRC) of the United Kingdom (research grant BB/K003801/1). Additionally, the 504 Pirbright Institute receives grant-aided support from the BBSRC (projects BB/E/I/00007035, 505 BB/E/I/00007036 and BBS/E/I/00007037).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF44\": {\n",
      "            \"text\": \"Jones TJD, Rushton J. 2013. The economic impacts of foot and mouth disease 511 -What are they, how big are they and where do they occur? Prev Vet Med 112:161-M, Parida S. 2018. Foot and mouth disease vaccine strain selection: current 514 approaches and future perspectives. Expert Rev Vaccines 17:577-591.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF45\": {\n",
      "            \"text\": \"J-H. 2013. Requirements for improved vaccines against foot-and-mouth disease 516 epidemics. Clin Exp Vaccine Res 2:8-18.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF46\": {\n",
      "            \"text\": \"C, Eschbaumer M, Rekant SI, Pacheco JM, Smoliga GR, Hartwig EJ, 518 Rodriguez LL, Arzt J. 2016. The Foot-and-Mouth Disease Carrier State Divergence in 519 Cattle. J Virol 90:6344-64.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF47\": {\n",
      "            \"text\": \"C, Tulman ER, Delhon G, Lu Z, Carreno A, Vagnozzi A, Kutish GF, Rock 521 DL. 2005. Comparative genomics of foot-and-mouth disease virus. J Virol 79:6487-D. 1997. Dissecting the roles of VP0 cleavage and RNA packaging in 525 picornavirus capsid stabilization: the structure of empty capsids of foot-and-mouth 526 disease virus. J Virol 71:9743-52. 527 7. Gao Y, Sun S-Q, Guo H-C. 2016. Biological function of Foot-and-mouth disease virus 528 non-structural proteins and non-coding elements. Virol J 13:107. 529 8. Herod MR, Gold S, LaseckaDykes L, Wright C, Ward JC, McLean TC, Forrest S, 530 Jackson T, Tuthill TJ, Rowlands DJ, Stonehouse NJ. 2017. Genetic economy in 531 picornaviruses: Foot-and-mouth disease virus replication exploits alternative precursor 532 cleavage pathway. PLOS Pathog 13:e1006666. 533 9. Tulloch F, Pathania U, Luke GA, Nicholson J, Stonehouse NJ, Rowlands DJ, Jackson 534 T, Tuthill T, Haas J, Lamond AI, Ryan MD. 2014. FMDV replicons encoding green 535 fluorescent protein are replication competent. J Virol Methods 209:35-40. 536 10. Herod MR, Tulloch F, Loundras E-A, Ward JC, Rowlands DJ, Stonehouse NJ. 2015. 537 Employing transposon mutagenesis to investigate foot-and-mouth disease virus 538 replication. J Gen Virol 96:3507-3518. 539 11. Mellor EJC, Brown F, Harris TJR. 1985. Analysis of the Secondary Structure of the 540 Poly(C) Tract in Foot-and-Mouth Disease Virus RNAs. J Gen Virol 66:1919-1929. 541 12. Clarke BE, Brown AL, Currey KM, Newton SE, Rowlands DJ, Carroll AR. 1987. 542 Potential secondary and tertiary structure in the genomic RNA of foot and mouth 543 disease virus. Nucleic Acids Res 15:7067-7079. 544 13. Nayak A, Goodfellow IG, Woolaway KE, Birtley J, Curry S, Belsham GJ. 2006. Role 545 of RNA structure and RNA binding activity of foot-and-mouth disease virus 3C 546 protein in VPg uridylylation and virus replication. J Virol 80:9865-75. Kloc A, Diaz-San Segundo F, Schafer EA, Rai DK, Kenney M, de los Santos T, 555 Rieder E. 2017. Foot-and-mouth disease virus 5'-terminal S fragment is required for 556 replication and modulation of the innate immune response in host cells. Virology 557 512:132-143. 558 17. Kloc A, Rai DK, Rieder E. 2018. The roles of picornavirus untranslated regions in 559 infection and innate immunity. Front Microbiol. Frontiers Media S.A. 560 18. Zhu Z, Yang F, Cao W, Liu H, Zhang K, Tian H, Dang W, He J, Guo J, Liu X, Zheng 561 H. 2019. The Pseudoknot Region of the 5' Untranslated Region Is a Determinant of 562 Viral Tropism and Virulence of Foot-and-Mouth Disease Virus. J Virol 93. 563 19. Mohapatra JK, Pawar SS, Tosh C, Subramaniam S, Palsamy R, Sanyal A, Hemadri D, 564 Pattnaik B. 2011. Genetic characterization of vaccine and field strains of serotype A 565 foot-andmouth disease virus from India. Acta Virol 55:349-352. 566 20. Escarm\\u00eds C, Dopazo J, D\\u00e1vila M, Palma EL, Domingo E. 1995. Large deletions in the 567 5'-untranslated region of foot-and-mouth disease virus of serotype C. Virus Res 568 35:155-67.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF48\": {\n",
      "            \"text\": \"Carocci M, Bakkali-Kassimi L. 2012. The encephalomyocarditis virus. Virulence\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF49\": {\n",
      "            \"text\": \". Wutz G, Auer H, Nowotny N, Grosse B, Skern T, Kuechler E. 1996. Equine rhinovirus Xrn1 produce a pathogenic Dengue virus RNA. Elife 3. 576 24. Kieft JS, Rabe JL, Chapman EG. 2015. New hypotheses derived from the structure of 577 a flaviviral Xrn1-resistant RNA: Conservation, folding, and host adaptation. RNA Biol 578 12:1169-77.579 25. Gultyaev AP, Olsthoorn RCL. 2010. A family of non-classical pseudoknots in 580 influenza A and B viruses. RNA Biol 7:125-9. 581 26. Moss WN, Dela-Moss LI, Priore SF, Turner DH. 2012. The influenza A segment 7 582 mRNA 3' splice site pseudoknot/hairpin family. RNA Biol 9:1305-10. 583 27. Plant EP, Dinman JD. 2008. The role of programmed-1 ribosomal frameshifting in 584 coronavirus propagation. Front Biosci 13:4873-81. 585 28. Herod MR, Ferrer-Orta C, Loundras E-A, Ward JC, Verdaguer N, Rowlands DJ, 586 Stonehouse NJ. 2016. Both cis and trans Activities of Foot-and-Mouth Disease Virus 587 3D Polymerase Are Essential for Viral RNA Replication. J Virol 90:6864-6883. 588 29. Karabiber F, McGinnis JL, Favorov O V., Weeks KM. 2013. QuShape: Rapid, 589 accurate, and best-practices quantification of nucleic acid probing information, 590 resolved by capillary electrophoresis. RNA 19:63-73. 591 30. Darty K, Denise A, Ponty Y. 2009. VARNA: Interactive drawing and editing of the 592 RNA secondary structure. Bioinformatics 25:1974-1975. 593 31. King AMQ, Blakemore WE, Ellard FM, Drew J, Stuart DI. 1999. Evidence for the role 594 of His-142 of protein 1C in the acid-induced disassembly of foot-and-mouth disease 595 virus capsids. J Gen Virol 80:1911-1918. 596 32. Herod MR, Gold S, Lasecka-Dykes L, Wright C, Ward JC, McLean TC, Forrest S, 597 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF50\": {\n",
      "            \"text\": \"Replicon and PK schematic. Schematic of the FMDV O1K sub-genomic replicon, 627 showing both 5' and 3' untranslated regions (UTRs) together with the RNA structures present 628 in these regions. IRES-driven translation produces a single polyprotein. Here, the structural 629 proteins have been replaced with a green fluorescent reporter, upstream of the non-structural 630 proteins 2A-3D (A). Predicted PK structures, with putative interactions highlighted in hot-pink 631 are shown. Numbers indicate nucleotide positions after the poly-C-tract (B). Sequence 632 alignment of the 4 PKs, with the interacting regions shown in hot-pink and invariant 633 nucleotides represented by asterisk (C).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF51\": {\n",
      "            \"text\": \"SHAPE NMIA reactivity of the PK region. NMIA reactivity at nucleotides 640 following the poly-C-tract (PCT). High reactivity indicates increased chance of the nucleotide 641 being non base-paired at that position (A). NMIA reactivity of each PK overlaid onto the 642 predicted PK structure using VARNA (30). Loop and downstream interactions represent those 643 supported by SHAPE data (B). NMIA reactivity is represented on a colour scale from low 644 (white) to high (red) (n = 4). 645 646 author/funder. All rights reserved. No reuse allowed without permission.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF52\": {\n",
      "            \"text\": \"Disrupting the PK structure and reversing the orientation of a PK reduces 663 replication. Cartoon representations of disrupting and restoring mutations made to PK 1, 664 where nucleotides in the bulge of the stem loop and interacting region downstream were 665 mutated to disrupt structure formation 'PK disrupt', or mutated to maintain bulge and 666 downstream interaction but with different nucleotides 'PK restore' (Ai). Replication of PK 667 disrupt and restore mutants were measured by transfection of RNA into BHK-21 cells and 668 shown here at 8 hours post-transfection alongside wt, 3D-GNN and C11 \\u0394PK 1234 controls.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF54\": {\n",
      "            \"text\": \"Visual representation of the reversing of the nucleotide sequence of PK1 creating the C11 PK 671 Rvs construct (Bi). Replication of PK Rvs at 8 hours post transfection of BHK-21 cells (Bii).\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"FIGREF56\": {\n",
      "            \"text\": \"More than 2 PKs provides a replicative advantage in co-transfection\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"figure\"\n",
      "        },\n",
      "        \"TABREF0\": {\n",
      "            \"text\": \"encephalomyocarditis virus (EMCV) and equine rhinitis A virus (ERAV) (21, 22). However, in both cases the PKs are located at the 5\\u2032 side of the poly-C-tract, making their location in the FMDV genome unique. PKs have been reported to have roles in several aspects of viral replication including splicing 97 (e.g. HIV and influenza), ribosomal frameshifting (e.g. coronaviruses) and RNase protection 98 (e.g. Dengue virus) (23-27). In the work reported here, the role of the PKs in the FMDV life cycle was investigated, together with biochemical probing of PK structures. The combination of both virus and replicon systems allowed us to distinguish effects on genome replication and 101 other aspects of the viral life cycle. Selective mutation within the PK domain and sequential 102 deletion of PKs confirmed the importance of PK structure and that although genome replication\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"table\"\n",
      "        },\n",
      "        \"TABREF1\": {\n",
      "            \"text\": \"Materials and MethodsCells lines.BHK-21 cells obtained from the ATCC (LGC Standard) were maintained in Dulbecco's modified Eagle's Medium with glutamine (Sigma-Aldrich) supplemented with 10 % foetal calf serum (FCS), 50 U/ml penicillin and 50 \\u00b5g/ml streptomycin.\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"table\"\n",
      "        },\n",
      "        \"TABREF2\": {\n",
      "            \"text\": \"Wt, C11, \\u2206PK 34 and \\u2206PK 234 constructs all resulted in the production of infectious virus as was expected from the replicon experiments, with no alteration to input sequence. However, the C11 \\u2206PK 1234, which replicated (albeit to a lesser degree) as a replicon, produced no recoverable infectious virus(Table 1). Interestingly, there were differences noted in both the\",\n",
      "            \"latex\": null,\n",
      "            \"type\": \"table\"\n",
      "        }\n",
      "    },\n",
      "    \"back_matter\": [\n",
      "        {\n",
      "            \"text\": \"author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.10.901801 doi: bioRxiv preprint\",\n",
      "            \"cite_spans\": [],\n",
      "            \"ref_spans\": [],\n",
      "            \"section\": \"annex\"\n",
      "        }\n",
      "    ]\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "#check json schema, as per [1]\n",
    "with open(all_json[0]) as file:\n",
    "    first_entry = json.load(file)\n",
    "    print(json.dumps(first_entry, indent=4))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "#helper class taken from [1]\n",
    "class FileReader:\n",
    "    def __init__(self, file_path):\n",
    "        with open(file_path) as file:\n",
    "            content = json.load(file)\n",
    "            self.paper_id = content['paper_id']\n",
    "            self.source = file_path.split(\"\\\\\")[0]\n",
    "            self.title = content['metadata']['title']\n",
    "            self.abstract = []\n",
    "            self.body_text = []\n",
    "            self.bib_entries = []\n",
    "            # Abstract\n",
    "            for entry in content['abstract']:\n",
    "                self.abstract.append(entry['text'])\n",
    "            # Body text\n",
    "            for entry in content['body_text']:\n",
    "                self.body_text.append(entry['text'])\n",
    "            # Bibliography\n",
    "            for k, entry in content['bib_entries'].items():\n",
    "                self.bib_entries.append(str(entry)) \n",
    "                # just stringify everything we'll probably clean later\n",
    "            self.abstract = ' '.join(self.abstract)\n",
    "            self.bib_entries = str(self.bib_entries)\n",
    "            \n",
    "            #Strip copyright disclaimers\n",
    "            self.body_text = ' '.join(self.body_text)\n",
    "            reg = '((The copyright holder for this preprint \\(which was not peer-reviewed\\) is the . )(https:\\/\\/doi.org\\/10.[0-9]{4,}?\\/[0-9]{4}.[0-9]{2}.[0-9]{2}.[^\\s]+)( doi: bioRxiv preprint)|(author\\/funder. All rights reserved. No reuse allowed without permission.))'\n",
    "            re.sub(reg, '', self.body_text)\n",
    "            \n",
    "            \n",
    "            '''\n",
    "            stringify the whole bib_entries list so we can use eval() \n",
    "            on a df cell at a later time to get the full list of dicts\n",
    "            '''\n",
    "    def __repr__(self):\n",
    "        return f'{self.paper_id}: {self.abstract[:200]}... {self.body_text[:200]}...'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0015023cc06b5362d332b3baf348d11567ca2fbb: word count: 194 22 Text word count: 5168 23 24 25 author/funder. All rights reserved. No reuse allowed without permission. Abstract 27 The positive stranded RNA genomes of picornaviruses comprise a si... VP3, and VP0 (which is further processed to VP2 and VP4 during virus assembly) (6). The P2 64 and P3 regions encode the non-structural proteins 2B and 2C and 3A, 3B (1-3) (VPg), 3C pro and 4 structura...\n"
     ]
    }
   ],
   "source": [
    "#checking it out\n",
    "first_row = FileReader(all_json[0])\n",
    "print(first_row)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['',\n",
       " '70',\n",
       " '120',\n",
       " '120',\n",
       " '120',\n",
       " '135',\n",
       " '136',\n",
       " '144',\n",
       " '144',\n",
       " '144',\n",
       " '301',\n",
       " '350',\n",
       " '368',\n",
       " '368',\n",
       " '468',\n",
       " '468',\n",
       " '479',\n",
       " '479',\n",
       " '479',\n",
       " '479']"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[re.sub('\\D', '', each['section']) for each in first_entry['body_text']]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# The data is a bit messy\n",
    "This is specifically about the biorxiv data, since all papers in this set were probably scraped the same way.\n",
    "    \n",
    "    1. There are random numbers. these are caused by one of the following:\n",
    "        * superscripts/subscripts \n",
    "        * preprint pdf line numbers\n",
    "\n",
    "    2. The preprint pdf includes a copyright header that gets scraped into the text. This is almost the exact same each time, so it can probably be removed pretty easily\n",
    "\n",
    "    3. Section values have something to do with the line number that the text paragraph starts on, but I can't find a consistent rule for how they were scraped\n",
    "\n",
    "For now I'll just read everything into a dataframe with the helper class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing index: 0 of 29315\n",
      "Processing index: 2931 of 29315\n",
      "Processing index: 5862 of 29315\n",
      "Processing index: 8793 of 29315\n",
      "Processing index: 11724 of 29315\n",
      "Processing index: 14655 of 29315\n",
      "Processing index: 17586 of 29315\n",
      "Processing index: 20517 of 29315\n",
      "Processing index: 23448 of 29315\n",
      "Processing index: 26379 of 29315\n",
      "Processing index: 29310 of 29315\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_id</th>\n",
       "      <th>source</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>body_text</th>\n",
       "      <th>bib_entries</th>\n",
       "      <th>abstract_word_count</th>\n",
       "      <th>body_word_count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>21050</td>\n",
       "      <td>21050</td>\n",
       "      <td>21050</td>\n",
       "      <td>21050</td>\n",
       "      <td>21050</td>\n",
       "      <td>21050</td>\n",
       "      <td>21050.000000</td>\n",
       "      <td>21050.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>unique</th>\n",
       "      <td>21050</td>\n",
       "      <td>4</td>\n",
       "      <td>19993</td>\n",
       "      <td>21050</td>\n",
       "      <td>21046</td>\n",
       "      <td>21035</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>top</th>\n",
       "      <td>cfe74dcee96b3f90d0b1a0bbba20a8206490a26b</td>\n",
       "      <td>custom_license</td>\n",
       "      <td></td>\n",
       "      <td>The presence of Giardia and Cryptosporidium wa...</td>\n",
       "      <td>Gastrointestinal disorders are frequently repo...</td>\n",
       "      <td>[\"{'ref_id': 'b0', 'title': 'Situation Report ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>freq</th>\n",
       "      <td>1</td>\n",
       "      <td>10527</td>\n",
       "      <td>912</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mean</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>227.180855</td>\n",
       "      <td>4593.006698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>std</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>168.553834</td>\n",
       "      <td>5556.557109</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>min</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>1.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>149.000000</td>\n",
       "      <td>2735.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>203.000000</td>\n",
       "      <td>3861.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75%</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>263.750000</td>\n",
       "      <td>5469.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>max</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4767.000000</td>\n",
       "      <td>258598.000000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        paper_id          source  title  \\\n",
       "count                                      21050           21050  21050   \n",
       "unique                                     21050               4  19993   \n",
       "top     cfe74dcee96b3f90d0b1a0bbba20a8206490a26b  custom_license          \n",
       "freq                                           1           10527    912   \n",
       "mean                                         NaN             NaN    NaN   \n",
       "std                                          NaN             NaN    NaN   \n",
       "min                                          NaN             NaN    NaN   \n",
       "25%                                          NaN             NaN    NaN   \n",
       "50%                                          NaN             NaN    NaN   \n",
       "75%                                          NaN             NaN    NaN   \n",
       "max                                          NaN             NaN    NaN   \n",
       "\n",
       "                                                 abstract  \\\n",
       "count                                               21050   \n",
       "unique                                              21050   \n",
       "top     The presence of Giardia and Cryptosporidium wa...   \n",
       "freq                                                    1   \n",
       "mean                                                  NaN   \n",
       "std                                                   NaN   \n",
       "min                                                   NaN   \n",
       "25%                                                   NaN   \n",
       "50%                                                   NaN   \n",
       "75%                                                   NaN   \n",
       "max                                                   NaN   \n",
       "\n",
       "                                                body_text  \\\n",
       "count                                               21050   \n",
       "unique                                              21046   \n",
       "top     Gastrointestinal disorders are frequently repo...   \n",
       "freq                                                    2   \n",
       "mean                                                  NaN   \n",
       "std                                                   NaN   \n",
       "min                                                   NaN   \n",
       "25%                                                   NaN   \n",
       "50%                                                   NaN   \n",
       "75%                                                   NaN   \n",
       "max                                                   NaN   \n",
       "\n",
       "                                              bib_entries  \\\n",
       "count                                               21050   \n",
       "unique                                              21035   \n",
       "top     [\"{'ref_id': 'b0', 'title': 'Situation Report ...   \n",
       "freq                                                    2   \n",
       "mean                                                  NaN   \n",
       "std                                                   NaN   \n",
       "min                                                   NaN   \n",
       "25%                                                   NaN   \n",
       "50%                                                   NaN   \n",
       "75%                                                   NaN   \n",
       "max                                                   NaN   \n",
       "\n",
       "        abstract_word_count  body_word_count  \n",
       "count          21050.000000     21050.000000  \n",
       "unique                  NaN              NaN  \n",
       "top                     NaN              NaN  \n",
       "freq                    NaN              NaN  \n",
       "mean             227.180855      4593.006698  \n",
       "std              168.553834      5556.557109  \n",
       "min                0.000000         1.000000  \n",
       "25%              149.000000      2735.000000  \n",
       "50%              203.000000      3861.000000  \n",
       "75%              263.750000      5469.000000  \n",
       "max             4767.000000    258598.000000  "
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# adapted from [1]\n",
    "#Making Dataframe\n",
    "dict_ = {'paper_id': [], 'source': [], 'title': [], 'abstract': [], 'body_text': [], 'bib_entries': []}\n",
    "for idx, entry in enumerate(all_json):\n",
    "    if idx % (len(all_json) // 10) == 0:\n",
    "        print(f'Processing index: {idx} of {len(all_json)}')\n",
    "    content = FileReader(entry)\n",
    "    dict_['paper_id'].append(content.paper_id)\n",
    "    dict_['source'].append(content.source)\n",
    "    dict_['title'].append(content.title)\n",
    "    dict_['abstract'].append(content.abstract)\n",
    "    dict_['body_text'].append(content.body_text)\n",
    "    dict_['bib_entries'].append(content.bib_entries)\n",
    "df_covid = pd.DataFrame(dict_, columns=['paper_id', 'source', 'title', 'abstract', 'body_text', 'bib_entries'])\n",
    "\n",
    "#Adding the Word Count ColumnÂ¶\n",
    "df_covid['abstract_word_count'] = df_covid['abstract'].apply(lambda x: len(x.strip().split()))\n",
    "df_covid['body_word_count'] = df_covid['body_text'].apply(lambda x: len(x.strip().split()))\n",
    "\n",
    "#Clean duplicates\n",
    "df_covid.drop_duplicates(['abstract'], inplace=True)\n",
    "df_covid.describe(include='all')\n",
    "\n",
    "## This will definitely take a while (5-10 min?)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_id</th>\n",
       "      <th>source</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>body_text</th>\n",
       "      <th>bib_entries</th>\n",
       "      <th>abstract_word_count</th>\n",
       "      <th>body_word_count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0015023cc06b5362d332b3baf348d11567ca2fbb</td>\n",
       "      <td>biorxiv_medrxiv</td>\n",
       "      <td>The RNA pseudoknots in foot-and-mouth disease ...</td>\n",
       "      <td>word count: 194 22 Text word count: 5168 23 24...</td>\n",
       "      <td>VP3, and VP0 (which is further processed to VP...</td>\n",
       "      <td>[\"{'ref_id': 'b0', 'title': 'Genetic economy i...</td>\n",
       "      <td>241</td>\n",
       "      <td>1728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>004f0f8bb66cf446678dc13cf2701feec4f36d76</td>\n",
       "      <td>biorxiv_medrxiv</td>\n",
       "      <td>Healthcare-resource-adjusted vulnerabilities t...</td>\n",
       "      <td></td>\n",
       "      <td>The 2019-nCoV epidemic has spread across China...</td>\n",
       "      <td>[\"{'ref_id': 'b0', 'title': 'World Health Orga...</td>\n",
       "      <td>0</td>\n",
       "      <td>755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>00d16927588fb04d4be0e6b269fc02f0d3c2aa7b</td>\n",
       "      <td>biorxiv_medrxiv</td>\n",
       "      <td>Real-time, MinION-based, amplicon sequencing f...</td>\n",
       "      <td>Infectious bronchitis (IB) causes significant ...</td>\n",
       "      <td>Infectious bronchitis (IB), which is caused by...</td>\n",
       "      <td>[\"{'ref_id': 'b0', 'title': 'Emergence of nove...</td>\n",
       "      <td>1647</td>\n",
       "      <td>4003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0139ea4ca580af99b602c6435368e7fdbefacb03</td>\n",
       "      <td>biorxiv_medrxiv</td>\n",
       "      <td>A Combined Evidence Approach to Prioritize Nip...</td>\n",
       "      <td>Nipah Virus (NiV) came into limelight recently...</td>\n",
       "      <td>Nipah is an infectious negative-sense single-s...</td>\n",
       "      <td>[\"{'ref_id': 'b0', 'title': 'Molecular biology...</td>\n",
       "      <td>326</td>\n",
       "      <td>2399</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>013d9d1cba8a54d5d3718c229b812d7cf91b6c89</td>\n",
       "      <td>biorxiv_medrxiv</td>\n",
       "      <td>Assessing spread risk of Wuhan novel coronavir...</td>\n",
       "      <td>Background: A novel coronavirus (2019-nCoV) em...</td>\n",
       "      <td>In December 2019, a cluster of patients with p...</td>\n",
       "      <td>[\"{'ref_id': 'b0', 'title': 'A Novel Coronavir...</td>\n",
       "      <td>22</td>\n",
       "      <td>4642</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                   paper_id           source  \\\n",
       "0  0015023cc06b5362d332b3baf348d11567ca2fbb  biorxiv_medrxiv   \n",
       "1  004f0f8bb66cf446678dc13cf2701feec4f36d76  biorxiv_medrxiv   \n",
       "2  00d16927588fb04d4be0e6b269fc02f0d3c2aa7b  biorxiv_medrxiv   \n",
       "3  0139ea4ca580af99b602c6435368e7fdbefacb03  biorxiv_medrxiv   \n",
       "4  013d9d1cba8a54d5d3718c229b812d7cf91b6c89  biorxiv_medrxiv   \n",
       "\n",
       "                                               title  \\\n",
       "0  The RNA pseudoknots in foot-and-mouth disease ...   \n",
       "1  Healthcare-resource-adjusted vulnerabilities t...   \n",
       "2  Real-time, MinION-based, amplicon sequencing f...   \n",
       "3  A Combined Evidence Approach to Prioritize Nip...   \n",
       "4  Assessing spread risk of Wuhan novel coronavir...   \n",
       "\n",
       "                                            abstract  \\\n",
       "0  word count: 194 22 Text word count: 5168 23 24...   \n",
       "1                                                      \n",
       "2  Infectious bronchitis (IB) causes significant ...   \n",
       "3  Nipah Virus (NiV) came into limelight recently...   \n",
       "4  Background: A novel coronavirus (2019-nCoV) em...   \n",
       "\n",
       "                                           body_text  \\\n",
       "0  VP3, and VP0 (which is further processed to VP...   \n",
       "1  The 2019-nCoV epidemic has spread across China...   \n",
       "2  Infectious bronchitis (IB), which is caused by...   \n",
       "3  Nipah is an infectious negative-sense single-s...   \n",
       "4  In December 2019, a cluster of patients with p...   \n",
       "\n",
       "                                         bib_entries  abstract_word_count  \\\n",
       "0  [\"{'ref_id': 'b0', 'title': 'Genetic economy i...                  241   \n",
       "1  [\"{'ref_id': 'b0', 'title': 'World Health Orga...                    0   \n",
       "2  [\"{'ref_id': 'b0', 'title': 'Emergence of nove...                 1647   \n",
       "3  [\"{'ref_id': 'b0', 'title': 'Molecular biology...                  326   \n",
       "4  [\"{'ref_id': 'b0', 'title': 'A Novel Coronavir...                   22   \n",
       "\n",
       "   body_word_count  \n",
       "0             1728  \n",
       "1              755  \n",
       "2             4003  \n",
       "3             2399  \n",
       "4             4642  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_covid.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# What to do next?\n",
    "The idea is to organize this information so that it can be understood more efficiently.\n",
    "\n",
    "How can we \"tag\" papers?\n",
    "\n",
    "After we tag them, how can we separate the tags into different classes (probably by frequency of occurrence if that's a good representaiton of generality)\n",
    "\n",
    "From the corpus that possesses these tags, how can we recompute the frequency of other tags within this corpus, then stratify those\n",
    "\n",
    "Rinse and repeat. \n",
    "\n",
    "We want to be able to go from general -> specific.\n",
    "\n",
    "This is one way.\n",
    "\n",
    "Idea: Titles usually have one noun which can be used as a tag\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "counts = df_covid['abstract'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "titles = df_covid['title'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\alexd\\Anaconda3\\envs\\kaggle\\lib\\site-packages\\ipykernel_launcher.py:1: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "  \"\"\"Entry point for launching an IPython kernel.\n"
     ]
    },
    {
     "ename": "PermissionError",
     "evalue": "[Errno 13] Permission denied: 'titles.csv'",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mPermissionError\u001b[0m                           Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-15-b4b98e29c6db>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[1;32m----> 1\u001b[1;33m \u001b[0mtitles\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mto_csv\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'titles.csv'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mindex\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;32mTrue\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[1;32m~\\Anaconda3\\envs\\kaggle\\lib\\site-packages\\pandas\\core\\series.py\u001b[0m in \u001b[0;36mto_csv\u001b[1;34m(self, *args, **kwargs)\u001b[0m\n\u001b[0;32m   4683\u001b[0m             )\n\u001b[0;32m   4684\u001b[0m             \u001b[0mkwargs\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;34m\"header\"\u001b[0m\u001b[1;33m]\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;32mFalse\u001b[0m  \u001b[1;31m# Backwards compatibility.\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 4685\u001b[1;33m         \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mto_frame\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mto_csv\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   4686\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4687\u001b[0m     \u001b[1;33m@\u001b[0m\u001b[0mAppender\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mgeneric\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_shared_docs\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;34m\"isna\"\u001b[0m\u001b[1;33m]\u001b[0m \u001b[1;33m%\u001b[0m \u001b[0m_shared_doc_kwargs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\envs\\kaggle\\lib\\site-packages\\pandas\\core\\generic.py\u001b[0m in \u001b[0;36mto_csv\u001b[1;34m(self, path_or_buf, sep, na_rep, float_format, columns, header, index, index_label, mode, encoding, compression, quoting, quotechar, line_terminator, chunksize, date_format, doublequote, escapechar, decimal)\u001b[0m\n\u001b[0;32m   3226\u001b[0m             \u001b[0mdecimal\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mdecimal\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   3227\u001b[0m         )\n\u001b[1;32m-> 3228\u001b[1;33m         \u001b[0mformatter\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0msave\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   3229\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   3230\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0mpath_or_buf\u001b[0m \u001b[1;32mis\u001b[0m \u001b[1;32mNone\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\envs\\kaggle\\lib\\site-packages\\pandas\\io\\formats\\csvs.py\u001b[0m in \u001b[0;36msave\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    181\u001b[0m                 \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mmode\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    182\u001b[0m                 \u001b[0mencoding\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mencoding\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 183\u001b[1;33m                 \u001b[0mcompression\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mcompression\u001b[0m\u001b[1;33m,\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    184\u001b[0m             )\n\u001b[0;32m    185\u001b[0m             \u001b[0mclose\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;32mTrue\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\envs\\kaggle\\lib\\site-packages\\pandas\\io\\common.py\u001b[0m in \u001b[0;36m_get_handle\u001b[1;34m(path_or_buf, mode, encoding, compression, memory_map, is_text)\u001b[0m\n\u001b[0;32m    397\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0mencoding\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    398\u001b[0m             \u001b[1;31m# Encoding\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 399\u001b[1;33m             \u001b[0mf\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mopen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mpath_or_buf\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmode\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mencoding\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mencoding\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mnewline\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;34m\"\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    400\u001b[0m         \u001b[1;32melif\u001b[0m \u001b[0mis_text\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    401\u001b[0m             \u001b[1;31m# No explicit encoding\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mPermissionError\u001b[0m: [Errno 13] Permission denied: 'titles.csv'"
     ]
    }
   ],
   "source": [
    "titles.to_csv('titles.csv', index = True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_id</th>\n",
       "      <th>source</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>body_text</th>\n",
       "      <th>bib_entries</th>\n",
       "      <th>abstract_word_count</th>\n",
       "      <th>body_word_count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>9401</th>\n",
       "      <td>ee06cad1229867a6389ed8a7aaffb5148dd5f8ed</td>\n",
       "      <td>comm_use_subset</td>\n",
       "      <td>A*02:06-p4 (LTA-J-FLIFL) A*02:07-p5 (LLDS-J-YE...</td>\n",
       "      <td>A Supporting Information Figure 1 A. Design of...</td>\n",
       "      <td>IT-TOF. The mass to charge ratios (m/z) of pep...</td>\n",
       "      <td>[\"{'ref_id': 'b1', 'title': 'Cytolytic T lymph...</td>\n",
       "      <td>307</td>\n",
       "      <td>330</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                      paper_id           source  \\\n",
       "9401  ee06cad1229867a6389ed8a7aaffb5148dd5f8ed  comm_use_subset   \n",
       "\n",
       "                                                  title  \\\n",
       "9401  A*02:06-p4 (LTA-J-FLIFL) A*02:07-p5 (LLDS-J-YE...   \n",
       "\n",
       "                                               abstract  \\\n",
       "9401  A Supporting Information Figure 1 A. Design of...   \n",
       "\n",
       "                                              body_text  \\\n",
       "9401  IT-TOF. The mass to charge ratios (m/z) of pep...   \n",
       "\n",
       "                                            bib_entries  abstract_word_count  \\\n",
       "9401  [\"{'ref_id': 'b1', 'title': 'Cytolytic T lymph...                  307   \n",
       "\n",
       "      body_word_count  \n",
       "9401              330  "
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = 'A*02:06-p4 (LTA-J-FLIFL) A*02:07-p5 (LLDS-J-YERL) A*02:07-p6 (LLDSD-J-ERL)'\n",
    "df_covid[df_covid.title == 'A*02:06-p4 (LTA-J-FLIFL) A*02:07-p5 (LLDS-J-YERL) A*02:07-p6 (LLDSD-J-ERL)']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"IT-TOF. The mass to charge ratios (m/z) of peptides were measured after liquid chromatography (LC) separation on a C18 column. As both isomers of the photocleavable 3-amino-3-(2nitro) phenyl-propanoic acid (Anp) used during solid phase peptide synthesis of the conditional ligand, 2 diasteroisomers could usually be identified. In some cases, however, the two isomers failed to separate during chromatography. The data displayed is for one of the two isomers, mostly for doubly charged molecules (M 2+ ), together with the amino acid sequences of conditional ligands where the letter J represents Anp.\\nA. Cells were first analyzed for their size and granularity based on their forward scatter area (FSC-A) and side scatter area (SSC-A) profile. Doublets were excluded using side-scatter height (SSC-H) vs side-scatter width (SSC-W) and forward-scatter height (FSC-H) vs forwardscatter width (FSC-W) parameters. Cells present in all three gates were analyzed for their time vs forward scatter area (FSC-A) to exclude cells that were read at the beginning and end of the flow cytometry run. Viable cells were gated based on their negative staining with the Live/DeadÂ® dye (Invitrogen), and these cells were finally analyzed for anti-human CD8 antibodies and HLA-tetramer stainings. All data were analyzed using FlowJo (Tree Star C. Culturing of PBMCs to develop antigen-specific T cell lines. PBMCs isolated from healthy volunteers' blood were stimulated by antigens from EBV-LMP2419-427 (TYGPVFMSL) or HCMV-IE-142-50 (KEVNSQLSL) and cultured in 25 U/mL IL-2, 5% AB serum in RPMI for 14 days. HLA tetramer staining for antigen-specific T cells was performed on day 0, 7, 10 and 14 days post stimulation. The conditional ligands are based on peptides found to be associated to the various HLA molecules. The epitope sequence, HLA restriction element, organism and protein source of the peptide and its location within the protein, and the literature references (Ref) are given. For (-), further information is not available, and ID stands for the epitope identification number in the immune epitope database and analysis resource (Immune epitope database. URL: http://www.immuneepitope.org/). \""
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_covid[df_covid.title == 'A*02:06-p4 (LTA-J-FLIFL) A*02:07-p5 (LLDS-J-YERL) A*02:07-p6 (LLDSD-J-ERL)'].iloc[0]['body_text']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Removing Stopwords and unnecessary text\n",
    "\n",
    "   - medrxiv https://www.medrxiv.org/content/medrxiv/early/2020/03/24/2020.03.22.20040915.full.pdf\n",
    "   - biorxiv https://www.biorxiv.org/content/biorxiv/early/2020/01/11/2020.01.10.901801.full.pdf\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "custom_license        10527\n",
       "comm_use_subset        8177\n",
       "noncomm_use_subset     1570\n",
       "biorxiv_medrxiv         776\n",
       "Name: source, dtype: int64"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_covid.source.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'In December 2019, a cluster of patients with pneumonia of unknown cause were reported in the city of Wuhan, Hubei Province, China, and epidemiologically linked to a seafood wholesale market [1, 2] . It has been determined that the pathogen causing the viral pneumonia among affected individuals is a new coronavirus (2019-nCoV) [1, 3] . The pathogen exhibits high human-to-human transmissibility and has spread rapidly within and beyond Wuhan city [4, 5] . On January 30 th , 2020, World Health Organization (WHO) declared the 2019-nCoV outbreak a Public Health Emergency of International Concern [6] .\\nWuhan is central China\\'s transportation hub with a population of 11 million residents and a large number of higher-education students (~1.3 million in 89 universities and colleges), a particularly mobile population [7] . Beyond these factors, viral spread was likely exacerbated further by the surge in domestic and international travel during the 40-day Lunar New Year (LNY) celebrations (from January 10 th , 2020 to February 18 th , 2020) -the largest annual human migration in the world, comprised of hundreds of millions of people travelling across the country. As of February 4 th , 2020, China has reported 20,530 confirmed cases and 23,314 suspected cases with 2019-nCoV infections [8] . Of the confirmed cases, 2788 are severe and 426 people have died. Most cases were reported from Wuhan and other cities in Hubei Province, and all provinces have confirmed cases imported from Wuhan and secondary transmission has been reported in some provinces. Additionally, there were 153 cases reported in 23 countries outside of China, with most having a travel history involving Wuhan [6] .\\nThe potential pathway from this local outbreak in Wuhan to a pandemic might involve four steps: i) local transmission in Wuhan (primary city of epidemic); ii) spread from Wuhan to other cities within and beyond China via infected travellers, causing sporadic secondary transmission in these cities (secondary city of epidemic), iii) further spreading from secondary cities with local transmission to other tertiary cities in China and international cities via returning travellers after the LNY holiday; iv) onward transmission across multiple countries and leading to a pandemic. To interrupt the spread, a cordon sanitaire of Wuhan and surrounding cities in Hubei Province has been in place since January 23 rd , 2020, just two days before the LNY\\'s Day. However, significant numbers of people had likely already travelled back to their hometowns for the holiday by this time. According to Wuhan authorities, it is . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint likely that more than five million residents had already left the city before the lockdown, but where they went and how high the risk of spreading the virus remains an open question [9] .\\nHere we conducted a travel network-based analysis to explore patterns of domestic and international population movements from high-risk cities in China, and provide preliminary estimates of the potential risk of 2019-nCoV spreading across and beyond the country. Given the current epidemic and limited understanding of the epidemiology of this disease, our findings on travel patterns from historical data can help contribute to tailoring public health interventions.\\nTo identify the areas that are most vulnerable to virus importation, we performed and integrated a series of analyses, by using de-identified and aggregated mobile phone-based population movement data, air passenger itinerary data, and case reports. We defined the potential risk and geographic range of 2019-nCoV virus spread across three scenarios: 1) from the primary city (Wuhan) into other cities in mainland China (31 provincial regions), 2) from high-risk secondary cities into other cities across China, and 3) from high-risk cities in mainland China into cities in other countries or regions during the LNY holiday and the following three months. We also estimated the number of airline travellers likely needing to be quarantined or screened to capture travellers potentially exposed to 2019-nCoV in high-risk cities of mainland China.\\nTo define daily patterns and the connectivity of population movements at county and prefecture (city) level across mainland China during the LNY holiday and the following three months, we used the aggregated and de-identified daily flow of the users of Baidu, the largest Chinese search engine [10] . Baidu offers location-based service (LBS), based on the global positioning system (GPS), IP address, location of signalling towers and WIFI, for online searching, mapping, and a large variety of apps and software for mobile devices. These data have been used to visualize population migration around Chinese New Year [10] . Two Baidu datasets were used in this study. The first one covers daily movement data at county level from December 1 st , 2013 to April 30 th , 2014, as described elsewhere [11] . We calculated relative . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint netflow following the equation below to extract daily patterns during LNY holiday at county level, with the population in 2014 obtained from the Chinese Bureau of Statistics [12] .\\nRelative netflow = (inflow -outflow) / population of each county\\nThe second dataset is a more recent daily movement matrix at the city level based on data from Baidu\\'s search app from January 1 st , 2015 to April 30 th , 2015. The last recorded locations of a user (or device) for each day were compared, and if the location changed, then we counted the user (or device) as someone who had moved from one city to another city. To understand the spread risk of 2019-nCoV from Wuhan into other cities via domestic population movement, we aggregated daily outflows of people from Wuhan to other cities across mainland China for the two weeks (the quarantine period of the virus) prior to the cordon sanitaire of Wuhan. As the travel ban took place on January 23 rd , 2020, just 2 days before LNY\\'s day, and given that LNY\\'s day in 2015 was February 19 th , we took February 17 th , 2015 as a reference of the lockdown day in our dataset. Using the second dataset, then the risk of importation for each destination city or province was preliminarily defined as the percentage of travellers received by each city or province out of the total volume of travellers leaving Wuhan during the two weeks before the city\\'s lockdown.\\nThese secondary cities may have a high risk of community-level transmission through the introduction of infected travellers from Wuhan, and then spread the virus to other tertiary cities by returning population movements after the LNY holiday, causing an even wider spread of the virus. As most of the cities in Hubei province have implemented the same travel controls as Wuhan before LNY, we defined the high-risk secondary cities outside of Hubei province as the cities within top 30 ranked cities (Supplementary Table S1 ) with the highest risk of importation from Wuhan defined above. Based on the 2015 Baidu dataset on population movement, the risk of spreading the virus from high-risk secondary cities to tertiary cities was preliminarily calculated as the averaged percentage of travellers received by each tertiary city out of the total volume of travellers leaving each high-risk secondary city during the four weeks following LNY\\'s Day. We chose a period of four weeks because the returning flow of LNY\\'s population movement, Chunyun, generally lasts four weeks.\\n. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint\\nTo define the connectivity and risk of 2019n-CoV spreading from Wuhan and high-risk secondary cities defined above, into the cities beyond mainland China, we obtained aggregated itinerary data from the International Air Travel Association (IATA) [13] . IATA data accounts for approximately 90% of passenger travel itineraries on commercial flights, and these data represent direct origin (Wuhan) to destination trips, and indirect trips that originated in Wuhan, but had connecting flights to a final destination. We quantified monthly volumes of airline travellers departing Wuhan and high-risk secondary cities from February 1 st , 2018, through April 30 th , 2018. With the assumption that the population movements around the LNY holiday in 2020 was consistent with the pattern in 2018, all final destinations were ranked by volumes of airline travellers, and the relative risk of importation was defined as the percentage of airline travellers received by each destination city out of the total volume of travellers leaving high-risk cities in China.\\nWe also estimated the number of airline travellers that may have needed to be kept in quarantine from Wuhan during the two weeks prior to the city\\'s travel ban. The LNY\\'s day in 2018 was on February 16 and the lockdown of Wuhan happened two days before LNY\\'s day, corresponding to the date of February 14 th , 2018. We therefore defined the number of travellers needing to be quarantined as half of the volume of airline travellers from Wuhan in February, 2018, representing the 2-week total number of travellers for the first half of February. We then estimated the number of infections and its 95% uncertainty interval (UI) in these airline travellers from Wuhan, based on a binomial distribution and the proportion of 2019-nCoV infections in the citizens evacuated from Wuhan reported by Singapore, South Korea, Japan, and Germany, as of January 31 st , 2020 [6, [14] [15] [16] [17] .\\nAdditionally, to capture travellers potentially exposed to virus, we also estimated the volume of airline travellers that would be required to be screened at origin high-risk cities in China and destinations across the globe for the following three months of February to April. Considering air traffic flows from China have changed due to the airline flight cancellations, travel restrictions imposed by countries or regions, or changes in travel behaviours, we calculated the volume of travellers using different scenarios of reductions (50%, 75%, and 90%) in total passenger volumes.\\n. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint\\nAs we used historial data to predict travel patterns in 2020, to ensure that the seasonal patterns observed over LNY and holidays more generally are consistent over multiple years and countries, we collated country-level domestic and international passenger statistics for air travel from 2010 to 2018, and compared these against a comprehensive time series of public and school holidays across the world during the same period (Supplementary Note). Additionally, we also compared the spatial patterns of the risks of Chinese cities importing the virus from Wuhan, estimated by the population movement data in 2014 and 2015, and more recent data on the top 50 ranked origin and destination cities in January 2020, available from the Baidu Migration site (https://qianxi.baidu.com/). To futher validate our results, we also compared the importation risk estimated in this study with the number of reported imported cases from Wuhan to other provinces in mainland China, as of January 25 th , 2020, and the number of imported cases reported by other countries or regions, as of February 3 rd , 2020 [6] . The distribution between days of travelling from Wuhan, illness onset, first medical visit, and hospitalization of imported cases, as of January 25 th , 2020, were also analysed. These case data were collated from the websites of WHO, national and local health authorities or new agencies within and beyond China (Supplementary Note). R version 3.6.1 (R Foundation for Statistical Computing, Vienna, Austria) was used to perform data analyses.\\nSignificant migratory flows occurred in opposite directions before and after LNY\\'s Day. In Wuhan City, mass movements of people began about three weeks prior to LNY, with the first peak of population leaving the city before the start of the winter holiday for universities, especially in the three counties that contain many universities and students (Figure 1 ). Although a cordon sanitaire of Wuhan and some cities in Hubei Province has been in place since January 23 rd , 2020, the timing of this may have occurred during the latter stages of peak population numbers leaving Wuhan, as another peak of movements out of the city was seen 2 days before LNY\\'s day.\\nWe found that a large number of travellers were likely departing Wuhan into neighbouring cities and other megacities in China before Wuhan\\'s . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint lockdown (Supplementary Tables S1). Other cities in Hubei Province received huge amounts of people during the two weeks before LNY and showed a decreasing population since LNY, following a peak of outflow at the end of the LNY holiday (Figure 1b and Supplementary Table S2 ). If Wuhan\\'s lockdown had not have been undertaken, our analyses suggest the main destination cities of population outflow since LNY would have been similar to the situation two weeks prior to LNY (Supplementary Table S3 ).\\nIn terms of the initial importation risk of virus for each city during the two weeks before Wuhan\\'s lockdown, nearly all other cities in Hubei Province were estimated to be high-risk areas (Figure 2a ). Other places with high risks were Beijing, Shanghai, Guangzhou and other large cities. In terms of the provincial level, the risks were high in Guangdong and Hunan, followed by Henan and Zhejiang (Figure 2b ). There was a significant correlation (r-squared = 0.59, p < 0.001) between the number of imported cases and the risk of importation estimated from traveller numbers from Wuhan within the two weeks before LNY\\'s Day (Figure 3a ). Further, a high proportion of cases travelled with symptoms at the early stage of the outbreak, and the lag from illness onset to hospitalization decreased from a median of 6 days (Interquartile range: 4-7 days) in the first half of January 2020 to 3 days (1-5 days) in the second half ( Figure 4 ).\\nAccording to our definition outlined in the methods, the 17 high-risk secondary cities outside of Hubei Province were identified as: Beijing, Shanghai, Guangzhou, Zhengzhou, Tianjin, Hangzhou, Jiaxing, Changsha, Xi\\'an, Nanjing, Shenzhen, Chongqing, Nanchang, Chengdu, Hefei, Fuzhou, and Dongguan ( Figure 5) . Should community-level outbreaks occur in these cities, they could contribute to further spread of infection to other highly connected cities within China via movement after the holiday (Supplementary Figures S1 and S2 ).\\nBased on historical air travel data, the connectivity and spread risk between high-risk cities in mainland China and cities in other countries or regions was defined for the three months around the LNY holiday (Supplementary Figure S3 and Tables S4 and S5). Bangkok, Hong Kong, and Taipei ranked in the top three, followed by Seoul, Tokyo and Singapore. The main destinations were presented by region in the supplemental materials (Supplementary Figures S3-S9 and Table S4 -S7). During the two weeks before Wuhan\\'s lockdown, there were an estimated total 59,912 airline . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint travellers from Wuhan that may have needed to be kept in quarantine at the 382 destinations outside of mainland China (Supplementary Table S8) . Thailand, Japan, and Taiwan ranked in the top three, followed by Malaysia, Hong Kong, and Singapore. Based on an overall infection rate of 1.39% (16/1149; 95% UI: 0.80% -2.25%) in citizens evacuated from Wuhan before Table S8 ). As of February 3 rd , 2020, a significant correlation (r-squared = 0.58, p < 0.001) was seen between the number of imported cases reported in those countries or regions and the risk of importation via travellers defined in our study (Figure 3b ).\\nBecause significant intranational and international spread from Wuhan has already occurred, a very large number of airline travellers (6.5 million under the scenario of 50% travel reduction as usual, 3.3 million under 75% reduction, and 1.3 million under 90% reduction, respectively) would be required to be screened at origin high-risk cities in China ( Figure 5 ) and destinations across the globe for the following three months of February to April, 2020 (Table 1 and  Supplementary Tables S9 and S10) , to ensure that all travellers from high risk cities were covered.\\nAddtionally, based on monthly air passenger travel statistics and time series of public and school holidays in 91 countries from 2010 to 2018 (Supplementary Note), we found the seasonal pattens of domestic and international population movements across years were highly consistent with the timing and duration of public and school holidays in different countries (Supplementary Figure S10 and S11) . Moreover, we also found similar spatial patterns in the risks of Chinese cities importing the virus from Wuhan via population movements estimated by both the Baidu data in 2014 and 2015, and the more recent data covering the top 50 ranked destinations in 2020, (Figure 1a and Supplementary Figure S12) , highlighting the value of using historical data to rapidly assess present day risks.\\nMobile phone-based population movement data and air passenger . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint itinerary data have been widely used to quantify the connectivity and transmission risk of pathogens via domestic and international human travel [18] [19] [20] [21] [22] [23] . Given the rapidly growing number of confirmed 2019-nCoV infections, increasing evidence of human-to-human transmission within and beyond China [4, 24] , and our limited understanding of this novel virus [25, 26] , the findings here from travel patterns in historical data and spread risk estimation can help guide public health preparedness and intervention design across the world [27] .\\nIn terms of domestic connectivity and risk, the high population outflows from three counties in Wuhan with many colleges and universities in the first two weeks of January were likely college students leaving the city to avoid the peak traffic just one week before the New Year. Because of the early timing of this movement, many students might have avoided the period when the virus spread rapidly in Wuhan, and their risk of spreading the virus may be low as well. However, our results suggest that during the two weeks prior to Wuhan\\'s travel ban, a large number of travellers still departed Wuhan into neighbouring cities and other megacities in China and may have spread the virus to new areas, as the timing of the lockdown occurred during the latter stages of peak population numbers leaving Wuhan. Further exacerbating this risk, we found that during the outbreak\\'s initial stages, a particularly high proportion of cases travelled with illness caused by the virus, together with the transmissibility of 2019-nCoV through asymptomatic contacts, potentially causing additional transmission during travel [28] .\\nMoreover, several destination cities ( Figure 5 ) outside of Hubei Province that received high volume of travellers from Wuhan prior to LNY\\'s day may serve as significant secondary cities in the outbreak. Most of these cities have large populations and international airports, highly connected with other regions within and beyond China. The initial imported seed cases likely caused the local community transmission, and further spread the virus into wider geographical ranges following the population flows occurring due to the LNY holiday [19] . Therefore, substantial public health interventions have been immediately applied across the country since LNY, including the cordon sanitaire in several of the most affected cities, cancellation of mass gatherings, reduction of travel and contact rate, as well as the extension of the LNY and school winter holiday, which might mitigate subsequent local establishment of 2019-nCoV introduced by travellers.\\nBeyond the cases that have occurred in China, air passengers have spread 2019-nCoV across countries and continents within a short time period [28, 29] . In particular, a high volume of international airline travellers left . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint Wuhan for hundreds of destination cities across the world during the two weeks prior to the travel restriction implemented in the city. Substantial preparedness efforts at destination cities should be taken to prevent further international seeding and transmission, otherwise the local establishment of epidemics, and even a pandemic, might become inevitable. For example, exit and entry screening may be futher extended to capture travellers with fever who have potentially been exposed to the virus in high-risk Chinese cities where local transmission has been established. However, we estimated that a huge volume of airline travellers would be required to be screened in February to April, even under the scenario of significant reduction (90%) in air passengers compared with the same period of previous years. Therefore, ensuring that surveillance and health systems around the world are ready and sufficiently strong to detect and deal with cases seen is a priority.\\nIt is expected that further international exportation of cases from China will occur and cases may appear in any country [30] . Thus, all countries should be prepared for containment, including active surveillance, early detection, isolation and case management, contact tracing and prevention of onward spread of the 2019-nCoV infection [6, 31] . Of additional concern is that the anticipated destinations of hundreds of thousands of travellers departing China are to low-income or lower-middle income countries, where inadequately resourced medical and public health systems might be unable to detect and adequately manage an imported case of 2019n-CoV, including possible subsequent community spread.\\nDue to current limited knowledge of the epidemiology of the virus at the time of writing (e.g. the proportion and infectiousness of asymptomatic or subclinical infections) and the rapidly changing situation of the outbreak, the simplicity of our approach to define importation risk can help to quickly update risk assessments, prioritise surveillance, target limited resources and understand the potential of 2019n-CoV introduction at specific destinations [32] . Compared with other studies [33, 34] , we explored the various scenarios of travel restriction and used a more comprehensive and spatio-temporally detailed population movement matrix, together with details on the actual final destination cities of air passengers based on the global itinerary dataset. These novel datasets provide new insights on the impacts of internal and international connectivity on potential transmission of this emerging pathogen during the LNY holiday and over the next three months.\\nNevertheless, it is important to note that our study has several major limitations. Firstly, while we do present simple scenarios of reduced air travel volumes, our primary analyses assume \"business as usual\" travel based on . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint previous non-outbreak years, when significant spatio-temporal changes to human travel behaviours across and beyond China have likely occurred recently. Second, the mobile phone data used may provide an incomplete and biased picture of travellers, as the data only cover the population owning a smart phone and using Baidu apps. Third, the case data used in this study likely varies in quality and completeness due to the timeliness of reporting, varying laboratory diagnosis capacities, and differences in details announced on health authority websites. Fourth, compared with airline travellers leaving Wuhan prior to January 23 rd evacuees from Wuhan during the January 29 th -31 st period might have a higher risk of infection due to their longer stay in Wuhan during the potential continued spread of the virus since January 23 rd . This may result in overestimates of the number of infections in airline travellers from Wuhan prior to the city\\'s lockdown. Based on more recent population movement and epidemiological data, we aim to conduct more sophisticated modelling approaches to assess the effectiveness of control measures in China, the impact of movements of people returning from LNY holiday, as well as the risks of a 2019-nCoV global pandemic.\\n. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint\\nEthical clearance for collecting and using secondary data in this study was granted by the institutional review board of the University of Southampton (No. 48002). All data were supplied and analysed in an anonymous format, without access to personal identifying information. 1 4 IIB: Data collection and analysis, interpreted the findings and commented on and revised drafts of the manuscript NWR: Data collection and analysis, interpreted the findings and commented on and revised drafts of the manuscript AW: Data collection and analysis, interpreted the findings and commented on and revised drafts of the manuscript XL: Data collection and analysis, interpreted the findings and commented on and revised drafts of the manuscript\\nThe corresponding authors had full access to all the data in the study and had final responsibility for the decision to submit for publication. The corresponding authors attest that all listed authors meet authorship criteria and that no others meeting the criteria have been omitted.\\nThe datasets on monthly air passenger data in February -April, 2018 used in this study are available from Dr. Kamran Khan (Kamran@bluedot.global). The case data are available from Dr. Shengjie Lai (Shengjie.Lai@soton.ac.uk). The datasets on holidays and air travel statistics from 2010 through 2018 used for validation are available on the WorldPop website (www.worldpop.org). The mobile phone datasets analysed during the current study are not publicly available since this would compromise the agreement with the data provider, but information on the process of requesting access to the data that support the findings of this study are available from Dr. Shengjie Lai (Shengjie.Lai@soton.ac.uk).\\n. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nis the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.04.20020479 doi: medRxiv preprint the first half (N=67) and second half (N=80) of January, 2020, respectively. A total of 164 cases with available data as of January 25th, 2020, were included.\\nThe risk of importation at city level was preliminarily defined as the averaged percentage of travellers received by each city out of the total volume of travellers leaving each high-risk city, based on the population movement data in 2015 obtained from Baidu, Inc. The high-risk cities include Wuhan in Hubei province and 17 cities (Beijing, Shanghai, Guangzhou, Zhengzhou, Tianjin, Hangzhou, Jiaxing, Changsha, Xi\\'an, Nanjing, Shenzhen, Chongqing, Nanchang, Chengdu, Hefei, Fuzhou, and Dongguan) in other provinces receiving high volume of travellers from Wuhan during the two weeks before the city\\'s lockdown on January 23 rd , 2020.\\n. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. '"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_covid.body_text[4]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[4,\n",
       " 5,\n",
       " 11,\n",
       " 12,\n",
       " 13,\n",
       " 16,\n",
       " 21,\n",
       " 22,\n",
       " 26,\n",
       " 29,\n",
       " 30,\n",
       " 32,\n",
       " 34,\n",
       " 40,\n",
       " 41,\n",
       " 42,\n",
       " 44,\n",
       " 45,\n",
       " 46,\n",
       " 48,\n",
       " 52,\n",
       " 53,\n",
       " 55,\n",
       " 56,\n",
       " 60,\n",
       " 61,\n",
       " 65,\n",
       " 66,\n",
       " 67,\n",
       " 68,\n",
       " 70,\n",
       " 71,\n",
       " 72,\n",
       " 73,\n",
       " 78,\n",
       " 82,\n",
       " 84,\n",
       " 90,\n",
       " 94,\n",
       " 96,\n",
       " 101,\n",
       " 102,\n",
       " 104,\n",
       " 105,\n",
       " 109,\n",
       " 111,\n",
       " 113,\n",
       " 114,\n",
       " 116,\n",
       " 121,\n",
       " 122,\n",
       " 125,\n",
       " 126,\n",
       " 127,\n",
       " 128,\n",
       " 129,\n",
       " 132,\n",
       " 134,\n",
       " 137,\n",
       " 143,\n",
       " 145,\n",
       " 148,\n",
       " 153,\n",
       " 155,\n",
       " 160,\n",
       " 161,\n",
       " 162,\n",
       " 167,\n",
       " 169,\n",
       " 171,\n",
       " 174,\n",
       " 178,\n",
       " 180,\n",
       " 181,\n",
       " 184,\n",
       " 189,\n",
       " 190,\n",
       " 194,\n",
       " 196,\n",
       " 198,\n",
       " 201,\n",
       " 204,\n",
       " 208,\n",
       " 210,\n",
       " 218,\n",
       " 219,\n",
       " 220,\n",
       " 221,\n",
       " 222,\n",
       " 223,\n",
       " 228,\n",
       " 229,\n",
       " 232,\n",
       " 233,\n",
       " 236,\n",
       " 238,\n",
       " 244,\n",
       " 249,\n",
       " 250,\n",
       " 253,\n",
       " 255,\n",
       " 257,\n",
       " 260,\n",
       " 261,\n",
       " 262,\n",
       " 263,\n",
       " 264,\n",
       " 267,\n",
       " 271,\n",
       " 272,\n",
       " 273,\n",
       " 276,\n",
       " 278,\n",
       " 281,\n",
       " 282,\n",
       " 286,\n",
       " 287,\n",
       " 288,\n",
       " 289,\n",
       " 291,\n",
       " 292,\n",
       " 298,\n",
       " 299,\n",
       " 300,\n",
       " 301,\n",
       " 304,\n",
       " 308,\n",
       " 316,\n",
       " 317,\n",
       " 318,\n",
       " 324,\n",
       " 334,\n",
       " 339,\n",
       " 341,\n",
       " 343,\n",
       " 345,\n",
       " 346,\n",
       " 347,\n",
       " 350,\n",
       " 356,\n",
       " 359,\n",
       " 361,\n",
       " 366,\n",
       " 367,\n",
       " 371,\n",
       " 373,\n",
       " 374,\n",
       " 383,\n",
       " 385,\n",
       " 386,\n",
       " 388,\n",
       " 389,\n",
       " 390,\n",
       " 391,\n",
       " 393,\n",
       " 401,\n",
       " 402,\n",
       " 403,\n",
       " 407,\n",
       " 410,\n",
       " 411,\n",
       " 412,\n",
       " 414,\n",
       " 415,\n",
       " 418,\n",
       " 422,\n",
       " 426,\n",
       " 432,\n",
       " 438,\n",
       " 440,\n",
       " 444,\n",
       " 445,\n",
       " 448,\n",
       " 450,\n",
       " 452,\n",
       " 456,\n",
       " 458,\n",
       " 459,\n",
       " 461,\n",
       " 464,\n",
       " 466,\n",
       " 470,\n",
       " 472,\n",
       " 473,\n",
       " 474,\n",
       " 481,\n",
       " 482,\n",
       " 485,\n",
       " 486,\n",
       " 488,\n",
       " 493,\n",
       " 494,\n",
       " 504,\n",
       " 511,\n",
       " 512,\n",
       " 519,\n",
       " 520,\n",
       " 522,\n",
       " 523,\n",
       " 524,\n",
       " 525,\n",
       " 527,\n",
       " 532,\n",
       " 535,\n",
       " 536,\n",
       " 538,\n",
       " 540,\n",
       " 541,\n",
       " 548,\n",
       " 549,\n",
       " 552,\n",
       " 553,\n",
       " 554,\n",
       " 555,\n",
       " 557,\n",
       " 558,\n",
       " 560,\n",
       " 563,\n",
       " 565,\n",
       " 567,\n",
       " 572,\n",
       " 575,\n",
       " 576,\n",
       " 577,\n",
       " 579,\n",
       " 583,\n",
       " 584,\n",
       " 587,\n",
       " 589,\n",
       " 591,\n",
       " 593,\n",
       " 595,\n",
       " 596,\n",
       " 597,\n",
       " 599,\n",
       " 600,\n",
       " 601,\n",
       " 604,\n",
       " 611,\n",
       " 614,\n",
       " 620,\n",
       " 622,\n",
       " 623,\n",
       " 632,\n",
       " 635,\n",
       " 636,\n",
       " 639,\n",
       " 643,\n",
       " 645,\n",
       " 648,\n",
       " 652,\n",
       " 653,\n",
       " 656,\n",
       " 659,\n",
       " 662,\n",
       " 664,\n",
       " 665,\n",
       " 666,\n",
       " 673,\n",
       " 676,\n",
       " 681,\n",
       " 683,\n",
       " 690,\n",
       " 691,\n",
       " 696,\n",
       " 701,\n",
       " 704,\n",
       " 705,\n",
       " 707,\n",
       " 709,\n",
       " 713,\n",
       " 715,\n",
       " 716,\n",
       " 718,\n",
       " 722,\n",
       " 723,\n",
       " 724,\n",
       " 725,\n",
       " 726,\n",
       " 727,\n",
       " 728,\n",
       " 729,\n",
       " 730,\n",
       " 734,\n",
       " 735,\n",
       " 737,\n",
       " 738,\n",
       " 739,\n",
       " 741,\n",
       " 742,\n",
       " 754,\n",
       " 755,\n",
       " 757,\n",
       " 762,\n",
       " 763,\n",
       " 765,\n",
       " 770]"
      ]
     },
     "execution_count": 65,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "medRxiv = []\n",
    "for i, each in enumerate(df_covid.body_text):\n",
    "    if ('medRxiv' in each):\n",
    "        medRxiv.append(i)\n",
    "medRxiv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'The sudden outbreak of the new coronavirus (SARS-CoV-2) at the end of December 2019 poses a huge threat to human health worldwide. The SARS-CoV-2 virus causes severe respiratory disease that can quickly spread from person to person and in some cases lead to death.\\nResearchers have found that the new SARS-CoV-2 and SARS coronaviruses invade human cells in target tissues in a similar manner via high-affinity binding to angiotensin-converting enzyme 2 (ACE2) [1] . In recent epidemiological investigations of the spread of the SARS-CoV-2 and a preliminary study of the clinical characteristics of this disease [2] [3] [4] [5] [6] , researchers have found that patients infected with the new coronavirus have severe symptoms similar to those of the SARS infection. The first batch of clinical data reports of SARS-CoV-2 infection cases in China revealed \"cytokine storms\" in critically ill patients [7, 8] . However, the mechanism of the viral infection and pathological changes in the immune system is still lacking. The sooner this information is added to the current clinical knowledge of these viruses, the better the control and treatment of this disease.\\nHere, we present an automated topic-wise inference method called TWIRLS (Topic-wise inference engine of massive biomedical literatures) for processing the massive biomedical literature to summarize coronavirus host-related entities. TWIRLS is capable of collecting, classifying, and analyzing reported coronavirus studies to reveal these entities based on the distribution of specific genes in the text of the articles. By combining with general protein interaction data, links between certain functional cellular/physiological components can be inferred to fill the knowledge gaps on the probable mechanism of host pathological changes. eventually leads to acute lung injury in the host. Therefore, TWIRLS can be used to guide human researchers by providing further potential therapeutic target information for the treatment of acute viral lung injury based on the regulation of RAS.\\nCoronavirus-study specific entities and host genes As of February 21, 2020, the PubMed database included 14,878 biomedical articles on coronaviruses. We obtained text data (called local samples) from all related articles on the coronavirus that had been peer reviewed and published by human experts, which included the title, abstracts, author and affiliation information (total 3,182,687 words). The goal of the literature mining was to identify host genes and entities that are relevant to coronavirus research and to establish connections between them. An entity can refers to a word or phrase of the concept name (including related concepts, e.g., virus structure and chemical composition, source of infection, and virus type). The gene names were defined using the mammalian official gene symbols in the Hugo Gene Naming Committee (HGNC) database. We directly retrieved 667 candidate genes from the local samples. By establishing a random distribution of one of the candidate genes in a control sample, the significance of this gene appearing in the local samples can be determined when the frequency of the current gene is an outlier of the random distribution of the control samples (see Methods for details). By calculating the odds ratio, we can also further determine the specificity of the association between this gene and the local samples. In this paper, we selected an odds ratio > 6 as the threshold for this judgment, which resulted in 123 coronavirus study-specific host genes (CSHGs).\\nTo determine the specificity of the entity, we made a choice between different texts in the local samples. We removed numbers, symbols, verbs, and garbled characters to obtain clean versions of the local samples. The coronavirus study-specific entities (CSSE) were then identified in only the clean texts containing CSHGs. Based on the clean selected samples, we next built a local dictionary of candidate CSSEs containing 49,293 words after deduplication. Before calculating the random distribution of each entity, we included the synonymous entities into a same entity number (including singular or plural words, active and passive forms, different tenses, suffixes that do not change the meaning, etc.). For example, synonymous entities such as coronaviral, coronavirus, coronaviruses were grouped into one entity as coronavirus and assigned the same number (see entity number in Table S1 , Sheet 1 first column). The previous method of merging synonymous entities was based on a dictionary [9, 10] , which not only relied on the integrity of the dictionary, but also required a long retrieval time. To automatically solve the synonymous entity problem, TWIRLS classifies similar strings based on whether there is a significant statistical association between the character blocks in a set of candidate entities including various synonymous entities (see Methods). After cleaning and processing, CSSEs were identified by TWIRLS using a similar method to that for CSHG as described above.\\nFor the candidate CSSE dictionary, a random distribution model for each entity was built by TWIRLS using the control samples. We identified 623 CSSEs (Table Sl, Sheet 1) based on the outliers discriminated by the random model and calculated odds ratio. For example, TWIRLS found 100 CSSEs close to ACE2, the receptor of SARS and SARS-CoV-2 viruses (see left panel in Figure 1 ). The size of the entity represents the relative distance to ACE2, with a larger size indicating a closer distance to ACE2. Additionally, we present the CSSE cloud of the human receptor gene DPP4 of the MERS virus (see right panel in Figure 1 ).\\nAlthough TWIRLS only identified 623 CSSEs after collation, for human researchers, the information is scattered in words, which is limited for reconstructing understandable mechanistic models. Therefore, TWIRLS clusters CSSEs according to the rules defined by CSHG distribution, as genetic level research can accurately answer and solve physiological and pathological problems.\\nTWIRLS first calculated the specific co-distribution between CSHGs in local samples, then determined the distance between each pair of CSSEs and performs dichotomy clustering according to the linkage relationship between CSSEs and CSHGs. This classified the 623 entities into 32 categories represented as C0-C31 (see category number in Table S1 , Sheet 1 second column). In addition, for each category, TWIRLS also cited the top ten most relevant references for human researchers (Table S2) . Therefore, in any category, according to the CSSE and the most relevant literature, we can quickly provide \"Labels of conclusion-drawn-by-human-researcher\" (HR Labels) for this category. This label outlines the most relevant research directions of the current entity category. For example, for category C3, the HR label is \"Neurotrophic Coronavirus Related to Immune-Mediated Demyelination\". We have summarized the HR labels for the 32 entity categories in Table 1 .\\nThe relative position of any CSHG to a certain CSSE can be estimated by TWIRLS (see Table S1 , the ranking matrix in Sheet 1). As each category contains different entities, we can determine whether a certain CSHG is significantly closer to each entity in the current category based on the ranking matrix between CSHG and CSSE. For example, the average distance between ACE2 and the 92 entities in category C5 is first calculated, then a random distribution model of the average distance between ACE2 and any of the 92 entities (3000-5000 times) is built, and finally, we determine if the average distance between ACE2 and entities in category C5 is significantly less than and deviates from the mean of the random distribution (Z score = -5.8416). The significance of each category associated with each CSHG is then scored by TWIRLS ranging between -10 and +10, with a smaller score indicating the current CSHG is more relevant to the current category (see the Z-score matrix in Table S1 , sheet 2). For an entity category, the associated CSHGs (e.g., Ci CSHGs, where i represents the category number) can thus be selected by a Z score <-3 (the Z scores describing the association between CSHG and any category is summarized in Sheet2 of Table S1 , and the category labels of all CSHGs are provided in Sheet 3).\\nSpecifically, Spike proteins (S proteins) of different coronaviruses recognize different receptor molecules on human cells, such as ACE2 (binds to Spike proteins in SARS and SARS-CoV-2 virus) and DPP4 (binds to Spike protein in MERS virus). We found that these two genes are assigned to the C5 category, which has a corresponding HR label of \"Spike protein (S) of coronavirus\", suggesting that TWIRLS can automatically provide an interface to summarize human findings and help human experts quickly understand the research directions and necessary knowledge in this field.\\nThe distribution and meaning of the data can be compared to specific expression values of CSHG under different conditions (here, the category is used as a condition). Therefore, based on the All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint distribution of the pathway signatures, TWIRLS can recommend the most likely and least likely signaling pathways (Table 2) . On the other hand, TWIRLS can also recommend the most likely and least likely categories for each signaling pathway. As an example, Table 3 shows the signaling pathways most likely associated with category C3 with the most unlikely corresponding category.\\nWe coupled the data with gene interaction/regulation databases and constructed a generalized protein-protein interaction network (PPI network) among 119 genes out of the 123 CSHGs. We defined the direct interaction between two genes as a 1 degree (1 o ) interaction, and the indirect interaction connecting two genes through a gene as a 2 degree (2 o ) interaction. All the genes in the 1 o networks mined in the PPI database are shown in Figure 2 . The results after deduplication showed 2,004 pairs in the 119 CSHGs (see Table S1 , Sheet 4). As a control, the average interactions of 119 randomly selected genes in the database showed between 252 to 612 pairs (average 220.16, standard deviation 35.15). Compared to random genes, the regulatory connections between CSHGs were significantly enriched (Z score = 50.97).\\nThose CSHGs associated with a certain category had much closer interactions. For example, CSHGs associated with category C3 (or associated with C5 or C10) were closer to each other in the 1 o networks (Figure 2 ), suggesting that TWIRLS can possibly highlight important research directions and biology systems involved in coronavirus-specific research and can provide reliable interfaces for further automatic inference.\\nSeveral hub genes among the 119 CSHGs were further recommended by TWIRLS. Compared to a random sampling from all interactions recorded in the database, these hub genes had significantly increased numbers of interactions with the other 118 CSHGs. The recommended results showed that the three members of the IFITMs family (IFITM1-3) ranked first, second, and sixth among the top ten hub genes (CSSE cloud of the IFITMs family genes is shown in Figure 3 ; detailed ranking recommendation results are shown in Table S1 , Sheet 5). These IFITMs genes showed 115 interactions, accounting for 8.59% out of all 1,338 interactions of the 119 CSHGs. These IFITMs were significantly enriched in the local samples representing updated coronavirus-related All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint studies (average 0.03% in the control test of random samplings, p <1.5676e-61). The IFITMs family plays crucial roles in the induction of interferons during viral infections. Under the action of interferon, IFITMs disrupts intracellular cholesterol homeostasis and prevents the virus from entering the host cell [11] . However, TWIRLS did not directly associate IFITMs with any category, so we needed to provide more information so that TWIRLS can determine which part of these genes might be involved in the coronavirus infection and host body response.\\nCombining with generalized interaction databases provides richer interactions and regulatory linkages. We extended the 119 CSHGs to their 2 o networks based on the interactions with higher likelihood of connections (Combined score> 800). The 2 o networks expanded the number of genes from 119 host genes to 3,494 genes that may be associated with coronavirus (see Table S1 , Sheet 6 with DPP4 [12] . The different distribution of these receptors in the respiratory tract results in different degrees of infection. Although the infection ability of MERS is lower than in SARS, the mortality is higher (in about one-third of patients) because of the deeper infection site [13] . Similar to the SARS virus, viral genomics and structural biology studies have shown that ACE2 is also a functional receptor for the new SARS-CoV-2 coronavirus. After binding to ACE2 via its Spike protein, SARS-CoV-2 undergoes membrane fusion and enters the host cells by endocytosis. The ACE2 peptidase is a key regulator of the Renin-Angiotensin System (RAS). It is highly expressed in the heart, kidney, and testis, and is also expressed at lower levels in other tissues (mainly in the intestine and lungs) [14, 15] . Recent studies have shown that the binding of the S protein to ACE2 in the new coronavirus is 10 to 20 times stronger than in the SARS virus [16] , which may help the new coronavirus infect the host through the upper respiratory tract, significantly increasing its infectivity. Using TWIRLS, we were able to identify both ACE2 and DPP4 genes as CSHGs, and both were significantly associated with the C5 category. The HR label for this category is \"associated with S protein.\"\\nIn addition to ACE2 and DPP4, other CSHGs that are significantly associated with the C5 category include FURIN and TMPRSS2. The former may be required for the H7N1 and H5N1 influenza virus infections, probably via hemagglutinin-induced lysis, whereas the latter is widely reported to mediate and assist in the invasion of host cells by multiple viruses. Transmembrane protease serine 2 (TMPRSS2) is a serine protease that hydrolyzes and activates the spike glycoproteins of human coronavirus 229E (HCoV-229E), human coronavirus EMC (HCoV-EMC), Sendai virus (SeV) and human interstitial pneumovirus (HMPV), and 1,2,3 fusion glycoproteins of F0, 4a, and 4b human parainfluenza viruses (HPIV) [17, 18] . The function of this gene is essential for the transmission and pathogenesis of influenza A viruses (H1N1, H3N2 and H7N9 strains). It is also involved in the hydrolysis and activation of hemagglutinin proteins, which are essential for viral infectivity [19, 20] . Although entities in the C5 category and in the cited literature mainly show that virus invasion is facilitated by virus-binding receptors and membrane proteases, the biological mechanism of the receptor binding to viruses leading to pathological changes has been reported less frequently. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint TWIRLS recommend new genes that interact with C5 CSHGs, and other 1 o or 2 o CSGHs linked to this gene might be enriched in other categories. These inferences are based on a process that finds new genes connected to different categories. The connected categories suggested potential regulatory relationships between different biological functions or phenotypes. The genes that serve as linkers are potential targets for gain-and loss-of-function experiments to identify those systems described by the meaningful entities in these categories.\\nIn this paper, TWIRLS found the 2 o networks showed connections with certain CSHGs associated with categories or with no category. For example, TWIRLS found that CSHGs in the 2 o connections of IFITM1 were mainly concentrated in the C3 category (see Figure 4 ). Interestingly, CSHGs in the 2 o connections of ACE2 and DPP4 associated with C5 category were also enriched in C3 category, inferring that the information summarized in C3 category probably describe the underlying mechanisms of the pathological changes after coronavirus infection. In our analysis, the signaling pathways in C3 were mainly RAS, Vitamin D and RXR activation, and Chemokine signaling, with RAS being the most significant (as shown in Table 3 , which summarizes C3-related signaling pathways). which then linked to C3-associated cytokines including CCL5, CXCL1, CXCL10, CXCL11, CXCL2, CXCL9, CXCR2, and CXCR3 ( Figure 5 ). Subsequently, these linker genes may contain information on the biological mechanisms that may be important for understanding the disease.\\nFor example, TWIRLS recommended angiotensinogen (AGT) and angiotensin II receptor type 2 (AGTR2 or AT2R) genes in the C3 category associated with ACE2. This supports that RAS is probably involved in the pathological changes caused by cytokine storms after S protein binds to ACE2, as suggested by other reports.\\nWe next used TWIRLS to calculate the 1 o and 2 o networks of all 119 CSHGs. Based on the significantly enriched categories of CSHGs in the above networks, TWIRLS separately constructed models for the complex relationships of each CSHG. We found that 45.53% of the All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint CSHGs in these networks were associated with C3 or C10 categories, and five genes (CCL3, CCL5, CXCL1, CXCL2, and STAT2) were associated with both. This suggests that the biological mechanisms described by the C3 and C10 categories might be universally involved. Research on the entities, genes, pathways, and linker genes participating in the C3 and C10 categories could lead to new directions for the prevention, treatment, and clinical management of coronavirus infections.\\nIn this study, we used TWIRLS, a machine-based approach to collect, summarize, and analyze about 15,000 biomedical articles related to coronavirus, with the aim to elucidate the mechanisms underlying coronavirus-induced host pathological changes. Using TWIRLS, we found a possible mechanism involving ACE2/AT2R-RAS-Cytokine signaling, which becomes imbalanced under virus infection leading to cytokine storms. The TWIRLS system is an automated process that can summarize the entities and genes specifically related to coronaviruses. By combining this system with generalized interaction databases, we can reveal further associations that can provide a deeper understanding of the biological mechanisms of the disease phenotype caused by virus-host interactions.\\nvirus to ACE2 may disrupt this balance, which causes a steady-state imbalance of RAS, leading to subsequent pathological changes.\\nAlthough Ang II was originally described as an effective vasoconstrictor, there is growing evidence that it is closely involved in the inflammatory response of the immune system.\\nPro-inflammatory cytokines derived from immune cells normally regulate the RAS component, which further accelerates the formation of systemic and local Ang II [28] [29] [30] . In particular, pro-inflammatory cytokines regulate the production of AGT in the liver and kidney [31] [32] [33] . On the other hand, RAS has also been implicated in mediating the cytokine storm and has functional relationships with the immune system. Angiotensin II regulates vascular tension and stimulates the release of pro-inflammatory cytokines [34, 35] . The production and release of CXC chemokines can induce the accumulation of neutrophils in vivo [36] . Meanwhile, ACE inhibitors and Ang II receptor blockers have been used in a number of cytokine-mediated inflammatory pathologies, and AT1R blockers (angiotensin receptor blocker) were shown to have beneficial effects that were commonly attributed to AT2R activation [37] . At the same time, it was reported that Ang II-stimulated human endothelial cells had increased release of a CXC chemokine, IP-10. The IFN-Î³-inducible protein 10 (IP-10 or CXCL10) is mainly expressed in the lung and is a chemoattractant for activated T cells. The expression of IP-10 has been observed in many Th1-type inflammatory diseases, where it is thought to play an important role in recruiting activated T cells to sites of tissue inflammation. Therefore, RAS dysfunction may result in the accumulation of cytokines, such as in the lungs leading to excessive accumulation of immune cells and interstitial fluid, blocking the airways and causing eventual death. As reported in the first severely infected patients diagnosed with COVID-19, a large number of patients experienced \"cytokine storms\" that was fatal [7] . Figure 6 summarizes the functional changes and pathological consequences of RAS system after ACE2 combines with the coronavirus.\\nWe expect the mechanism summarized and reasoned by TWIRLS can be further supported by pathological evidence. To date, only one report of a post-mortem biopsy has been published with pathological data. Although histological examination showed bilateral diffuse alveolar damage with cellular fibromyxoid exudates, the right lung showed evidence of desquamation of All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint pneumocytes and hyaline membrane formation, indicating acute respiratory distress syndrome (ARDS), whereas the left lung showed pulmonary edema with hyaline membrane formation, suggestive of early-phase ARDS. The pathological evidence suggests that ARDS symptoms are closely related to cytokine storm [38] . However, there is still a lack of histopathology-related data to support our preliminary findings generated by our machine approach.\\nAll rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint\\nWe used PubMed, the most widely used database of biological literature, as the resource for the text mining. The schematic representation of the overall study design is shown in Figure 1 and can be summarized in the following steps.\\nThe dataset used in this pipeline were from PubMed articles. First, PubMed was searched for articles including titles, abstracts, author and affiliation information containing the subject keyword \"coronavirus\". The search results were downloaded in txt format for compiling into structured information. The text in the subject abstract set was organized and cleaned, and then assigned to specific corpuses related to coronavirus (specific corpus) and compiled into the subject dictionary. To enhance the accuracy of the effective entities associated with the keyword, we used a random corpus for comparisons. We searched for article abstracts containing the keyword \"public health\" and compiled the abstract set into a random corpus, and then compiled them into a randomized control dictionary, which contains a wide range of proteins, genes, and related biological entities. We also considered a balanced amount of information by setting relevant parameters to adjust the amount of text before carrying out the statistical analyses.\\nBiological entity identification is a key step in the literature mining process [7, 8] . To ensure functionality of the extracted entity, we first compared the entity from the subject dictionary with the human official gene symbols in the Hugo Gene Nomenclature Commission (HGNC) database [9] to generate subject candidate genes using standard nomenclature. In addition, the entities in the abstract were capitalized to avoid errors in the identification process. To obtain widely used gene entities that are precisely related to the subject and to determine the significance of the gene distribution in the specific texts, we calculated the difference in the distribution proportions. We searched for the subject candidate genes in the subject dictionary and the randomized control dictionary, respectively. We also counted the number of abstracts containing each subject candidate gene in each abstract set, respectively. Finally, we calculated the odds ratio of each subject candidate gene and sorted them into a list of precisely related genes (CSHG).\\nSimilar to the process of identifying CSHG, we calculated whether entities were significantly distributed in a specific corpus. We counted the number of texts containing each CSHG in a specific corpus, and then counted the number of each candidate entity in the corpus subset. Next, we randomly selected the same amount of text from the random control corpus and then counted the number of each candidate entity in this subset of the random corpus. This was repeated 100-10000 times in the random corpus to generate candidate entities in the specified amount of text of the random distribution model. According to the central limit theorem (CLT), the distribution of random sampling averages of randomly distributed data always conforms to a normal distribution. Therefore, we can use the Z score to evaluate whether an entity is significant in a specific text. Here, we used a Z score cutoff value > 6.\\nIn addition, some entities have singular and plural noun forms, and synonyms with multiple forms in the abstracts. Therefore, we numbered the subject-related entity and automatically combined nouns with plural forms and homologous words with adjectives and adverb roots into the same entity, and then assigned them the same number. Figure 1 . the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint CSHGs, the blue nodes represent genes that interact with CSHG in the string database (combination score> 800), and the red squares mark the most relevant entity category of CSHG.\\nAll rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . The yellow nodes represent the ACE2, DPP4 and IFITM1 genes, purple nodes represent genes that have 1 degree of interaction with the core genes, green circled purple nodes represent the genes connecting CSHG and C3 category-related genes, and pink nodes represent genes with 2 degrees of interaction with the core gene. The red diamonds show the most relevant entity category symbol for CSHG. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint Tables   Table 1. Coronavirus-entity category labels and genes associated with each category. MISC indicates the label cannot be summarized. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\\nThe copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint '"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_covid.body_text[5]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'The sudden outbreak of the new coronavirus (SARS-CoV-2) at the end of December 2019 poses a huge threat to human health worldwide. The SARS-CoV-2 virus causes severe respiratory disease that can quickly spread from person to person and in some cases lead to death. Researchers have found that the new SARS-CoV-2 and SARS coronaviruses invade human cells in target tissues in a similar manner via high-affinity binding to angiotensin-converting enzyme 2 (ACE2) [1] . In recent epidemiological investigations of the spread of the SARS-CoV-2 and a preliminary study of the clinical characteristics of this disease [2] [3] [4] [5] [6] , researchers have found that patients infected with the new coronavirus have severe symptoms similar to those of the SARS infection. The first batch of clinical data reports of SARS-CoV-2 infection cases in China revealed \"cytokine storms\" in critically ill patients [7, 8] . However, the mechanism of the viral infection and pathological changes in the immune system is still lacking. The sooner this information is added to the current clinical knowledge of these viruses, the better the control and treatment of this disease. Here, we present an automated topic-wise inference method called TWIRLS (Topic-wise inference engine of massive biomedical literatures) for processing the massive biomedical literature to summarize coronavirus host-related entities. TWIRLS is capable of collecting, classifying, and analyzing reported coronavirus studies to reveal these entities based on the distribution of specific genes in the text of the articles. By combining with general protein interaction data, links between certain functional cellular/physiological components can be inferred to fill the knowledge gaps on the probable mechanism of host pathological changes. eventually leads to acute lung injury in the host. Therefore, TWIRLS can be used to guide human researchers by providing further potential therapeutic target information for the treatment of acute viral lung injury based on the regulation of RAS. Coronavirus-study specific entities and host genes As of February 21, 2020, the PubMed database included 14,878 biomedical articles on coronaviruses. We obtained text data (called local samples) from all related articles on the coronavirus that had been peer reviewed and published by human experts, which included the title, abstracts, author and affiliation information (total 3,182,687 words). The goal of the literature mining was to identify host genes and entities that are relevant to coronavirus research and to establish connections between them. An entity can refers to a word or phrase of the concept name (including related concepts, e.g., virus structure and chemical composition, source of infection, and virus type). The gene names were defined using the mammalian official gene symbols in the Hugo Gene Naming Committee (HGNC) database. We directly retrieved 667 candidate genes from the local samples. By establishing a random distribution of one of the candidate genes in a control sample, the significance of this gene appearing in the local samples can be determined when the frequency of the current gene is an outlier of the random distribution of the control samples (see Methods for details). By calculating the odds ratio, we can also further determine the specificity of the association between this gene and the local samples. In this paper, we selected an odds ratio > 6 as the threshold for this judgment, which resulted in 123 coronavirus study-specific host genes (CSHGs). To determine the specificity of the entity, we made a choice between different texts in the local samples. We removed numbers, symbols, verbs, and garbled characters to obtain clean versions of the local samples. The coronavirus study-specific entities (CSSE) were then identified in only the clean texts containing CSHGs. Based on the clean selected samples, we next built a local dictionary of candidate CSSEs containing 49,293 words after deduplication. Before calculating the random distribution of each entity, we included the synonymous entities into a same entity number (including singular or plural words, active and passive forms, different tenses, suffixes that do not change the meaning, etc.). For example, synonymous entities such as coronaviral, coronavirus, coronaviruses were grouped into one entity as coronavirus and assigned the same number (see entity number in Table S1 , Sheet 1 first column). The previous method of merging synonymous entities was based on a dictionary [9, 10] , which not only relied on the integrity of the dictionary, but also required a long retrieval time. To automatically solve the synonymous entity problem, TWIRLS classifies similar strings based on whether there is a significant statistical association between the character blocks in a set of candidate entities including various synonymous entities (see Methods). After cleaning and processing, CSSEs were identified by TWIRLS using a similar method to that for CSHG as described above. For the candidate CSSE dictionary, a random distribution model for each entity was built by TWIRLS using the control samples. We identified 623 CSSEs (Table Sl, Sheet 1) based on the outliers discriminated by the random model and calculated odds ratio. For example, TWIRLS found 100 CSSEs close to ACE2, the receptor of SARS and SARS-CoV-2 viruses (see left panel in Figure 1 ). The size of the entity represents the relative distance to ACE2, with a larger size indicating a closer distance to ACE2. Additionally, we present the CSSE cloud of the human receptor gene DPP4 of the MERS virus (see right panel in Figure 1 ). Although TWIRLS only identified 623 CSSEs after collation, for human researchers, the information is scattered in words, which is limited for reconstructing understandable mechanistic models. Therefore, TWIRLS clusters CSSEs according to the rules defined by CSHG distribution, as genetic level research can accurately answer and solve physiological and pathological problems. TWIRLS first calculated the specific co-distribution between CSHGs in local samples, then determined the distance between each pair of CSSEs and performs dichotomy clustering according to the linkage relationship between CSSEs and CSHGs. This classified the 623 entities into 32 categories represented as C0-C31 (see category number in Table S1 , Sheet 1 second column). In addition, for each category, TWIRLS also cited the top ten most relevant references for human researchers (Table S2) . Therefore, in any category, according to the CSSE and the most relevant literature, we can quickly provide \"Labels of conclusion-drawn-by-human-researcher\" (HR Labels) for this category. This label outlines the most relevant research directions of the current entity category. For example, for category C3, the HR label is \"Neurotrophic Coronavirus Related to Immune-Mediated Demyelination\". We have summarized the HR labels for the 32 entity categories in Table 1 . The relative position of any CSHG to a certain CSSE can be estimated by TWIRLS (see Table S1 , the ranking matrix in Sheet 1). As each category contains different entities, we can determine whether a certain CSHG is significantly closer to each entity in the current category based on the ranking matrix between CSHG and CSSE. For example, the average distance between ACE2 and the 92 entities in category C5 is first calculated, then a random distribution model of the average distance between ACE2 and any of the 92 entities (3000-5000 times) is built, and finally, we determine if the average distance between ACE2 and entities in category C5 is significantly less than and deviates from the mean of the random distribution (Z score = -5.8416). The significance of each category associated with each CSHG is then scored by TWIRLS ranging between -10 and +10, with a smaller score indicating the current CSHG is more relevant to the current category (see the Z-score matrix in Table S1 , sheet 2). For an entity category, the associated CSHGs (e.g., Ci CSHGs, where i represents the category number) can thus be selected by a Z score <-3 (the Z scores describing the association between CSHG and any category is summarized in Sheet2 of Table S1 , and the category labels of all CSHGs are provided in Sheet 3). Specifically, Spike proteins (S proteins) of different coronaviruses recognize different receptor molecules on human cells, such as ACE2 (binds to Spike proteins in SARS and SARS-CoV-2 virus) and DPP4 (binds to Spike protein in MERS virus). We found that these two genes are assigned to the C5 category, which has a corresponding HR label of \"Spike protein (S) of coronavirus\", suggesting that TWIRLS can automatically provide an interface to summarize human findings and help human experts quickly understand the research directions and necessary knowledge in this field. The distribution and meaning of the data can be compared to specific expression values of CSHG under different conditions (here, the category is used as a condition). Therefore, based on the All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint distribution of the pathway signatures, TWIRLS can recommend the most likely and least likely signaling pathways (Table 2) . On the other hand, TWIRLS can also recommend the most likely and least likely categories for each signaling pathway. As an example, Table 3 shows the signaling pathways most likely associated with category C3 with the most unlikely corresponding category. We coupled the data with gene interaction/regulation databases and constructed a generalized protein-protein interaction network (PPI network) among 119 genes out of the 123 CSHGs. We defined the direct interaction between two genes as a 1 degree (1 o ) interaction, and the indirect interaction connecting two genes through a gene as a 2 degree (2 o ) interaction. All the genes in the 1 o networks mined in the PPI database are shown in Figure 2 . The results after deduplication showed 2,004 pairs in the 119 CSHGs (see Table S1 , Sheet 4). As a control, the average interactions of 119 randomly selected genes in the database showed between 252 to 612 pairs (average 220.16, standard deviation 35.15). Compared to random genes, the regulatory connections between CSHGs were significantly enriched (Z score = 50.97). Those CSHGs associated with a certain category had much closer interactions. For example, CSHGs associated with category C3 (or associated with C5 or C10) were closer to each other in the 1 o networks (Figure 2 ), suggesting that TWIRLS can possibly highlight important research directions and biology systems involved in coronavirus-specific research and can provide reliable interfaces for further automatic inference. Several hub genes among the 119 CSHGs were further recommended by TWIRLS. Compared to a random sampling from all interactions recorded in the database, these hub genes had significantly increased numbers of interactions with the other 118 CSHGs. The recommended results showed that the three members of the IFITMs family (IFITM1-3) ranked first, second, and sixth among the top ten hub genes (CSSE cloud of the IFITMs family genes is shown in Figure 3 ; detailed ranking recommendation results are shown in Table S1 , Sheet 5). These IFITMs genes showed 115 interactions, accounting for 8.59% out of all 1,338 interactions of the 119 CSHGs. These IFITMs were significantly enriched in the local samples representing updated coronavirus-related All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint studies (average 0.03% in the control test of random samplings, p <1.5676e-61). The IFITMs family plays crucial roles in the induction of interferons during viral infections. Under the action of interferon, IFITMs disrupts intracellular cholesterol homeostasis and prevents the virus from entering the host cell [11] . However, TWIRLS did not directly associate IFITMs with any category, so we needed to provide more information so that TWIRLS can determine which part of these genes might be involved in the coronavirus infection and host body response. Combining with generalized interaction databases provides richer interactions and regulatory linkages. We extended the 119 CSHGs to their 2 o networks based on the interactions with higher likelihood of connections (Combined score> 800). The 2 o networks expanded the number of genes from 119 host genes to 3,494 genes that may be associated with coronavirus (see Table S1 , Sheet 6 with DPP4 [12] . The different distribution of these receptors in the respiratory tract results in different degrees of infection. Although the infection ability of MERS is lower than in SARS, the mortality is higher (in about one-third of patients) because of the deeper infection site [13] . Similar to the SARS virus, viral genomics and structural biology studies have shown that ACE2 is also a functional receptor for the new SARS-CoV-2 coronavirus. After binding to ACE2 via its Spike protein, SARS-CoV-2 undergoes membrane fusion and enters the host cells by endocytosis. The ACE2 peptidase is a key regulator of the Renin-Angiotensin System (RAS). It is highly expressed in the heart, kidney, and testis, and is also expressed at lower levels in other tissues (mainly in the intestine and lungs) [14, 15] . Recent studies have shown that the binding of the S protein to ACE2 in the new coronavirus is 10 to 20 times stronger than in the SARS virus [16] , which may help the new coronavirus infect the host through the upper respiratory tract, significantly increasing its infectivity. Using TWIRLS, we were able to identify both ACE2 and DPP4 genes as CSHGs, and both were significantly associated with the C5 category. The HR label for this category is \"associated with S protein.\" In addition to ACE2 and DPP4, other CSHGs that are significantly associated with the C5 category include FURIN and TMPRSS2. The former may be required for the H7N1 and H5N1 influenza virus infections, probably via hemagglutinin-induced lysis, whereas the latter is widely reported to mediate and assist in the invasion of host cells by multiple viruses. Transmembrane protease serine 2 (TMPRSS2) is a serine protease that hydrolyzes and activates the spike glycoproteins of human coronavirus 229E (HCoV-229E), human coronavirus EMC (HCoV-EMC), Sendai virus (SeV) and human interstitial pneumovirus (HMPV), and 1,2,3 fusion glycoproteins of F0, 4a, and 4b human parainfluenza viruses (HPIV) [17, 18] . The function of this gene is essential for the transmission and pathogenesis of influenza A viruses (H1N1, H3N2 and H7N9 strains). It is also involved in the hydrolysis and activation of hemagglutinin proteins, which are essential for viral infectivity [19, 20] . Although entities in the C5 category and in the cited literature mainly show that virus invasion is facilitated by virus-binding receptors and membrane proteases, the biological mechanism of the receptor binding to viruses leading to pathological changes has been reported less frequently. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint TWIRLS recommend new genes that interact with C5 CSHGs, and other 1 o or 2 o CSGHs linked to this gene might be enriched in other categories. These inferences are based on a process that finds new genes connected to different categories. The connected categories suggested potential regulatory relationships between different biological functions or phenotypes. The genes that serve as linkers are potential targets for gain-and loss-of-function experiments to identify those systems described by the meaningful entities in these categories. In this paper, TWIRLS found the 2 o networks showed connections with certain CSHGs associated with categories or with no category. For example, TWIRLS found that CSHGs in the 2 o connections of IFITM1 were mainly concentrated in the C3 category (see Figure 4 ). Interestingly, CSHGs in the 2 o connections of ACE2 and DPP4 associated with C5 category were also enriched in C3 category, inferring that the information summarized in C3 category probably describe the underlying mechanisms of the pathological changes after coronavirus infection. In our analysis, the signaling pathways in C3 were mainly RAS, Vitamin D and RXR activation, and Chemokine signaling, with RAS being the most significant (as shown in Table 3 , which summarizes C3-related signaling pathways). which then linked to C3-associated cytokines including CCL5, CXCL1, CXCL10, CXCL11, CXCL2, CXCL9, CXCR2, and CXCR3 ( Figure 5 ). Subsequently, these linker genes may contain information on the biological mechanisms that may be important for understanding the disease. For example, TWIRLS recommended angiotensinogen (AGT) and angiotensin II receptor type 2 (AGTR2 or AT2R) genes in the C3 category associated with ACE2. This supports that RAS is probably involved in the pathological changes caused by cytokine storms after S protein binds to ACE2, as suggested by other reports. We next used TWIRLS to calculate the 1 o and 2 o networks of all 119 CSHGs. Based on the significantly enriched categories of CSHGs in the above networks, TWIRLS separately constructed models for the complex relationships of each CSHG. We found that 45.53% of the All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint CSHGs in these networks were associated with C3 or C10 categories, and five genes (CCL3, CCL5, CXCL1, CXCL2, and STAT2) were associated with both. This suggests that the biological mechanisms described by the C3 and C10 categories might be universally involved. Research on the entities, genes, pathways, and linker genes participating in the C3 and C10 categories could lead to new directions for the prevention, treatment, and clinical management of coronavirus infections. In this study, we used TWIRLS, a machine-based approach to collect, summarize, and analyze about 15,000 biomedical articles related to coronavirus, with the aim to elucidate the mechanisms underlying coronavirus-induced host pathological changes. Using TWIRLS, we found a possible mechanism involving ACE2/AT2R-RAS-Cytokine signaling, which becomes imbalanced under virus infection leading to cytokine storms. The TWIRLS system is an automated process that can summarize the entities and genes specifically related to coronaviruses. By combining this system with generalized interaction databases, we can reveal further associations that can provide a deeper understanding of the biological mechanisms of the disease phenotype caused by virus-host interactions. virus to ACE2 may disrupt this balance, which causes a steady-state imbalance of RAS, leading to subsequent pathological changes. Although Ang II was originally described as an effective vasoconstrictor, there is growing evidence that it is closely involved in the inflammatory response of the immune system. Pro-inflammatory cytokines derived from immune cells normally regulate the RAS component, which further accelerates the formation of systemic and local Ang II [28] [29] [30] . In particular, pro-inflammatory cytokines regulate the production of AGT in the liver and kidney [31] [32] [33] . On the other hand, RAS has also been implicated in mediating the cytokine storm and has functional relationships with the immune system. Angiotensin II regulates vascular tension and stimulates the release of pro-inflammatory cytokines [34, 35] . The production and release of CXC chemokines can induce the accumulation of neutrophils in vivo [36] . Meanwhile, ACE inhibitors and Ang II receptor blockers have been used in a number of cytokine-mediated inflammatory pathologies, and AT1R blockers (angiotensin receptor blocker) were shown to have beneficial effects that were commonly attributed to AT2R activation [37] . At the same time, it was reported that Ang II-stimulated human endothelial cells had increased release of a CXC chemokine, IP-10. The IFN-Î³-inducible protein 10 (IP-10 or CXCL10) is mainly expressed in the lung and is a chemoattractant for activated T cells. The expression of IP-10 has been observed in many Th1-type inflammatory diseases, where it is thought to play an important role in recruiting activated T cells to sites of tissue inflammation. Therefore, RAS dysfunction may result in the accumulation of cytokines, such as in the lungs leading to excessive accumulation of immune cells and interstitial fluid, blocking the airways and causing eventual death. As reported in the first severely infected patients diagnosed with COVID-19, a large number of patients experienced \"cytokine storms\" that was fatal [7] . Figure 6 summarizes the functional changes and pathological consequences of RAS system after ACE2 combines with the coronavirus. We expect the mechanism summarized and reasoned by TWIRLS can be further supported by pathological evidence. To date, only one report of a post-mortem biopsy has been published with pathological data. Although histological examination showed bilateral diffuse alveolar damage with cellular fibromyxoid exudates, the right lung showed evidence of desquamation of All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint pneumocytes and hyaline membrane formation, indicating acute respiratory distress syndrome (ARDS), whereas the left lung showed pulmonary edema with hyaline membrane formation, suggestive of early-phase ARDS. The pathological evidence suggests that ARDS symptoms are closely related to cytokine storm [38] . However, there is still a lack of histopathology-related data to support our preliminary findings generated by our machine approach. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint We used PubMed, the most widely used database of biological literature, as the resource for the text mining. The schematic representation of the overall study design is shown in Figure 1 and can be summarized in the following steps. The dataset used in this pipeline were from PubMed articles. First, PubMed was searched for articles including titles, abstracts, author and affiliation information containing the subject keyword \"coronavirus\". The search results were downloaded in txt format for compiling into structured information. The text in the subject abstract set was organized and cleaned, and then assigned to specific corpuses related to coronavirus (specific corpus) and compiled into the subject dictionary. To enhance the accuracy of the effective entities associated with the keyword, we used a random corpus for comparisons. We searched for article abstracts containing the keyword \"public health\" and compiled the abstract set into a random corpus, and then compiled them into a randomized control dictionary, which contains a wide range of proteins, genes, and related biological entities. We also considered a balanced amount of information by setting relevant parameters to adjust the amount of text before carrying out the statistical analyses. Biological entity identification is a key step in the literature mining process [7, 8] . To ensure functionality of the extracted entity, we first compared the entity from the subject dictionary with the human official gene symbols in the Hugo Gene Nomenclature Commission (HGNC) database [9] to generate subject candidate genes using standard nomenclature. In addition, the entities in the abstract were capitalized to avoid errors in the identification process. To obtain widely used gene entities that are precisely related to the subject and to determine the significance of the gene distribution in the specific texts, we calculated the difference in the distribution proportions. We searched for the subject candidate genes in the subject dictionary and the randomized control dictionary, respectively. We also counted the number of abstracts containing each subject candidate gene in each abstract set, respectively. Finally, we calculated the odds ratio of each subject candidate gene and sorted them into a list of precisely related genes (CSHG). Similar to the process of identifying CSHG, we calculated whether entities were significantly distributed in a specific corpus. We counted the number of texts containing each CSHG in a specific corpus, and then counted the number of each candidate entity in the corpus subset. Next, we randomly selected the same amount of text from the random control corpus and then counted the number of each candidate entity in this subset of the random corpus. This was repeated 100-10000 times in the random corpus to generate candidate entities in the specified amount of text of the random distribution model. According to the central limit theorem (CLT), the distribution of random sampling averages of randomly distributed data always conforms to a normal distribution. Therefore, we can use the Z score to evaluate whether an entity is significant in a specific text. Here, we used a Z score cutoff value > 6. In addition, some entities have singular and plural noun forms, and synonyms with multiple forms in the abstracts. Therefore, we numbered the subject-related entity and automatically combined nouns with plural forms and homologous words with adjectives and adverb roots into the same entity, and then assigned them the same number. Figure 1 . the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint CSHGs, the blue nodes represent genes that interact with CSHG in the string database (combination score> 800), and the red squares mark the most relevant entity category of CSHG. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . The yellow nodes represent the ACE2, DPP4 and IFITM1 genes, purple nodes represent genes that have 1 degree of interaction with the core genes, green circled purple nodes represent the genes connecting CSHG and C3 category-related genes, and pink nodes represent genes with 2 degrees of interaction with the core gene. The red diamonds show the most relevant entity category symbol for CSHG. All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02. 24.20025437 doi: medRxiv preprint Tables   Table 1. Coronavirus-entity category labels and genes associated with each category. MISC indicates the label cannot be summarized. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.02.24.20025437 doi: medRxiv preprint '"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reg = '(|(CC-BY-NC 4.0 International license It is made available under a)| (author\\/funder, who has granted medRxiv a license to display the preprint in perpetuity.)|(is the \\(which was not peer-reviewed\\))|(The copyright holder for this preprint)|(The copyright holder for this preprint . )|(https:\\/\\/doi.org\\/10.[0-9]{4,}?\\/[0-9]{4}.[0-9]{2}. [0-9]{2}.[^\\s]+)|(https:\\/\\/doi.org\\/10.[0-9]{4,}?\\/[0-9]{4}.[0-9]{2}.[0-9]{2}.[^\\s]+)| (doi: medRxiv preprint)|(author\\/funder. All rights reserved. No reuse allowed without permission.))'\n",
    "g = re.sub('\\\\n', ' ', df_covid.body_text[5])\n",
    "re.sub(reg, '', g)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
